{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\saveryme\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\utils.py:1209: UserWarning: detected Windows; aliasing chunkize to chunkize_serial\n",
      "  warnings.warn(\"detected Windows; aliasing chunkize to chunkize_serial\")\n"
     ]
    }
   ],
   "source": [
    "import gensim\n",
    "from gensim import corpora, models, similarities\n",
    "from gensim.models.doc2vec import Doc2Vec, TaggedDocument\n",
    "import pymongo as pm\n",
    "import pandas as pd\n",
    "import numpy as np\n",
    "import re\n",
    "from sklearn.model_selection import train_test_split\n",
    "from nltk.tokenize import word_tokenize\n",
    "from sklearn.model_selection import RepeatedStratifiedKFold\n",
    "from sklearn.metrics import f1_score\n",
    "from sklearn.metrics import recall_score\n",
    "from sklearn.metrics import accuracy_score\n",
    "from sklearn.metrics import precision_score\n",
    "from sklearn.metrics import confusion_matrix"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "client = pm.MongoClient('localhost', 27017)\n",
    "db = client['full_texts'] # creates database if not there\n",
    "#documents = db['texts'] # creates new collection if not there\n",
    "documents = db['train_val']"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "get data with each label"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "df_conclusion = pd.DataFrame(list(documents.find({'label': 'conclusion'}).limit(10000)))\n",
    "df_result = pd.DataFrame(list(documents.find({'label': 'result'}).limit(10000)))\n",
    "df_method = pd.DataFrame(list(documents.find({'label': 'method'}).limit(10000)))\n",
    "df_intro = pd.DataFrame(list(documents.find({'label': 'introduction'}).limit(10000)))\n",
    "\n",
    "dfs = [df_conclusion, df_result, df_method, df_intro]\n",
    "\n",
    "# merge the dfs!\n",
    "merged_df = pd.concat(dfs)\n",
    "merged_df.sort_values('date', inplace = True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "write data to tsv to check stuff out..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df.to_csv(\"dev_data.tsv\", sep = \"\\t\", index = False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "drop short rows"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(38933, 6)\n"
     ]
    }
   ],
   "source": [
    "merged_df = merged_df.loc[merged_df['text'].map(len) > 400]\n",
    "merged_df.reset_index(inplace = True, drop = True)\n",
    "print(merged_df.shape)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df = merged_df.sample(200) # use a small set for playin'"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "convert labels for binary classification"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [],
   "source": [
    "pattern = r'^(?!method).*$'\n",
    "merged_df['label'].replace(pattern, 'other', regex = True, inplace = True) # check this is working"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "split up sections into sentences"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "from nltk.tokenize import sent_tokenize"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "merged_df['sentences'] = merged_df.apply(lambda row: sent_tokenize(row['text']), axis=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "sentences = []\n",
    "label = []\n",
    "doc_id = []\n",
    "\n",
    "for i, row in merged_df.iterrows():\n",
    "    for sentence in row['sentences']:\n",
    "        sentences.append(sentence)\n",
    "        label.append(row['label'])\n",
    "        doc_id.append(row['article'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "sentence_df = pd.DataFrame({'id': doc_id, 'label': label, 'sentence': sentences})"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Split up the data. Note that I am in a development phase and really the test set is a validation set."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "my non-random way"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyError",
     "evalue": "31999",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mget_slice_bound\u001b[1;34m(self, label, side, kind)\u001b[0m\n\u001b[0;32m   4223\u001b[0m             \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4224\u001b[1;33m                 \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_searchsorted_monotonic\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mlabel\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mside\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4225\u001b[0m             \u001b[1;32mexcept\u001b[0m \u001b[0mValueError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36m_searchsorted_monotonic\u001b[1;34m(self, label, side)\u001b[0m\n\u001b[0;32m   4182\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4183\u001b[1;33m         \u001b[1;32mraise\u001b[0m \u001b[0mValueError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'index must be monotonic increasing or decreasing'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4184\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mValueError\u001b[0m: index must be monotonic increasing or decreasing",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-53-b1bb171aff70>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m()\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0mX_train\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmerged_df\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mloc\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;36m31999\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'text'\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      2\u001b[0m \u001b[0my_train\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmerged_df\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mloc\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;36m31999\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'label'\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0misin\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m'conclusion'\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mshape\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      4\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0misin\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m'method'\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mshape\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      5\u001b[0m \u001b[0mprint\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0my_train\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0misin\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m'introduction'\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mshape\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m__getitem__\u001b[1;34m(self, key)\u001b[0m\n\u001b[0;32m   1470\u001b[0m             \u001b[1;32mexcept\u001b[0m \u001b[1;33m(\u001b[0m\u001b[0mKeyError\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mIndexError\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1471\u001b[0m                 \u001b[1;32mpass\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 1472\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_getitem_tuple\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1473\u001b[0m         \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1474\u001b[0m             \u001b[1;31m# we by definition only have the 0th axis\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m_getitem_tuple\u001b[1;34m(self, tup)\u001b[0m\n\u001b[0;32m    868\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0m_getitem_tuple\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtup\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    869\u001b[0m         \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 870\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_getitem_lowerdim\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mtup\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    871\u001b[0m         \u001b[1;32mexcept\u001b[0m \u001b[0mIndexingError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    872\u001b[0m             \u001b[1;32mpass\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m_getitem_lowerdim\u001b[1;34m(self, tup)\u001b[0m\n\u001b[0;32m   1025\u001b[0m                     \u001b[1;32mreturn\u001b[0m \u001b[0msection\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1026\u001b[0m                 \u001b[1;31m# This is an elided recursive call to iloc/loc/etc'\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 1027\u001b[1;33m                 \u001b[1;32mreturn\u001b[0m \u001b[0mgetattr\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0msection\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mname\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mnew_key\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1028\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1029\u001b[0m         \u001b[1;32mraise\u001b[0m \u001b[0mIndexingError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m'not applicable'\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m__getitem__\u001b[1;34m(self, key)\u001b[0m\n\u001b[0;32m   1476\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1477\u001b[0m             \u001b[0mmaybe_callable\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mcom\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_apply_if_callable\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mobj\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 1478\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_getitem_axis\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mmaybe_callable\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1479\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1480\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0m_is_scalar_access\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m_getitem_axis\u001b[1;34m(self, key, axis)\u001b[0m\n\u001b[0;32m   1864\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mslice\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1865\u001b[0m             \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_validate_key\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 1866\u001b[1;33m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_get_slice_axis\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1867\u001b[0m         \u001b[1;32melif\u001b[0m \u001b[0mcom\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mis_bool_indexer\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1868\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_getbool_axis\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0maxis\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0maxis\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexing.py\u001b[0m in \u001b[0;36m_get_slice_axis\u001b[1;34m(self, slice_obj, axis)\u001b[0m\n\u001b[0;32m   1509\u001b[0m         \u001b[0mlabels\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mobj\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_get_axis\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0maxis\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1510\u001b[0m         indexer = labels.slice_indexer(slice_obj.start, slice_obj.stop,\n\u001b[1;32m-> 1511\u001b[1;33m                                        slice_obj.step, kind=self.name)\n\u001b[0m\u001b[0;32m   1512\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1513\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mindexer\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mslice\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mslice_indexer\u001b[1;34m(self, start, end, step, kind)\u001b[0m\n\u001b[0;32m   4088\u001b[0m         \"\"\"\n\u001b[0;32m   4089\u001b[0m         start_slice, end_slice = self.slice_locs(start, end, step=step,\n\u001b[1;32m-> 4090\u001b[1;33m                                                  kind=kind)\n\u001b[0m\u001b[0;32m   4091\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4092\u001b[0m         \u001b[1;31m# return a slice\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mslice_locs\u001b[1;34m(self, start, end, step, kind)\u001b[0m\n\u001b[0;32m   4295\u001b[0m         \u001b[0mend_slice\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4296\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mend\u001b[0m \u001b[1;32mis\u001b[0m \u001b[1;32mnot\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4297\u001b[1;33m             \u001b[0mend_slice\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_slice_bound\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mend\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;34m'right'\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mkind\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4298\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0mend_slice\u001b[0m \u001b[1;32mis\u001b[0m \u001b[1;32mNone\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4299\u001b[0m             \u001b[0mend_slice\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mlen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mget_slice_bound\u001b[1;34m(self, label, side, kind)\u001b[0m\n\u001b[0;32m   4225\u001b[0m             \u001b[1;32mexcept\u001b[0m \u001b[0mValueError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4226\u001b[0m                 \u001b[1;31m# raise the original KeyError\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4227\u001b[1;33m                 \u001b[1;32mraise\u001b[0m \u001b[0merr\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4228\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4229\u001b[0m         \u001b[1;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mslc\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mnp\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mndarray\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mget_slice_bound\u001b[1;34m(self, label, side, kind)\u001b[0m\n\u001b[0;32m   4219\u001b[0m         \u001b[1;31m# we need to look up the label\u001b[0m\u001b[1;33m\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4220\u001b[0m         \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 4221\u001b[1;33m             \u001b[0mslc\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_get_loc_only_exact_matches\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mlabel\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4222\u001b[0m         \u001b[1;32mexcept\u001b[0m \u001b[0mKeyError\u001b[0m \u001b[1;32mas\u001b[0m \u001b[0merr\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4223\u001b[0m             \u001b[1;32mtry\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36m_get_loc_only_exact_matches\u001b[1;34m(self, key)\u001b[0m\n\u001b[0;32m   4188\u001b[0m         \u001b[0mget_slice_bound\u001b[0m\u001b[1;33m.\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4189\u001b[0m         \"\"\"\n\u001b[1;32m-> 4190\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_loc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   4191\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   4192\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mget_slice_bound\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mlabel\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mside\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mkind\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\pandas\\core\\indexes\\base.py\u001b[0m in \u001b[0;36mget_loc\u001b[1;34m(self, key, method, tolerance)\u001b[0m\n\u001b[0;32m   3063\u001b[0m                 \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_engine\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_loc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   3064\u001b[0m             \u001b[1;32mexcept\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m-> 3065\u001b[1;33m                 \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_engine\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_loc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0m_maybe_cast_indexer\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   3066\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   3067\u001b[0m         \u001b[0mindexer\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mget_indexer\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mkey\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mmethod\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mmethod\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtolerance\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mtolerance\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32mpandas\\_libs\\index.pyx\u001b[0m in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[1;34m()\u001b[0m\n",
      "\u001b[1;32mpandas\\_libs\\index.pyx\u001b[0m in \u001b[0;36mpandas._libs.index.IndexEngine.get_loc\u001b[1;34m()\u001b[0m\n",
      "\u001b[1;32mpandas\\_libs\\hashtable_class_helper.pxi\u001b[0m in \u001b[0;36mpandas._libs.hashtable.Int64HashTable.get_item\u001b[1;34m()\u001b[0m\n",
      "\u001b[1;32mpandas\\_libs\\hashtable_class_helper.pxi\u001b[0m in \u001b[0;36mpandas._libs.hashtable.Int64HashTable.get_item\u001b[1;34m()\u001b[0m\n",
      "\u001b[1;31mKeyError\u001b[0m: 31999"
     ]
    }
   ],
   "source": [
    "X_train = merged_df.loc[:31999, 'text']\n",
    "y_train = merged_df.loc[:31999, 'label']\n",
    "print(y_train[y_train.isin(['conclusion'])].shape)\n",
    "print(y_train[y_train.isin(['method'])].shape)\n",
    "print(y_train[y_train.isin(['introduction'])].shape)\n",
    "print(y_train[y_train.isin(['result'])].shape)\n",
    "X_test = merged_df.loc[32000:, 'text']\n",
    "y_test = merged_df.loc[32000:, 'label']\n",
    "print(y_test[y_test.isin(['method'])].shape)\n",
    "\n",
    "# # and if using sentences...\n",
    "# X_train = sentence_df.loc[:122967, ['sentence', 'article']]\n",
    "# y_train = sentence_df.loc[:122967, 'label']\n",
    "# print(y_train[y_train.isin(['method'])].shape)\n",
    "\n",
    "# X_test = sentence_df.loc[:122968, ['sentence', 'article']]\n",
    "# y_test = sentence_df.loc[:122968, 'label']\n",
    "# print(y_test[y_test.isin(['method'])].shape)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "sklearn way"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [],
   "source": [
    "# sklearn way, that does things randomly\n",
    "X_train, X_test, y_train, y_test = train_test_split(\n",
    "    #merged_df[['text', 'article', 'label']], # use when I want doc id'\n",
    "    merged_df['text'],\n",
    "    merged_df['label'],\n",
    "    test_size=0.2,\n",
    "    stratify = merged_df['label'])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "strip all whitespace "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_train = [re.sub(r'\\s+', ' ', doc) for doc in X_train]\n",
    "X_test = [re.sub(r'\\s+', ' ', doc) for doc in X_test]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "tokenize with nltk"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "train_docs = [TaggedDocument(word_tokenize(row), [\"{}\".format(i)]) for i, row in enumerate(X_train)]\n",
    "test_docs = [TaggedDocument(word_tokenize(row), [\"{}\".format(i)]) for i, row in enumerate(X_test)]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "using doc2vec. If I want to use unique id for each document, use lines of code that include 'article' key.\n",
    "Otherwise, use lines that tag docs with label"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "31146 7787\n"
     ]
    }
   ],
   "source": [
    "# using doc pub# as doc id. why does model drop some with this method?\n",
    "test_docs = [TaggedDocument(gensim.utils.simple_preprocess(row['text']), [row['article']]) for i, row in X_test.iterrows()]\n",
    "train_docs = [TaggedDocument(gensim.utils.simple_preprocess(row['text']), [row['article']]) for i, row in X_train.iterrows()]\n",
    "# Here's an alternative way\n",
    "#tagged = X_train.apply(lambda r: TaggedDocument(words=gensim.utils.simple_preprocess(r['text']), tags=[r['article']]), axis=1)\n",
    "# use tag.values as input to Doc2Vec\n",
    "\n",
    "# unique doc id as doc tag\n",
    "# there is something wird going on with iterrows. model drops docs. However, below method drops no values\n",
    "# train_docs = [TaggedDocument(gensim.utils.simple_preprocess(row), [\"{}\".format(i)]) for i, row in enumerate(X_train)]\n",
    "# test_docs = [TaggedDocument(gensim.utils.simple_preprocess(row), [\"{}\".format(i)]) for i, row in enumerate(X_test)]\n",
    "\n",
    "# labels as doc tag\n",
    "# train_docs = [TaggedDocument(gensim.utils.simple_preprocess(row), [label]) for label, row in zip(y_train, X_train)]\n",
    "# test_docs = [TaggedDocument(gensim.utils.simple_preprocess(row), [label]) for label, row in zip(y_test, X_test)] \n",
    "\n",
    "print(len(train_docs), len(test_docs))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [],
   "source": [
    "import multiprocessing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "8"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "multiprocessing.cpu_count()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Build doc2vec model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "14051"
      ]
     },
     "execution_count": 74,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "model = Doc2Vec(train_docs, vector_size = 100, epochs = 20, window=10, workers = 5) #min_count=0)min_count=3, sample=1e-4 # window of 10 is used in paper\n",
    "\n",
    "# if I don't input train_docs to Doc2Vec, I can build vocab and train in separate steps\n",
    "#model = Doc2Vec(train_docs, epochs = 20, window=100)\n",
    "#model.build_vocab(train_docs)\n",
    "#model.train(train_docs, total_examples=model.corpus_count, epochs=model.epochs)\n",
    "\n",
    "model.docvecs.count # make sure model is same length as training data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "save model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [],
   "source": [
    "model.save(\"doc2vec_classid.model\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "or load new model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "#loaded_model = Doc2Vec.load(\"test_doc2vec_binary.model\")\n",
    "# on monday, 8-6, load model\n",
    "model = Doc2Vec.load(\"doc2vec_uniqueid_retrained.model\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\saveryme\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\ipykernel_launcher.py:1: DeprecationWarning: Call to deprecated `most_similar` (Method will be removed in 4.0.0, use self.wv.most_similar() instead).\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    },
    {
     "ename": "KeyError",
     "evalue": "\"word '1' not in vocabulary\"",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyError\u001b[0m                                  Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-12-edcc8e997cf3>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m()\u001b[0m\n\u001b[1;32m----> 1\u001b[1;33m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mmost_similar\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[1;34m\"1\"\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\utils.py\u001b[0m in \u001b[0;36mnew_func1\u001b[1;34m(*args, **kwargs)\u001b[0m\n\u001b[0;32m   1417\u001b[0m                     \u001b[0mstacklevel\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;36m2\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1418\u001b[0m                 )\n\u001b[1;32m-> 1419\u001b[1;33m                 \u001b[1;32mreturn\u001b[0m \u001b[0mfunc\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m*\u001b[0m\u001b[0margs\u001b[0m\u001b[1;33m,\u001b[0m \u001b[1;33m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1420\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1421\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0mnew_func1\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\models\\base_any2vec.py\u001b[0m in \u001b[0;36mmost_similar\u001b[1;34m(self, positive, negative, topn, restrict_vocab, indexer)\u001b[0m\n\u001b[0;32m   1239\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1240\u001b[0m         \"\"\"\n\u001b[1;32m-> 1241\u001b[1;33m         \u001b[1;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mwv\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mmost_similar\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mpositive\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mnegative\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtopn\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mrestrict_vocab\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mindexer\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m   1242\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m   1243\u001b[0m     \u001b[1;33m@\u001b[0m\u001b[0mdeprecated\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"Method will be removed in 4.0.0, use self.wv.wmdistance() instead\"\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36mmost_similar\u001b[1;34m(self, positive, negative, topn, restrict_vocab, indexer)\u001b[0m\n\u001b[0;32m    528\u001b[0m                 \u001b[0mmean\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mweight\u001b[0m \u001b[1;33m*\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    529\u001b[0m             \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 530\u001b[1;33m                 \u001b[0mmean\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mweight\u001b[0m \u001b[1;33m*\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mword_vec\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mword\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0muse_norm\u001b[0m\u001b[1;33m=\u001b[0m\u001b[1;32mTrue\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    531\u001b[0m                 \u001b[1;32mif\u001b[0m \u001b[0mword\u001b[0m \u001b[1;32min\u001b[0m \u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mvocab\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    532\u001b[0m                     \u001b[0mall_words\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0madd\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mvocab\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mword\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;32m~\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\models\\keyedvectors.py\u001b[0m in \u001b[0;36mword_vec\u001b[1;34m(self, word, use_norm)\u001b[0m\n\u001b[0;32m    449\u001b[0m             \u001b[1;32mreturn\u001b[0m \u001b[0mresult\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    450\u001b[0m         \u001b[1;32melse\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m--> 451\u001b[1;33m             \u001b[1;32mraise\u001b[0m \u001b[0mKeyError\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;34m\"word '%s' not in vocabulary\"\u001b[0m \u001b[1;33m%\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m    452\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m    453\u001b[0m     \u001b[1;32mdef\u001b[0m \u001b[0mget_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mself\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mword\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m:\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mKeyError\u001b[0m: \"word '1' not in vocabulary\""
     ]
    }
   ],
   "source": [
    "model.most_similar([\"\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "How good is the model?"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "do some assesment here https://github.com/RaRe-Technologies/gensim/blob/develop/docs/notebooks/doc2vec-lee.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\saveryme\\AppData\\Local\\Continuum\\anaconda3\\lib\\site-packages\\gensim\\matutils.py:737: FutureWarning: Conversion of the second argument of issubdtype from `int` to `np.signedinteger` is deprecated. In future, it will be treated as `np.int32 == np.dtype(int).type`.\n",
      "  if np.issubdtype(vec.dtype, np.int):\n"
     ]
    },
    {
     "ename": "ValueError",
     "evalue": "0 is not in list",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "\u001b[1;32m<ipython-input-71-3f3069532fd1>\u001b[0m in \u001b[0;36m<module>\u001b[1;34m()\u001b[0m\n\u001b[0;32m      4\u001b[0m     \u001b[0minferred_vector\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0minfer_vector\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mtrain_docs\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0mdoc_id\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mwords\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      5\u001b[0m     \u001b[0msims\u001b[0m \u001b[1;33m=\u001b[0m \u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mdocvecs\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mmost_similar\u001b[0m\u001b[1;33m(\u001b[0m\u001b[1;33m[\u001b[0m\u001b[0minferred_vector\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0mtopn\u001b[0m\u001b[1;33m=\u001b[0m\u001b[0mlen\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mmodel\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mdocvecs\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[1;32m----> 6\u001b[1;33m     \u001b[0mrank\u001b[0m \u001b[1;33m=\u001b[0m \u001b[1;33m[\u001b[0m\u001b[0mdocid\u001b[0m \u001b[1;32mfor\u001b[0m \u001b[0mdocid\u001b[0m\u001b[1;33m,\u001b[0m \u001b[0msim\u001b[0m \u001b[1;32min\u001b[0m \u001b[0msims\u001b[0m\u001b[1;33m]\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mindex\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mdoc_id\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0m\u001b[0;32m      7\u001b[0m     \u001b[0mranks\u001b[0m\u001b[1;33m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[1;33m(\u001b[0m\u001b[0mrank\u001b[0m\u001b[1;33m)\u001b[0m\u001b[1;33m\u001b[0m\u001b[0m\n\u001b[0;32m      8\u001b[0m \u001b[1;33m\u001b[0m\u001b[0m\n",
      "\u001b[1;31mValueError\u001b[0m: 0 is not in list"
     ]
    }
   ],
   "source": [
    "ranks = []\n",
    "second_ranks = []\n",
    "for doc_id in range(len(train_docs)):\n",
    "    inferred_vector = model.infer_vector(train_docs[doc_id].words)\n",
    "    sims = model.docvecs.most_similar([inferred_vector], topn=len(model.docvecs))\n",
    "    rank = [docid for docid, sim in sims].index(doc_id)\n",
    "    ranks.append(rank)\n",
    "    \n",
    "    second_ranks.append(sims[1])\n",
    "collections.Counter(ranks)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print('Document ({}): {}\\n'.format(doc_id, ' '.join(train_docs[doc_id].words)))\n",
    "print(u'SIMILAR/DISSIMILAR DOCS PER MODEL %s:\\n' % model)\n",
    "for label, index in [('MOST', 0), ('SECOND-MOST', 1), ('MEDIAN', len(sims)//2), ('LEAST', len(sims) - 1)]:\n",
    "    print(u'%s %s: «%s»\\n' % (label, sims[index], ' '.join(train_docs[sims[index][0]].words)))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If using unique doc ids, get vectors for sklearn from model, and use infer_vector to generate embeddings for testing"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "metadata": {},
   "outputs": [],
   "source": [
    "embeddings_train = [model[doc] for doc in range(len(model.docvecs))] # get d\n",
    "labels_train = y_train # set labels as y_train to keep variables consistent\n",
    "labels_test, embeddings_test = zip(*[(doc.tags[0], model.infer_vector(doc.words, epochs = 20)) for doc in test_docs])\n",
    "labels_test = y_test # the labels_test is no good if not using classes as labels"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If I have used the class labels, infer vectors for training and val data. \n",
    "Or, even if I have used unique ids, use the training data (seen data) to infer vectors anyway. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [],
   "source": [
    "# gives labels and vectors, though I already have the labels in y_train\n",
    "labels_train, embeddings_train = zip(*[(doc.tags[0], model.infer_vector(doc.words, steps=20)) for doc in train_docs])\n",
    "labels_test, embeddings_test = zip(*[(doc.tags[0], model.infer_vector(doc.words, steps=20)) for doc in test_docs])\n",
    "labels_train = y_train\n",
    "labels_test = y_test"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "and then run through sklearn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.svm import LinearSVC\n",
    "from sklearn.linear_model import LogisticRegression\n",
    "from sklearn.svm import SVC\n",
    "from sklearn.naive_bayes import BernoulliNB"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "rskf = RepeatedStratifiedKFold(n_splits = 5, n_repeats = 5, random_state = 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 156,
   "metadata": {},
   "outputs": [],
   "source": [
    "#sk_model = LinearSVC()\n",
    "#sk_model = SVC(kernel='linear')\n",
    "#sk_model = BernoulliNB()\n",
    "sk_model = LogisticRegression(random_state = 0)\n",
    "sk_model.fit(embeddings_train, labels_train)\n",
    "\n",
    "y_predictions = sk_model.predict(embeddings_test)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "or try out the one vs rest classifier:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.multiclass import OneVsRestClassifier"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_ovr = OneVsRestClassifier(LinearSVC(random_state=0)).fit(embeddings_train, labels_train)\n",
    "y_predictions = model_ovr.predict(embeddings_test)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "then score it up!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "f1 0.5984023238925199\n",
      "recall 0.6163051608077786\n",
      "accuracy 0.8404730996682532\n",
      "precision 0.5815102328863797\n",
      "[[ 824  513]\n",
      " [ 593 5003]]\n"
     ]
    }
   ],
   "source": [
    "# print(\"f1\", f1_score(labels_test, y_predictions, average = 'macro'))\n",
    "# print(\"recall\", recall_score(labels_test, y_predictions, average = 'macro'))\n",
    "# print(\"accuracy\", accuracy_score(labels_test, y_predictions))\n",
    "# print(\"precision\", precision_score(labels_test, y_predictions, average = 'macro'))\n",
    "# print(\"confusion_matrix\\n\", confusion_matrix(labels_test, y_predictions))\n",
    "\n",
    "# binary label\n",
    "print(\"f1\", f1_score(labels_test, y_predictions, pos_label = 'method', average = \"binary\"))\n",
    "print(\"recall\", recall_score(labels_test, y_predictions, pos_label = 'method', average = \"binary\"))\n",
    "print(\"accuracy\", accuracy_score(labels_test, y_predictions))\n",
    "print(\"precision\", precision_score(labels_test, y_predictions, pos_label = 'method', average = \"binary\"))\n",
    "print(confusion_matrix(labels_test, y_predictions, labels=['method', 'other']))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "and here a cool thing: a classification report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.metrics import classification_report"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 161,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "             precision    recall  f1-score   support\n",
      "\n",
      "     method       0.58      0.62      0.60      1337\n",
      "      other       0.91      0.89      0.90      5596\n",
      "\n",
      "avg / total       0.84      0.84      0.84      6933\n",
      "\n"
     ]
    }
   ],
   "source": [
    "report = classification_report(labels_test, y_predictions)#, target_names = ['method', 'other'])\n",
    "print(report)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "check out the misclassified sections"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "RangeIndex(start=32000, stop=38933, step=1)"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "X_test.index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(array([   6,    7,   10,   11,   12,   13,   14,   16,   18,   21,   24,\n",
      "         26,   28,   30,   31,   32,   33,   34,   35,   36,   40,   41,\n",
      "         42,   45,   49,   50,   53,   55,   56,   58,   60,   63,   65,\n",
      "         68,   69,   70,   73,   74,   75,   76,   78,   81,   83,   84,\n",
      "         87,   88,   89,   90,   91,   92,   94,   96,   99,  101,  102,\n",
      "        103,  106,  107,  108,  111,  113,  115,  116,  117,  118,  120,\n",
      "        123,  126,  129,  130,  132,  133,  135,  138,  139,  142,  143,\n",
      "        144,  145,  147,  151,  155,  156,  160,  162,  163,  165,  166,\n",
      "        167,  169,  172,  178,  230,  238,  310,  329,  334,  335,  340,\n",
      "        352,  357,  381,  396,  398,  409,  444,  448,  452,  459,  533,\n",
      "        549,  652,  739,  763,  767,  768,  773,  774,  775,  778,  779,\n",
      "        781,  810,  815,  825,  828,  829,  830,  831,  833,  834,  836,\n",
      "        837,  838,  839,  841,  843,  846,  847,  848,  853,  857,  858,\n",
      "        861,  863,  865,  867,  868,  872,  873,  875,  876,  881,  882,\n",
      "        883,  884,  885,  886,  887,  888,  889,  890,  892,  893,  894,\n",
      "        901,  902,  906,  908,  909,  910,  912,  914,  928,  949,  952,\n",
      "        953,  956,  970,  986, 1000, 1010, 1015, 1069, 1073, 1075, 1080,\n",
      "       1081, 1082, 1084, 1085, 1086, 1088, 1089, 1090, 1091, 1097, 1098,\n",
      "       1099, 1100, 1101, 1102, 1103, 1104, 1105, 1106, 1108, 1111, 1112,\n",
      "       1116, 1117, 1121, 1124, 1126, 1127, 1128, 1133, 1138, 1141, 1142,\n",
      "       1143, 1144, 1145, 1147, 1149, 1150, 1151, 1153, 1154, 1156, 1159,\n",
      "       1161, 1164, 1165, 1168, 1169, 1174, 1175, 1177, 1179, 1180, 1181,\n",
      "       1182, 1184, 1185, 1189, 1191, 1195, 1203, 1204, 1213, 1217, 1218,\n",
      "       1220, 1221, 1222, 1223, 1224, 1226, 1229, 1233, 1234, 1237, 1243,\n",
      "       1244, 1249, 1250, 1258, 1261, 1262, 1271, 1284, 1286, 1287, 1289,\n",
      "       1290, 1291, 1293, 1294, 1295, 1299, 1302, 1304, 1308, 1311, 1316,\n",
      "       1322, 1323, 1324, 1325, 1326, 1327, 1334, 1336, 1337, 1339, 1340,\n",
      "       1342, 1345, 1347, 1350, 1351, 1353, 1354, 1355, 1357, 1358, 1359,\n",
      "       1362, 1364, 1365, 1366, 1375, 1378, 1383, 1386, 1393, 1395, 1397,\n",
      "       1402, 1404, 1429, 1508, 1516, 1523, 1524, 1531, 1539, 1541, 1547,\n",
      "       1564, 1572, 1583, 1584, 1587, 1588, 1589, 1590, 1593, 1596, 1597,\n",
      "       1600, 1601, 1608, 1609, 1614, 1615, 1616, 1618, 1619, 1621, 1622,\n",
      "       1623, 1624, 1625, 1627, 1628, 1629, 1632, 1635, 1640, 1641, 1645,\n",
      "       1651, 1652, 1664, 1666, 1667, 1671, 1672, 1674, 1676, 1679, 1680,\n",
      "       1682, 1684, 1686, 1687, 1693, 1734, 1735, 1736, 1737, 1738, 1739,\n",
      "       1741, 1745, 1746, 1747, 1748, 1749, 1752, 1753, 1757, 1758, 1759,\n",
      "       1761, 1763, 1764, 1771, 1790, 1792, 1793, 1794, 1795, 1800, 1801,\n",
      "       1802, 1803, 1811, 1813, 1820, 1824, 1827, 1831, 1837, 1851, 1862,\n",
      "       1865, 1872, 1888, 1889, 1891, 1892, 1893, 1894, 1895, 1896, 1898,\n",
      "       1911, 1916, 1922, 1930, 1937, 1944, 1951, 1953, 1958, 1961, 1977,\n",
      "       2003, 2005, 2008, 2032, 2045, 2071, 2084, 2097, 2123, 2132, 2133,\n",
      "       2135, 2136, 2137, 2171, 2175, 2198, 2199, 2205, 2207, 2216, 2221,\n",
      "       2248, 2257, 2276, 2286, 2291, 2335, 2336, 2337, 2338, 2348, 2363,\n",
      "       2406, 2415, 2426, 2427, 2430, 2432, 2443, 2451, 2484, 2512, 2519,\n",
      "       2520, 2522, 2529, 2552, 2561, 2563, 2572, 2606, 2608, 2631, 2656,\n",
      "       2680, 2686, 2699, 2707, 2711, 2714, 2715, 2731, 2738, 2745, 2768,\n",
      "       2782, 2789, 2795, 2810, 2815, 2827, 2842, 2878, 2883, 2905, 2927,\n",
      "       2930, 2932, 2937, 2946, 2949, 2958, 2960, 2962, 2995, 3006, 3014,\n",
      "       3022, 3029, 3065, 3066, 3072, 3089, 3120, 3135, 3136, 3141, 3144,\n",
      "       3151, 3154, 3177, 3190, 3200, 3202, 3234, 3235, 3238, 3246, 3253,\n",
      "       3260, 3261, 3285, 3311, 3316, 3320, 3322, 3330, 3350, 3359, 3383,\n",
      "       3402, 3437, 3442, 3445, 3446, 3462, 3463, 3466, 3489, 3502, 3509,\n",
      "       3511, 3516, 3530, 3534, 3546, 3556, 3562, 3583, 3641, 3642, 3643,\n",
      "       3666, 3704, 3709, 3717, 3761, 3786, 3789, 3794, 3802, 3809, 3816,\n",
      "       3819, 3829, 3831, 3843, 3848, 3856, 3872, 3875, 3957, 3959, 3967,\n",
      "       3968, 3969, 3971, 3976, 4006, 4051, 4061, 4067, 4068, 4071, 4080,\n",
      "       4094, 4117, 4130, 4134, 4139, 4142, 4164, 4165, 4183, 4186, 4196,\n",
      "       4203, 4212, 4218, 4220, 4227, 4229, 4230, 4233, 4237, 4252, 4264,\n",
      "       4266, 4267, 4272, 4283, 4316, 4318, 4323, 4330, 4337, 4350, 4371,\n",
      "       4377, 4378, 4391, 4400, 4409, 4416, 4425, 4434, 4470, 4474, 4480,\n",
      "       4517, 4563, 4567, 4577, 4587, 4589, 4591, 4611, 4613, 4619, 4624,\n",
      "       4625, 4626, 4638, 4693, 4699, 4702, 4710, 4716, 4736, 4742, 4744,\n",
      "       4771, 4789, 4802, 4804, 4810, 4837, 4839, 4841, 4851, 4867, 4870,\n",
      "       4871, 4876, 4878, 4882, 4886, 4892, 4901, 4907, 4924, 4927, 4936,\n",
      "       4940, 4951, 4984, 4991, 4993, 5000, 5001, 5018, 5048, 5079, 5121,\n",
      "       5132, 5136, 5140, 5148, 5153, 5157, 5161, 5196, 5202, 5212, 5224,\n",
      "       5227, 5228, 5256, 5261, 5262, 5264, 5265, 5267, 5293, 5319, 5323,\n",
      "       5325, 5326, 5327, 5330, 5342, 5345, 5371, 5383, 5402, 5403, 5417,\n",
      "       5460, 5471, 5491, 5493, 5494, 5495, 5502, 5517, 5518, 5524, 5527,\n",
      "       5534, 5570, 5584, 5601, 5620, 5621, 5627, 5673, 5674, 5675, 5689,\n",
      "       5712, 5734, 5743, 5747, 5748, 5749, 5757, 5758, 5762, 5763, 5767,\n",
      "       5775, 5783, 5784, 5834, 5837, 5874, 5886, 5890, 5916, 5919, 5920,\n",
      "       5927, 5932, 5946, 5947, 5957, 5982, 5999, 6007, 6010, 6011, 6015,\n",
      "       6016, 6045, 6047, 6104, 6109, 6121, 6128, 6132, 6145, 6187, 6195,\n",
      "       6245, 6248, 6273, 6279, 6286, 6292, 6295, 6296, 6319, 6354, 6359,\n",
      "       6375, 6382, 6385, 6413, 6420, 6422, 6430, 6434, 6442, 6475, 6497,\n",
      "       6546, 6549, 6570, 6571, 6613, 6627, 6629, 6687, 6704, 6711, 6714,\n",
      "       6751, 6768, 6776, 6781, 6786, 6791, 6794, 6796, 6812, 6813, 6814,\n",
      "       6815, 6852, 6855, 6895, 6906], dtype=int64),)\n"
     ]
    }
   ],
   "source": [
    "np_y_test = np.asarray(labels_test)\n",
    "misclassified = np.where(np_y_test != y_predictions) # returns indices where condition is true\n",
    "print(misclassified)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2714 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"results\"><title>RESULTS</title><p>The results from the measurements aboard the vessels and from the headspace of the linseed oil are summarized in <xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>. Particularly notable are the high CO and CO<sub>2</sub> concentrations and the low oxygen levels. No indication of the presence of H<sub>2</sub>S was seen. When the oxygen level from each vessel was plotted as function of the CO level, a high degree of correlation was obtained (<italic>r</italic><sup>2</sup>) = 0.92, see <xref ref-type=\"fig\" rid=\"fig2\">Fig. 2</xref>. Good correlation was also found between CO and butane equivalents (<italic>r</italic><sup>2</sup> = 0.93) and CO<sub>2</sub> and propylene (<italic>r</italic><sup>2</sup> = 0.98).</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1.</label><caption><p>Summary of all measurements from each vessel and headspace of linseed oil</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\"><hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #1 Hold<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #2 Hold<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #3 Hold<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #3 Stair<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #4 Stair<xref ref-type=\"table-fn\" rid=\"tblfn1\">a</xref><hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #5 Hold 2<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #5 Stair 2<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #5 Hold 4<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel # 5 Stair 4<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #5 Hold 6<hr/></td><td rowspan=\"1\" colspan=\"1\">Vessel #5 Stair 6<hr/></td><td rowspan=\"1\" colspan=\"1\">Linseed Oil Headspace<hr/></td></tr><tr><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\">November 2006</td><td rowspan=\"1\" colspan=\"1\">January 2007</td><td rowspan=\"1\" colspan=\"1\">February 2007</td><td rowspan=\"1\" colspan=\"1\">February 2007</td><td rowspan=\"1\" colspan=\"1\">March 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\">October 2007</td><td rowspan=\"1\" colspan=\"1\"/></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">CO</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5850</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6980</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14 650</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10 960</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11 510</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11 950</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7710</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7240</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1460</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8860</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5300</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1090</td></tr><tr><td rowspan=\"1\" colspan=\"1\">CO<sub>2</sub></td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9340</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3240</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7070</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5450</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5160</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21 570</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12 360</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17 430</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2960</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18 820</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8690</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">570</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Methane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">614</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">216</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">632</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">468</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">204</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">956</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">589</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">388</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">79.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">454</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">246</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">40</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ethylene</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Propylene</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Methanol</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Propanal (aldehydes)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">120</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Ethane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">380</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Butane equivalents<xref ref-type=\"table-fn\" rid=\"tblfn2\">b</xref></td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">237</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">275</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">842</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">626</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">591</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">512</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">321</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">265</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">63</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">393</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">219</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">83</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Oxygen %</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6%</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5%<xref ref-type=\"table-fn\" rid=\"tblfn3\">c</xref></td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.8%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.4%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.6%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16.9%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.7%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10.9%</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.0%</td></tr></tbody></table><table-wrap-foot><fn><p>All components except oxygen were measured with FTIR spectroscopy. All values in ppm except oxygen which is in %.</p></fn><fn id=\"tblfn1\"><label>a</label><p>The results from Vessel #4 are means from measurements at four different levels of the stairway.</p></fn><fn id=\"tblfn2\"><label>b</label><p>Sum concentration of alkanes (C<italic><sub>n</sub></italic>H<sub>2<italic>n</italic>+2</sub>) with three or more carbons expressed as ppmv butane equivalents.</p></fn><fn id=\"tblfn3\"><label>c</label><p>Parallel reading from head space in cargo hold.</p></fn></table-wrap-foot></table-wrap><fig id=\"fig2\" position=\"float\"><label>Fig. 2.</label><caption><p>The correlation between oxygen and CO concentrations in hatches and stairways. The point representing 20.9% oxygen and 0% CO is added manually.</p></caption><graphic xlink:href=\"annhygmen013f02_lw\"/></fig><p>The aldehyde concentrations aboard the vessels were much lower than in previous findings in pellet warehouses. The qualitative analysis of headspace air above linseed oil revealed the presence of all the gas species found in the cargo holds and the reduction of the oxygen level was similar. The relative concentrations of aldehydes and ethane were higher.</p><p>The concentrations in the stairways were considerably lower than the levels found in the adjacent cargo holds, (Vessel #3 and #5). The stairway on Vessel #4 was sampled on four different levels, showing &lt;1% stratification throughout the air column of the stairway.</p><p>The results from the measurements aboard the vessels of oxygen at the bottom of the stairway and in the headspace of the cargo hold on Vessel #4 are described in <xref ref-type=\"fig\" rid=\"fig3\">Fig. 3</xref>. A steady state with ∼10% oxygen level was reached after ∼3 weeks at sea. The CO level in the headspace of the hold reached 900 ppm after 7 days. The onboard CO instrumentation was found to be unreliable over 1000 ppm and further evaluation of this parameter was not possible until the FTIR analysis was available at the time the cargo was discharged in the Port of Helsingborg.</p><fig id=\"fig3\" position=\"float\"><label>Fig. 3.</label><caption><p>The oxygen level in cargo hold headspace and bottom of stairway measured during ocean voyage (Vessel #4).</p></caption><graphic xlink:href=\"annhygmen013f03_lw\"/></fig><p>While Vessel #4 was discharging, the decay of CO at the bottom level of the stairway was monitored at 2.5-min intervals over a period of 2 h. The sample air was pumped to the mobile FTIR unit located alongside the vessel. Initially, the stairway deck-access door was closed and almost no decay could be seen. The door was then opened to simulate the usual self-ventilation procedure. Over the next 45 min, an exponential decay (<italic>r</italic><sup>2</sup> = 1.00) was recorded corresponding to a ventilation rate of 0.23 air exchanges per hour. In order for the CO concentration at the bottom level of the stairway to reach an acceptable 8-h occupational exposure limit of 35 ppm according to Swedish standards, 26 h of ventilation would have been required.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2715 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"results\"><title>RESULTS</title><p>In <xref ref-type=\"table\" rid=\"tbl3\">Table 3</xref>, geometric means (GMs) and geometric standard deviations (GSDs) for dust levels for all workers and stratified by work task are given together with results from persons only doing one job during the day. GM and GSD for inhalable wood dust concentrations were 0.95 mg/m<sup>3</sup> (2.05) for study 1 and 0.60 mg/m<sup>3</sup> (1.63) for study 2.</p><table-wrap id=\"tbl3\" position=\"float\"><label>Table 3.</label><caption><p>Wood dust exposure as GM and GSD of inhalable dust concentration (mg/m<sup>3</sup>) for all workers and during specific work tasks in study 1 (1997/98) and study 2 (2003/2004)<xref ref-type=\"table-fn\" rid=\"tblfn1\">a</xref></p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\">Group, (<italic>n/N</italic>)</td><td rowspan=\"1\" colspan=\"1\">Study 1, GM (GSD), <italic>n/N</italic></td><td rowspan=\"1\" colspan=\"1\">Study 2, GM (GSD), <italic>n/N</italic></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">All workers,<xref ref-type=\"table-fn\" rid=\"tblfn2\">b</xref> 2746/3713</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.95 (2.05), 1702/2358</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.60 (1.63), 1044/1355</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Sanding, 150/208</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">1.55 (1.87), 111/163</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.68 (1.67), 39/45</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cutting, 773/1043</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">1.13 (1.95), 504/694</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.67 (1.55), 269/349</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Cutting and sanding, 264/362</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">1.38 (1.76), 139/178</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.78 (1.66), 125/184</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Handling and assembling, 830/1144</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.71 (1.98), 539/750</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.55 (1.42), 291/394</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Truck driver, 116/166</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.78 (2.01), 49/70</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.45 (1.50), 67/96</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Foreman, 123/151</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.56 (2.04), 82/107</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.38 (1.62), 41/44</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Store man, 59/87</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.79 (1.79), 40/59</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.36 (1.66), 19/28</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Mixed task, 251/393</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">1.10 (1.78), 151/215</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.63 (1.57), 100/120</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Other tasks, 133/176</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.92 (2.10), 72/100</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.58 (2.02), 61/76</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Unknown task, 19/41</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">1.20 (2.37), 15/22</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.51 (1.69), 4/19</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Workers only doing one job during the day, 1907/2627</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.94 (2.10), 1239/1721</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.58 (1.65), 668/906</td></tr></tbody></table><table-wrap-foot><fn id=\"tblfn1\"><label>a</label><p>Every persons' measurements were averaged, the distribution was log-transformed and the log-transformed mean and SDs were exponentiated in order to get GM and GSM. <italic>N</italic> = number of measurements, <italic>n</italic> = number of persons.</p></fn><fn id=\"tblfn2\"><label>b</label><p>The total number of workers exceeds the sum of workers in each work task as workers with different tasks in three repeated measurements (<italic>n</italic> = 28) were excluded.</p></fn></table-wrap-foot></table-wrap><p>A trend towards lower dust levels in study 2 was seen for all work tasks. The highest exposure was found for sanding (study 1) and sanding and cutting (study 2). The lowest exposure was found for foremen (study 1) and store men (study 2). Overall, the yearly average percent decline was ∼7%, most pronounced for sanding.</p><p>Only small differences in overall GM and GSD were seen between workers doing more jobs during the day compared to those doing only one job.</p><p><xref ref-type=\"table\" rid=\"tbl4\">Table 4</xref> shows the frequency of potential determinants of exposures in studies 1 and 2. There were only minor differences between the observed and the adjusted differences in frequencies of determinants between the two studies.</p><table-wrap id=\"tbl4\" position=\"float\"><label>Table 4.</label><caption><p>Frequency of potential determinants of exposure among persons participating in study 1 and study 2</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\">Possible determinants of exposure</td><td rowspan=\"1\" colspan=\"1\">Study 1, % (number of persons)<xref ref-type=\"table-fn\" rid=\"tblfn3\">a</xref></td><td rowspan=\"1\" colspan=\"1\">Study 2, %, (number of persons)<xref ref-type=\"table-fn\" rid=\"tblfn3\">a</xref></td><td rowspan=\"1\" colspan=\"1\">Adjusted absolute % difference (study 2–study 1), (95% CI)<xref ref-type=\"table-fn\" rid=\"tblfn4\">b</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Personal and machine level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Use of compressed air</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">45 (725)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">42 (432)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−2 (−9; 4)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fully automatic</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">10 (160)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">8 (87)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−2 (−5; 5)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Semi-automatic</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">36 (590)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">38 (394)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">5 (12; −2)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Manual</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">40 (657)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">28 (288)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−12 (−23; −3)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Exhaust ventilation, adequate</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">48 (206)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">33 (101)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−12 (−24; 0)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Enclosure</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">56 (435)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">45 (226)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−7 (−18; 5)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Wood dust on the work piece</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">9 (100)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">6 (44)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−3 (−8; 1)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Department level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Supplementary fresh air intake</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">37 (437)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">49 (453)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">16 (3; 35)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Vacuum cleaning of rooms</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">81 (999)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">90 (828)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">9 (−3; 22)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning of rooms with compressed air</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">8 (98)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">2 (17)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−6 (−15; 3)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Vacuum cleaning of work pieces</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">21 (184)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">6 (51)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−15 (−29; 0)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning of work pieces with compressed air</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">55 (486)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">17 (161)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−39 (−57; −21)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Factory level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Pine factory</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">66 (1121)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">49 (516)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−11 (−29; 7)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Chair factory</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">5 (83)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">3 (34)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−6 (−12; 1)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Kitchen/shop factory</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">11 (189)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">21 (218)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">9 (−5; 22)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Particle board/medium density fibreboard furniture</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">18 (298)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">24 (250)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">7 (−12; 26)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Election of a safety representative within the last 2 years</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">84 (1352)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">99 (656)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">17 (1; 32)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Re-circulation of air to working rooms</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">56 (954)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">26 (249)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−27 (−49; −5)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Re-circulation of air to enclosed machines</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">67 (1131)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">86 (888)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">26 (8; 43)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Plan for cleaning of rooms</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">52 (877)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">58 (556)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">10 (−14; 35)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Plan for regularly check of central exhaust ventilation system</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">67 (1137)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">62 (642)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0 (−17; 18)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Special cleaning staff</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">58 (981)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">70 (690)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">12 (−10; 35)</td></tr></tbody></table><table-wrap-foot><fn id=\"tblfn3\"><label>a</label><p>The number of valid cases is highly variable. For example, enclosure is of relevance for some work tasks only.</p></fn><fn id=\"tblfn4\"><label>b</label><p>In order to estimate the true proportion of employees in the furniture industry in Viborg County exposed to potential determinants of exposures, we used inverse probability weighting for each of 13 strata (based on factory size and type of factory) to adjust for the different subsets of factories in the two studies,. The strata-specific probabilities were equal to the proportion of sampled factories in the strata. For each variable, a confidence interval (CI) on the absolute difference between the proportions in the two studies was obtained using bootstrap sampling (<italic>N</italic> = 200) of the factories in each strata and study.</p></fn></table-wrap-foot></table-wrap><p>From study 1 to study 2, there was a change towards (i) less manual work. (ii) Less pine furniture, and more furniture and cabinets made of particle board or medium density fibreboard. (iii) A ‘decrease’ in adequate exhaust ventilation systems, enclosure, cleaning of rooms with compressed air, cleaning of work pieces with vacuum cleaning and with compressed air and re-circulation of air to working rooms. (iv) An ‘increase’ in supplementary fresh air intake, vacuum cleaning of rooms, re-circulation of air to enclosed machines, special cleaning staff and election of a safety representative within the last 2 years.</p><sec><title>Multivariate model</title><p>Using the strategy of model making, we ended up with a model including four random terms, study as fixed effect, fixed effects from all the four levels and two interaction terms, namely manual × assembling/packing and sanding × adequate exhaust ventilation. Insignificant determinants of exposure kept in the model were handling/assembling and sanding (part of an interaction term), and day shift, use of hardwood and encapsulation (significant after one of each of the four modelling phases). A plot of predicted values versus residuals shows no trends apart from two straight lines caused by the 6% of observations below the limit of detection.</p><p>In <xref ref-type=\"table\" rid=\"tbl5\">Table 5</xref>, coefficients, standard error, <italic>P</italic>-value and percent change in dust level for determinants of exposure are given for fixed effect included in the final model. In addition, a model only including the 27 factories which participated in both study 1 and 2 is provided.</p><table-wrap id=\"tbl5\" position=\"float\"><label>Table 5.</label><caption><p>Variables describing potential determinants of exposure. Coefficients, standard error (SE) and <italic>P</italic>-value for fixed effect included in a mixed effect model for all factories, and for factories included in both studies 1 and 2</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\">Determinants of exposure</td><td colspan=\"4\" rowspan=\"1\">All 68 factories</td><td colspan=\"3\" rowspan=\"1\">27 Factories included in both studies 1 and 2</td></tr><tr><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\">Coefficient</td><td rowspan=\"1\" colspan=\"1\">SE</td><td rowspan=\"1\" colspan=\"1\"><italic>P</italic>-value</td><td rowspan=\"1\" colspan=\"1\">% Change (95% CI)<xref ref-type=\"table-fn\" rid=\"tblfn5\">a</xref></td><td rowspan=\"1\" colspan=\"1\">Coefficient</td><td rowspan=\"1\" colspan=\"1\">SE</td><td rowspan=\"1\" colspan=\"1\"><italic>P</italic>-value</td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Intercept</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.03</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.08</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">—</td><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.003</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.10</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">—</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Study (ref. study 1)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.54</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.0001</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−41 (−36; −46)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.52</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">—</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Personal level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Use of compressed air</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.03</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.0001</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">14 (8; 20)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.01</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Day shift</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.03</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">NS</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Machine level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Handling/assembling</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.03</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.60</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">NS</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.02</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.07</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Manual task</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">17 (8; 30)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.01</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sanding</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">53 (36; 86)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.44</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fully automatic task</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.01</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">16 (5; 27)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.01</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Adequate exhaust ventilation</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−10 (−0; −18)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.09</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Encapsulation</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.03</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.48</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">NS</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.07</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.20</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Use of hardwood</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.09</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.08</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.27</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">NS</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.08</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.09</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.38</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Department level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning of work pieces with compressed air</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.01</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">15 (3; 28)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.07</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.09</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Vacuum cleaning machines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.02</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−12 (−2; −21)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.07</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Kitchen production</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.09</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.04</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">19 (0; 41)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Factory level</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Factories &lt;20 employees</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.02</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">27 (4; 55)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Special cleaning staff</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.05</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.01</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−12 (−3; −19)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.01</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Handling/assembling × manual</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.07</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−25 (−13; −35)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sanding × adequate exhaust ventilation</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.01</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">−29 (−7; −46)</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">−0.35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.06</td></tr></tbody></table><table-wrap-foot><fn><p>The dependent variable was the log<sub>e</sub>-transformed dust concentration.</p></fn><fn id=\"tblfn5\"><label>a</label><p>The percent change in dust level caused by each determinant <italic>b</italic> was calculated as (1 – exp(<italic>b</italic><sub>coefficient</sub>)) × 100. NS: non-significant.</p></fn></table-wrap-foot></table-wrap><p>Determinants found in the model based on the 27 factories which participated in both the studies were similar to determinants found in the model that included all 68 factories.</p><p>No significant interactions between determinants of exposure and study were found.</p><p>The following significant determinants ‘increased’ the dust concentration: use of compressed air (14%), fully automatic tasks (16%), sanding (53%), manual tasks (17%), cleaning of work pieces with compressed air (15%), kitchen production (19%) and small factories (27%).</p><p>Significant determinants for ‘decrease’ in dust concentration were study 2 (41%), adequate exhaust ventilation (10%), vacuum cleaning of machines (12%), special cleaning staff (12%), handling/assembling × manual (25%) and sanding × adequate exhaust ventilation (29%).</p><p>The final model was repeated on the subgroup of 183 workers (540 measurements) surveyed in both studies 1 and 2. The intercept and coefficients were largely in the same range as the model based on all participants from the 68 factories (data not shown).</p><p>In order to explore the impact of the different exposure documentation in the two studies, we repeated the analyses and restricted the population to subjects who had the same job during the three repeated measuring rounds, and it did not significantly changed the coefficients for determinants of exposure given in <xref ref-type=\"table\" rid=\"tbl5\">Table 5</xref>.</p><p>In order to explore possible differences in mean dust level between factories participating in only one of the studies compared to factories participating in both studies, we included participation status in the mixed model described above, and repeated the analyses for study I and study 2 separately. We found no significant differences between factories participating one time versus factories participating two times, that is, they shared the same mean.</p></sec><sec><title>Trends in dust level</title><p>GM (GSD) for inhalable wood dust concentrations were 0.95 mg/m<sup>3</sup> (2.05) for study 1 and 0.60 mg/m<sup>3</sup> (1.63) for study 2, representing a 7.2% annual decrease in dust concentration. The decrease was even more pronounced in the multivariate model (coefficient for study = −0.54, equalling an 8.6% annual decrease). We repeated the analysis without any fixed effect but study, and this did not change the result (coefficient for study = −0.55, equalling an 8.8% annual decrease). The same multivariate model with all fixed effects was performed for the 27 factories which participated in both studies, and it did not change the results markedly (coefficient for study = −0.52, equalling an 8.3% annual decrease).</p></sec><sec><title>Variance components</title><p><xref ref-type=\"table\" rid=\"tbl6\">Table 6</xref> shows estimated variance components of random terms for the total study. The residual variance component is termed the within-worker variance components, but may include (smaller) contributions from, for example, measurement error.</p><table-wrap id=\"tbl6\" position=\"float\"><label>Table 6.</label><caption><p>Estimates of variance components for the total model (number of factories = 68)</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\">Variance components in model excluding fixed effects (%)</td><td rowspan=\"1\" colspan=\"1\">Variance components in model including fixed effects (%)</td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Total between-worker variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.31 (58)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.18 (34)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Between-factories variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.05 (9)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.03 (5)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Between-department variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.07 (14)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.02 (4)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Between-machines variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.07 (13)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.04 (7)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Between-workers variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.12 (22)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.09 (18)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Within-workers variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.22 (42)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.20 (38)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Sum of variance explained by random effects</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.53 (100)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.38 (72)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Total variance</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.53 (100)</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.53 (100)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Variance explained by fixed effects (%)</td><td align=\"center\" rowspan=\"1\" colspan=\"1\">—</td><td align=\"char\" char=\"(\" rowspan=\"1\" colspan=\"1\">0.15 (28)</td></tr></tbody></table></table-wrap><p>The proportion of variance explained by the fixed terms was 28%. The proportion of the total between-worker variance explained by the fixed terms was 42%, calculated as: ((sum of variance explained by random effects – within-worker variance<sub>excl fixed effects</sub>) − (sum of variance explained by random effects – within-worker variance<sub>incl fixed effects</sub>))/(sum of variance explained by random effects – within-worker variance<sub>excl fixed effects</sub>) × 100%.</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2731 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"results\"><title>RESULTS</title><sec><title>Follow-up of Patients Presenting with TIA</title><p>\n",
      "82 of 98 patients (83.7%) were diagnosed with stroke\n",
      "while the rest were diagnosed with TIA. Of patients\n",
      "presenting with TIA, only 38% were followed up by\n",
      "Stroke Specialists after discharge from the hospital upon resolution of their symptoms. The others were\n",
      "followed up by other medical specialists.</p><p>\n",
      "The rest of the audit results are on patients presenting\n",
      "with stroke.</p></sec><sec><title>Acute Management of Stroke</title><p>\n",
      "Of the 82 patients with stroke, 75 (91.5%) were\n",
      "diagnosed to have ischaemic stroke following\n",
      "exclusion of haemorrhage on a CT-head scan (Table\n",
      "<xref ref-type=\"table\" rid=\"T1\">1</xref>). Cases of haemorrhagic stroke accounted for only\n",
      "8.3% of cases with the majority of those being due to\n",
      "anterior fossa haemorrhage (Table <xref ref-type=\"table\" rid=\"T1\">1</xref>). Most of the\n",
      "patients with stroke had partial anterior circulations strokes which accounted for 53.7% of the total\n",
      "number of cases (Table <xref ref-type=\"table\" rid=\"T1\">1</xref>). Conformity with RCP\n",
      "guidelines on some relevant aspects of investigation\n",
      "and acute care are summarized in Table <xref ref-type=\"table\" rid=\"T2\">2</xref>. With regards\n",
      "to admissions and the timing of CT-head scans, the\n",
      "entire population of patients presenting with stroke (n\n",
      "= 82) was the reference population. With regards to\n",
      "commencement of anti-platelets, the total number of\n",
      "patients in which haemorrhage was excluded on a CT-head\n",
      "scan was the reference population (n = 75). With\n",
      "regards to the management of haemorrhagic stroke,\n",
      "the number of patients diagnosed with haemorrhagic\n",
      "stroke on a CT-head scan was the reference population\n",
      "(n = 7). All the patients presenting with stroke were admitted and a neurosurgical opinion was sought for\n",
      "all the patients with haemorrhagic stroke diagnosed\n",
      "on a CT-head scan. In contrast, less than 70% of\n",
      "patients had a CT-head scan done within 24 hours or\n",
      "anti-platelet therapy commenced for ischaemic stroke\n",
      "within 24 hours (Table <xref ref-type=\"table\" rid=\"T2\">2</xref>). Notably, more than 70%\n",
      "of patients had anti-platelets commenced only after\n",
      "the exclusion of intra-cranial haemorrhage on head\n",
      "CT-scan (Table <xref ref-type=\"table\" rid=\"T2\">2</xref>).</p></sec><sec><title>Secondary Prevention of Stroke</title><p>\n",
      "The group of patients with ischaemic stroke (n = 75)\n",
      "were the target of medical measures at secondary\n",
      "prevention of stroke. In such patients, with regards to\n",
      "the continuation of anti-hypertensives on admission,\n",
      "the control of blood glucose levels in diabetic patients\n",
      "and treatment with simvastatin, RCP guidelines were\n",
      "adhered to in more than 70% of patients (Table <xref ref-type=\"table\" rid=\"T3\">3</xref>). In\n",
      "contrast, as regards the utilization of thiazides/\n",
      "indapamide and ACE-inhibitors in secondary\n",
      "prevention of ischaemic stroke, RCP guidelines were\n",
      "adhered to in less than 70% of patients (Table <xref ref-type=\"table\" rid=\"T3\">3</xref>).\n",
      "Similarly, warfarin prophylaxis against embolism in\n",
      "chronic atrial fibrillation and carotid doppler\n",
      "ultrasound scans to detect critical internal carotid artery\n",
      "stenosis in anterior circulation strokes were done in\n",
      "less than 70% of patients (Table <xref ref-type=\"table\" rid=\"T3\">3</xref>).</p></sec><sec><title>Multi-disciplinary Care of Patients with Stroke</title><p>\n",
      "RCP guidelines were adhered to in less than 70% of\n",
      "the patients in any of the areas of multi-disciplinary\n",
      "care assessed (Table <xref ref-type=\"table\" rid=\"T4\">4</xref>). With regards to dysphagia,\n",
      "physiotherapy assessments and weekly MDT meetings,\n",
      "all the patients presenting with stroke except for those\n",
      "that died within a week (n = 79) were the reference\n",
      "population (3 patients died within a week and they all\n",
      "died within 24 hours of admission). The reference\n",
      "population for a speech and language therapist’s\n",
      "assessment (n = 43) were all the patients with stroke\n",
      "and dysphasia. The patients with stroke that were alive\n",
      "on discharge (n = 68) were the reference group\n",
      "expected to have had an occupational therapist and\n",
      "psychological assessment. However, 2 patients died\n",
      "after they had been assessed by an occupational\n",
      "therapist but prior to discharge from hospital.</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2738 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\"><title>RESULTS</title><sec><title>Random Walk versus continuous solutions</title><p>To confirm that the solution from the continuous model coincided with the mean concentrations from the RW model, we did one run with the continuous model and 40 runs of the RW model, using the same parameters (<xref rid=\"fig7\" ref-type=\"fig\">Fig. 7</xref> for result). The black lines are the concentration in the cleft given by the continuous model of, respectively, Ca<sup>2+</sup> (<italic>solid line</italic>), mobile buffer (<italic>dashed line</italic>), and stationary buffer (<italic>dash</italic>-<italic>dotted line</italic>). The colored lines, partly covered by the black lines, are 1), the concentration results from a single RW simulation (<italic>green lines</italic>); and 2), the average results from 40 RW runs (<italic>red lines</italic>). Note that the scale for the stationary buffer traces is given in the right <italic>y</italic> axis. One LCC was opened at <italic>t</italic> = 0, to act as a Ca<sup>2+</sup> source in the cleft. After ∼1 ms, the steady state, in which most of the stationary buffers were bound to Ca<sup>2+</sup>, was achieved. After 2 ms, a second LCC was opened. This time, the steady state occurred more quickly, due to the fact that less stationary buffer was available. We see that the [Ca<sup>2+</sup>] in the single RW run fluctuates a great deal in the steady-state period, but the mean concentration does not. After 4 ms, both LCCs were turned off and the Ca<sup>2+</sup> left the cleft quickly. Some Ca<sup>2+</sup> remained, due to the unbinding of Ca<sup>2+</sup> from the stationary buffer.</p><fig position=\"float\" id=\"fig7\"><label>FIGURE 7</label><caption><p>Simulation results from the continuous model (<italic>black lines</italic>) and from the Random Walk (RW) model (<italic>colored lines</italic>). The results represent the average concentrations from the whole cleft. The same simulation setup was used for the two models, including buffers from <xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>. One LCC is open from the start. After 2 ms, one more opens. Then, after 4 ms, both close. The black lines are the results from one simulation of the continuous model. Each line, solid, dashed, and dash-dotted, represents the concentration of Ca<sup>2+</sup>, mobile buffer, and stationary buffer, respectively. The right <italic>y</italic> axis shows the scale for the stationary buffer. The colored lines are the mean concentrations from 40 runs of the RW model (<italic>red lines</italic>), and the concentrations from a single RW simulation (<italic>green lines</italic>).</p></caption><graphic xlink:href=\"BIO.103523.wc.f7\"/></fig><p>The result confirms what others have pointed out, that the continuous solution coincides with the mean result from several RW simulations (<xref ref-type=\"bibr\" rid=\"bib1\">1</xref>–<xref ref-type=\"bibr\" rid=\"bib3\">3</xref>). We did see a difference between the mean concentration of the stationary buffer registered from the RW runs, and the corresponding concentration from the continuous solution. This error was introduced in the RW model, because we did not account for the absorbing boundary when calculating the probabilities that Ca<sup>2+</sup> ions and the stationary buffer molecules near the rim would bind. By placing the RyRs well inside the cleft, the outermost being 30 nm from the rim, we avoided this error when binding probabilities for the receptors were calculated.</p><p>It is interesting to note that the large Ca<sup>2+</sup> flux to the buffers, primarily to the stationary buffer, fluctuates significantly less than the outflux of Ca<sup>2+</sup> ions from the cleft. This is an effect of the low binding rate compared to the exit rate. The latter is approximately equal to the influx from the LCC during the steady state, not including the small outflux through the mobile buffer. The influx when two channels were open, between 2 and 4 ms, was ∼<italic>J</italic><sub>LCC</sub><inline-formula><tex-math id=\"M69\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\simeq}1870\\end{equation*}\\end{document}</tex-math></inline-formula> ions per ms, and the binding rate to the stationary buffer during the same steady state was <italic>J</italic><sub>SB</sub> = <italic>C</italic><sub>SS</sub> × <italic>k</italic><sup>+</sup> ≃ 20 ions per ms. This means that the outflux is 100-times larger than the flux to the buffers.</p></sec><sec><title>Statistics of single binding events</title><p>As seen in <xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>, where the results of the Student's <italic>t</italic>-tests and the KS-tests are presented, the predicted distributions of binding events from the continuous model fit the corresponding distributions of registered binding events from the RW model. The <italic>p</italic>-values and 95% CI are included for the <italic>t</italic>-tests and the <italic>p</italic>-values are included for the KS-tests. We found only one significant difference at the 5% level and none at our Bonferroni adjusted 0.4% level. Statistics of the binding event data are also presented graphically in <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8–10</xref><xref rid=\"fig9\" ref-type=\"fig\"/><xref rid=\"fig10\" ref-type=\"fig\"/>. These figures also visually support the results from the statistical tests presented in <xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>.</p><fig position=\"float\" id=\"fig8\"><label>FIGURE 8</label><caption><p>Statistical data for binding events registered from Random Walk simulations with one open LCC, which acted as the Ca<sup>2+</sup> source, situated in the center of the cleft. The binding events are registered at four different RyRs, positioned at 10, 30, 50, and 70 nm from the center of the cleft. Binding events are collected from 100 simulation runs. The registration started when the [Ca<sup>2+</sup>] had reached the steady state. The total time simulated was 30 ms. (<italic>A</italic>) Box-plot of the number of binding events from the runs at each receptor, together with a 95% confidence interval for the true means (<italic>red horizontal lines</italic>). The blue solid circles represent the expected number of binding events predicted by the continuous model. These values were computed on the basis of the fixed [Ca<sup>2+</sup>] at each receptor (see <italic>inset</italic>). In the box-plot, the green line represents the median of the data and the blue horizontal lines the limit of the upper and lower quartiles. The whiskers, i.e., the black lines extending from the blue boxes, represent the rest of the data up to a maximum length of 1.5 times the size of the two center quartiles. The green plus-signs are outliers. (<italic>B</italic>1–<italic>B</italic>4) Inter-event intervals (IEIs) from all runs presented in scaled histogram plots, corresponding to the receptor at positions 10, 30, 50, and 70 nm from the center of the cleft. The heights of the bars are scaled so the total area of a whole histogram = 1. The red lines show the probability distribution of the IEI from an homogeneous Poisson distribution with a rate based on the steady-state value of the [Ca<sup>2+</sup>] at the receptor. The blue asterisk indicates a significant difference, at 5% level, between the collected IEIs and the corresponding exponential function using a Kolmogorov-Smirnov test.</p></caption><graphic xlink:href=\"BIO.103523.wc.f8\"/></fig><fig position=\"float\" id=\"fig9\"><label>FIGURE 9</label><caption><p>Statistical data for binding events that are registered from Random Walk simulations. The Ca<sup>2+</sup> source was passive diffusion from the cytosol during diastole, i.e., the resulting [Ca<sup>2+</sup>] was, on average, 0.1 <italic>μ</italic>M. The binding events are registered at four different RyRs, positioned at 10, 30, 50, and 70 nm from the center of the cleft. The binding events were collected from 100 simulation runs. The total time simulated was 20 s. (<italic>A</italic>) Box-plot of the number of binding events from the runs at each receptor, together with a 95% confidence interval for the true means (<italic>red horizontal lines</italic>). The blue solid circles represent the expected number of binding events predicted by the continuous model. These values were computed on the basis of the fixed [Ca<sup>2+</sup>] at each receptor. (For an explanation of the box-plot, see the legend of <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8</xref>.) (<italic>B</italic>1–<italic>B</italic>4) Inter-event intervals from all runs presented in scaled histogram plots, corresponding to the receptor at positions 10, 30, 50, and 70 nm from the center of the cleft. (For an explanation of the histogram, see the legend of <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8</xref>.)</p></caption><graphic xlink:href=\"BIO.103523.wc.f9\"/></fig><fig position=\"float\" id=\"fig10\"><label>FIGURE 10</label><caption><p>Statistical data for binding events registered from Random Walk simulations. The Ca<sup>2+</sup> source is zero to three open LCCs, situated in the center of the cleft (<italic>A</italic>, <italic>inset</italic>). The binding events are registered at four different RyRs, positioned at 10, 30, 50, and 70 nm from the center of the cleft. The binding events were collected from 100 simulation runs. The total time simulated was 16 ms. (<italic>A</italic>) Box-plot of the number of binding events from the runs at each receptor, together with a 95% confidence interval for the true means (<italic>red horizontal lines</italic>). The blue solid circles represent the expected number of binding events predicted by the varying [Ca<sup>2+</sup>] from the continuous model at each receptor. (See <italic>inset</italic> for the [Ca<sup>2+</sup>] at the RyR at 70 nm.) No significant differences, at 5% level, were detected. (For an explanation of the box-plot, see the legend of <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8</xref>.) (<italic>B</italic>1–<italic>B</italic>4) Transformed inter-event intervals from all runs presented in scaled histogram plots, corresponding to the receptor at positions 10, 30, 50, and 70 nm from the center of the cleft. (For an explanation of the histogram, see the legend of <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8</xref>.)</p></caption><graphic xlink:href=\"BIO.103523.wc.f10\"/></fig><p>All three figures present the data in the same manner. In <xref rid=\"fig8\" ref-type=\"fig\">Figs. 8–10</xref><xref rid=\"fig9\" ref-type=\"fig\"/>,<xref rid=\"fig10\" ref-type=\"fig\"/> panels <italic>A</italic>, the number of binding events is presented in one box-plot for each RyR, together with a 95% CI of the true mean (<italic>red horizontal lines</italic>) and the expected number of binding events predicted by the continuous model (<italic>blue solid circles</italic>). The distributions of IEI, for each RyR, are presented in scaled histograms in <xref rid=\"fig8\" ref-type=\"fig\">Figs. 8–10</xref><xref rid=\"fig9\" ref-type=\"fig\"/>,<xref rid=\"fig10\" ref-type=\"fig\"/> panels <italic>B</italic>1–<italic>B</italic>4, in each of the three figures. The heights of the bars are scaled so that the total area of the histograms equals 1. This scaling enabled us to compare the distributions of IEIs with the expected distributions from the continuous model (<italic>red lines</italic>). <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8</xref> shows the results from the first set of simulations, where the Ca<sup>2+</sup> source was one LCC that was open constantly. The inset in <xref rid=\"fig8\" ref-type=\"fig\">Fig. 8 <italic>A</italic></xref>, shows the steady-state [Ca<sup>2+</sup>] in the cleft from the continuous model, where the concentration at each RyR is marked by solid blue circles. <xref rid=\"fig9\" ref-type=\"fig\">Fig. 9</xref> presents the results from the second set of simulations, where the Ca<sup>2+</sup> source was a passive influx from cytosol, which resulted in a [Ca<sup>2+</sup>] in the cleft that corresponds to a diastolic value of 0.1 <italic>μ</italic>M. The last figure, <xref rid=\"fig10\" ref-type=\"fig\">Fig. 10</xref>, presents the results from the last set of simulations. Here, the Ca<sup>2+</sup> source was three LCCs, which alternated between open and closed in the same manner during all simulations. The inset of <xref rid=\"fig10\" ref-type=\"fig\">Fig. 10 <italic>A</italic></xref> shows the varying [Ca<sup>2+</sup>] from the continuous model, at the RyR positioned at 70 nm from the center (<italic>red line</italic>), together with the shifting LCC current.</p><p>We observe that the number of binding events and the expected IEI distributions depend on the radial positions of the RyRs, for the first and third set of simulations, both of which are driven by a LCC current. This is not surprising, because the [Ca<sup>2+</sup>] are higher the closer they are to the channel. Perhaps more interesting, the RyRs in the cleft actually discriminate the [Ca<sup>2+</sup>] from a single Ca<sup>2+</sup> source. This is important for accounting for when the cleft is treated as a single compartment with the same lumped average [Ca<sup>2+</sup>] (<xref ref-type=\"bibr\" rid=\"bib15\">15</xref>). There is also no over- or underregistration of binding events on a certain RyR within each set of simulations. This means that the continuous model reproduces the binding events from the RW model independently of the radial position of the RyRs.</p></sec><sec><title>Mean binding rate registered at a single receptor</title><p>Our goodness-of-fit tests revealed that there are no significant differences among the registration of stochastic binding events in the two models. This is not an average result, but a result that holds on the level of single runs and at the level of IEI. To acquire a better understanding of how this could be true, we examined what we called a lumped binding rate, <inline-formula><tex-math id=\"M70\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {{\\lambda}}}^{{\\mathrm{L}}}(t),\\end{equation*}\\end{document}</tex-math></inline-formula> registered by a single RyR positioned 10 nm from the center during a run with one constantly open LCC.</p><p>Each Ca<sup>2+</sup> ion within a maximum distance of the receptor contributes, to a small extent, to the probability that a binding event will occur. This allows us to formulate the overall probability that a binding event will occur as a sum of small probabilities, where each is of the form <inline-formula><tex-math id=\"M71\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}P_{{\\mathrm{B}}}^{{\\mathrm{i}}}=1-{\\mathrm{e}}^{-{\\bar {{\\lambda}}}^{{\\mathrm{i}}}{\\Delta}{\\mathrm{t}}}\\end{equation*}\\end{document}</tex-math></inline-formula>; see <xref ref-type=\"disp-formula\" rid=\"fd24\">Eq. 24</xref>. Again, given small binding probabilities, this formulation can be approximated with <inline-formula><tex-math id=\"M72\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}P_{_{{\\mathrm{B}}}}^{{\\mathrm{i}}}{\\simeq}{\\bar {{\\lambda}}}^{{\\mathrm{i}}}{\\Delta}t.\\end{equation*}\\end{document}</tex-math></inline-formula> The resulting lumped binding probability is then <inline-formula><tex-math id=\"M73\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}P_{_{{\\mathrm{B}}}}^{{\\mathrm{L}}}={\\Delta}t{\\sum}_{{\\mathrm{i}}}{\\bar {{\\lambda}}}^{{\\mathrm{i}}}{\\Delta}t{\\bar {{\\lambda}}}^{{\\mathrm{L}}},\\end{equation*}\\end{document}</tex-math></inline-formula> where <inline-formula><tex-math id=\"M74\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {{\\lambda}}}^{{\\mathrm{L}}}\\end{equation*}\\end{document}</tex-math></inline-formula> represents the lumped binding rate.</p><p>During one simulation, we registered <inline-formula><tex-math id=\"M75\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {{\\lambda}}}^{{\\mathrm{L}}}\\end{equation*}\\end{document}</tex-math></inline-formula> at each time-step. These values are plotted against time in <xref rid=\"fig11\" ref-type=\"fig\">Fig. 11 <italic>A</italic></xref>. The right <italic>y</italic> axis gives the corresponding binding probabilities. The stochastic and discrete nature of the rates may be seen clearly in these chaotic data. The rate varies from time-step to time-step, as shown in the enlargement of the figure for <italic>t</italic> = [0, 0.01] ms, shown in <xref rid=\"fig11\" ref-type=\"fig\">Fig. 11 <italic>B</italic></xref>. The mean rate registered for the whole run was <inline-formula><tex-math id=\"M76\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {{\\lambda}}}^{{\\mathrm{L}}}=1.90\\end{equation*}\\end{document}</tex-math></inline-formula> ms<sup>−1</sup>. In 80% of the time-steps, the rate was smaller than this value, and in 11% of the time-steps, the rate equaled zero. In only 4.1% of the time-steps was the rate &gt;10 ms<sup>−1</sup> and the maximal registered rate for this run was 414 ms<sup>−1</sup>. These rates seem large but the resulting binding probabilities, <inline-formula><tex-math id=\"M77\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}P_{_{{\\mathrm{B}}}}^{{\\mathrm{L}}}={\\bar {{\\lambda}}}^{{\\mathrm{L}}}{\\Delta}t,\\end{equation*}\\end{document}</tex-math></inline-formula> were, as seen in the right <italic>y</italic> axis, all ≪1. We used the same size of time-step as earlier, Δ<italic>t</italic> = 1.25 × 10<sup>−4</sup> ms. The binding probability that corresponded with the mean rate for the whole run was 2.4 × 10<sup>−4</sup>. To be able to take the average of the binding rates over several time-steps, it has to make sense to take the sum of several binding rates. This measure is justified by the small binding probabilities that each receptor experiences every time-step (<xref rid=\"fig11\" ref-type=\"fig\">Fig. 11, <italic>A</italic> and <italic>B</italic></xref>). The crucial issue was how the average binding rate fluctuates on a larger timescale, i.e., do the large variations in binding rates in each time-step average-out at a larger timescale and if so, how small can this timescale be?</p><fig position=\"float\" id=\"fig11\"><label>FIGURE 11</label><caption><p>(<italic>A</italic> and <italic>B</italic>) Lumped binding rates for each time-step, registered from one RyR during a single Random Walk simulation. In the simulation, one constantly open LCC channel was used and the RyR was positioned 10 nm from the center of the cleft. (<italic>B</italic>) Enlargement of panel <italic>A</italic> for <italic>t</italic> = [0, 0.01] ms. The mean binding rate fluctuates a lot for each time-step. (<italic>C</italic>) Filtered version of the binding rate. A Gaussian kernel with <italic>σ</italic> = 0.26 ms, corresponding to the scale of the registered IEIs, was chosen for the filtering. (<italic>A</italic>–<italic>C</italic>) Corresponding binding probabilities are given by the right <italic>y</italic> axis. For the <italic>i</italic><sup>th</sup> time-step, this quantity is computed by <inline-formula><tex-math id=\"M78\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}P_{{\\mathrm{i}}}=1-e^{-{\\lambda}_{{\\mathrm{i}}}\\hspace{.167em}{\\Delta}{\\mathrm{t}}}{\\simeq}{\\lambda}_{{\\mathrm{i}}}\\hspace{.167em}{\\Delta}t\\end{equation*}\\end{document}</tex-math></inline-formula>.</p></caption><graphic xlink:href=\"BIO.103523.lw.f11\"/></fig><p>The averaged binding rate did not vary much from run to run. The mean averaged rate from 100 runs was [1.904 ± 0.019] ms<sup>−1</sup>. This value did not differ significantly from the constant rate from the continuous model, <italic>λ</italic><sub>c</sub> = 1.91 ms<sup>−1</sup>, <italic>p</italic>-value = 0.74. This result corresponds to the failure of detecting a significant difference between the average number of binding events that was registered in the RW simulations and the number given by the continuous model (see the results of the Student's <italic>t</italic>-tests in <xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>). The variations in the binding rate at the timescale of a whole simulation run thus averaged-out and were statistically indistinguishable from the continuous constant rate.</p><p>On a smaller timescale, we would expect the averaged rate to fluctuate more. For example, the mean rate for the interval shown in <xref rid=\"fig11\" ref-type=\"fig\">Fig. 11 <italic>B</italic></xref>, i.e., <italic>t</italic> = [0, 0.01], was 2.70 ms<sup>−1</sup>. This is greater than the average rate for the whole run, which was 1.90 ms<sup>−1</sup>. However, the fluctuation at this timescale does not tell us much, because the expected number of binding events with this rate at this timescale is 0.027. To investigate the effect on the actual binding events, we have to take the average on a larger timescale. A proper scale would be the mean IEI registered in the simulation runs. This was found to be 0.52 ms (<xref rid=\"fig8\" ref-type=\"fig\">Fig. 8 <italic>B</italic></xref>1). We filtered the registered rate with a Gaussian filter, which acts as a weighted mean over a certain time window defined by the width of the filter, <italic>σ</italic> (<xref ref-type=\"bibr\" rid=\"bib48\">48</xref>). This width was set to half the size of the mean IEI, 0.26 ms. The result is presented in <xref rid=\"fig11\" ref-type=\"fig\">Fig. 11 <italic>C</italic></xref>, together with the constant rate from the continuous model. The filtered rate is a continuous function of time and does not vary nearly as much as the unfiltered rate in <xref rid=\"fig11\" ref-type=\"fig\">Fig. 11 <italic>A</italic></xref>. The maximal value of the filtered signal was 2.1 ms<sup>−1</sup>, the minimal was 1.7 ms<sup>−1</sup>, and the standard deviation from the mean, which of course was the same as the unfiltered rate, was 0.1 ms<sup>−1</sup>. This small variation explains why the IEIs of the registered binding events from the RW model were statistically indistinguishable from those of the continuous model.</p></sec><sec><title>Parameter sensitivity</title><p>To check the dependency of some of the parameters we have used, we made five runs in which we altered the diffusion constant, <italic>D</italic><sub>c</sub>, together with the maximal input current from one open LCC, <inline-formula><tex-math id=\"M79\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {i}}_{{\\mathrm{LCC}}},\\end{equation*}\\end{document}</tex-math></inline-formula> in the same manner as Tanskanen et al. (<xref ref-type=\"bibr\" rid=\"bib32\">32</xref>) did. We scaled the <italic>D</italic><sub>c</sub> and <inline-formula><tex-math id=\"M80\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {i}}_{{\\mathrm{LCC}}}\\end{equation*}\\end{document}</tex-math></inline-formula> by factors of [5, 2, 1, 0.5, 0.1] and ran 100 runs of the steady-state condition, in which one LCC was open. The spatial resolution for the registration of binding events was set to <italic>σ</italic> = 5 nm for every run. We compared the number of registered binding events with the expected number from the continuous model. The number for the latter was constant in all runs, because the concentrations at the receptors were the same under the scaling. The result is shown in <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>A</italic></xref>. The figure shows a box-plot of the number of binding events registered at the receptor 30 nm from the center of the cleft versus the scale on the <italic>x</italic> axis. There are no significant differences for scale = [5, 2, 1, 0.5], but for scale = 0.1, there is. To investigate the dependency of the parameters further, we also altered <italic>σ</italic>. We used [5, 2, 1, 0.5] for <italic>σ</italic> and also did 100 runs for each different value. The result is presented in a similar box-plot in <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>B</italic></xref>. Note that the leftmost datapoints in this figure are identical to the rightmost datapoints from the previous figure. From the figure, we see that the number of registered binding events falls steadily. This illustrates that the binding event registration depends, not only on physical parameters, but also on the spatial resolution of the RW method. This observation coincides with the parameters used in the dimensionless on-rate <xref ref-type=\"disp-formula\" rid=\"fd29\">Eq. 29</xref>.</p><fig position=\"float\" id=\"fig12\"><label>FIGURE 12</label><caption><p>(<italic>A</italic> and <italic>B</italic>) Number of registered binding events from 100 runs each, where we altered different parameters. The data were collected from a receptor 30 nm from the center and are represented by the box-plots together with a 95% confidence interval for the true means (<italic>red horizontal lines</italic>). The blue solid circles represent the expected number of binding events that are predicted by the continuous model. (<italic>A</italic>) We scaled the number of Ca<sup>2+</sup> ions that enter the cleft, i.e., <inline-formula><tex-math id=\"M81\">\\documentclass[10pt]{article}\n",
      "\\usepackage{amsmath}\n",
      "\\usepackage{wasysym} \n",
      "\\usepackage{amsfonts} \n",
      "\\usepackage{amssymb} \n",
      "\\usepackage{amsbsy}\n",
      "\\usepackage{mathrsfs}\n",
      "\\usepackage{pmc}\n",
      "\\usepackage[Euler]{upgreek}\n",
      "\\pagestyle{empty}\n",
      "\n",
      "\\oddsidemargin -1.0in\n",
      "\n",
      "\\begin{document}\n",
      "\\begin{equation*}{\\bar {i}}_{{\\mathrm{LCC}}},\\end{equation*}\\end{document}</tex-math></inline-formula> together with the diffusion constant <italic>D</italic>, with a factor represented by the <italic>x</italic> axis. The spatial resolution was constant for these simulations, <italic>σ</italic> = 5 nm. The blue asterisk denotes a statistical difference between the continuous model and the RW model for scale = 0.1. (<italic>B</italic>) We kept the scale constant at 0.1, but altered the spatial resolution (see the <italic>x</italic> axis). Here, the difference between the RW model and the continuous model increased as the mean value of the collected binding events declined with the spatial resolution. (<italic>C</italic>) We ran the simulation 100 times. We collected the mean binding rates for each run that the receptor were exposed to. The data from each set of 100 runs are presented as 95% confidence intervals for the true means. The blue horizontal lines represent the binding rates collected from runs in which we registered binding events, as in panel <italic>B</italic>. The red horizontal lines represent binding rates collected from runs in which we did not register binding events, only the rate. In these runs we could not differentiate statistically between the registered binding rates and the rates predicted from the continuous model.</p></caption><graphic xlink:href=\"BIO.103523.wc.f12\"/></fig><p>One large difference between the continuous model and the RW model is that in the RW model, a binding event actually leads to a removal of an ion from the cleft, in contrast to the continuous model where nothing happens. To test whether this difference is crucial for the registered difference between the two models as seen in <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>B</italic></xref>, we performed the same simulations, but without registering any binding events. Instead we registered the mean binding rate from each run and compared this with the rate predicted from the continuous model. In <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>C</italic></xref>, the red lines represent 95% confidence intervals of the true mean binding rate from the 100 runs. The rate predicted from the continuous model is represented by the blue solid circles. We cannot differentiate statistically between the collected mean binding rates and those of the continuous model. We also collected the mean binding rates from the simulations we did in <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>B</italic></xref>, in which ions were removed from the solution after they were bound. The 95% confidence interval of the true mean for these binding rates is represented by the blue horizontal lines. Here, we see that the binding rates follow the number of registered binding events from <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>B</italic></xref>, and not the predicted rate from the continuous model. These results illustrate why the RW model starts to differ from the continuous model for low values of the diffusion constant together with small values of the spatial resolution.</p><p>As discussed above, the difference between the two models is parameter-sensitive. This sensitivity can be expressed by the dimensionless on-rate, <italic>k</italic><sup>+*</sup>, from <xref ref-type=\"disp-formula\" rid=\"fd29\">Eq. 29</xref>. This value depends on the diffusion constant, <italic>D</italic>, the macroscopic on-rate, <italic>k</italic><sup>+</sup>, and the spatial resolution, <italic>σ</italic>. For the simulations for which we registered a difference between the RW model and the continuous model, i.e., for scale = 0.1 in <xref rid=\"fig12\" ref-type=\"fig\">Fig. 12 <italic>A</italic>, <italic>k</italic></xref><sup>+*</sup> = 0.013. This value indicates an upper limit for when the two models start to diverge. The spatial resolution is, in a sense, a free parameter. One could, in theory, make it as small as one likes, thus forcing a difference between the two models. Alternatively, one could make it large to smooth out a potential difference. In practice, the value of this parameter is determined by the level of spatial detail that is required in the simulation.</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2745 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\"><title>Results</title><p>Eyes task and Hinting task performance were significantly correlated\n",
      " (<italic>r</italic>=0.389; <italic>P</italic>=0.001). Despite this, patients only performed\n",
      " significantly below controls on the Eyes task (<italic>t</italic>=–2.3;\n",
      " d.f.=149; <italic>P</italic>=0.023). Eyes task performance correlated with verbal and\n",
      " performance IQ, and verbal and spatial working memory\n",
      " (<italic>P</italic>=0.003–0.000006) but not with verbal episodic memory. Hinting\n",
      " task performance by contrast was correlated with verbal episodic and working\n",
      " memory (<italic>P</italic>=0.008–0.02) but not IQ or spatial memory performance.\n",
      " Neither task correlated with attention control. Eyes task performance alone\n",
      " was moderately negatively correlated with positive symptom severity\n",
      " (<italic>r</italic>=–0.25; <italic>P</italic>=0.03) but not negative symptom severity.\n",
      " Neither task was associated with age, gender, duration of illness or\n",
      " medication dosage.</p><p>Scores on the ILS were significantly positively associated with Eyes task\n",
      " performance (<italic>r</italic>=0.41; <italic>P</italic>=0.001) but not with Hinting task\n",
      " performance (<italic>P</italic>&gt;0.05). To test whether social functioning was\n",
      " predicted by mental state decoding performance (Eyes task performance) we\n",
      " performed a multiple regression analysis with ILS scores as the dependent\n",
      " variable. Symptom severity, which was significantly correlated with social\n",
      " function, was entered in the first step of the analysis, explaining 26.5% of\n",
      " variance in ILS scores (20% of which was contributed by negative symptom\n",
      " severity alone). This was followed by scores on the Eyes task and Hinting\n",
      " tasks on the second step. Eyes task performance explained a further 9.0% of\n",
      " the variance in social functioning (<italic>F</italic><sub>1,63</sub> change=8.5;\n",
      " <italic>P</italic>=0.005); Hinting task performance was not a significant variable in\n",
      " the regression equation. The statistical significance of these results was\n",
      " unchanged by considering patients with schizophrenia and patients with\n",
      " schizoaffective disorder separately.</p><p>We performed two further stepwise multiple regression analyses. In the\n",
      " first analysis, when verbal IQ (the neuropsychological measure most highly\n",
      " correlated with ILS performance) was entered on the first step of the analysis\n",
      " followed by Eyes task performance on the second step, Eyes task performance\n",
      " continued to significantly predict variance in ILS (<italic>r</italic><sup>2</sup>\n",
      " change=0.09, <italic>F</italic><sub>1,68</sub>=7.26, <italic>P</italic>=0.009). In the second\n",
      " regression analysis, when Eyes task performance was entered on the first step\n",
      " and verbal IQ on the second, verbal IQ’s ability to predict ILS scores\n",
      " was partly attenuated (<italic>r</italic><sup>2</sup> change reducing from 16.3% to\n",
      " 7.5%). Following Baron &amp; Kenny’s\n",
      " criteria<xref ref-type=\"bibr\" rid=\"ref16\">16</xref> for\n",
      " mediating variables this suggests that Eyes task performance partly mediated\n",
      " the relationship between verbal IQ and ILS scores. Again, the statistical\n",
      " significance of these results was unchanged by considering patients with\n",
      " schizophrenia and patients with schizoaffective disorder separately.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2768 prediction: method true label: other\n",
      "<sec><title>Results</title><p>The clinicopathologic, treatment and follow-up characteristics of the 325 patients in the study are summarised in <xref rid=\"tbl1\" ref-type=\"table\">Tables 1</xref>, <xref rid=\"tbl2\" ref-type=\"table\">2</xref> and <xref rid=\"tbl3\" ref-type=\"table\">3</xref>.</p><p><italic>(A) Patients who did not initiate endocrine therapy (n</italic>=<italic>38)</italic></p><p>In 10 cases, despite HR-positive carcinoma, a recommendation for endocrine therapy was not made. The reasons for this included a low-risk constellation (pT1a/b N0, favourable grading) and/or advanced age with considerable comorbidity. Twenty-eight patients (8.9%) refused the recommended endocrine therapy after extensive counselling and never began this treatment. Univariate analysis revealed that older patients, those who were initially diagnosed at the beginning of the study period and those who did not receive adjuvant chemotherapy were more likely to refuse therapy (advanced age: <italic>P</italic>=0.019; study period: 1997–2000 <italic>vs</italic> 2001–2003: <italic>P</italic>=0.0070; no chemotherapy: <italic>P</italic>&lt;0.0001;<xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>). Additionally, patients who had a favourable disease stage (stage I) tended to refuse the recommended therapy (<italic>P</italic>=0.0546).</p><p><italic>(B) Patients who initiated endocrine therapy (n</italic>=<italic>287)</italic></p><p>The initial endocrine agents prescribed are listed in <xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>. Forty-five patients were treated within the randomised, phase III, double-blind Breast International Group (BIG) 1-98 four-arm study (letrozole or tamoxifen as monotherapy or sequential therapy) (<xref ref-type=\"other\" rid=\"bib4\">Coates <italic>et al</italic>, 2007</xref>). The medication was unblinded in 25 of these patients. Even when the study drug had been unblinded, these cases were listed under the category ‘study medication (BIG 1-98)'. Of the 45 BIG 1-98 study patients, 30 (66.7%) had completed 5 years of treatment according to the study guidelines. Of the 14 patients who prematurely discontinued their randomly assigned trial medication for reasons other than disease recurrence, 8 women fully completed the endocrine therapy over a 5-year course outside of the trial (57.3%).</p><p>In 50 patients, there was a change of the antihormonal agent prescribed within the planned 5-year period. Of these, four patients changed their medication twice and one patient three times. Twenty-five of the 50 patients changed their medication due to adverse effects of the endocrine therapy (three changed the medication twice and one three times). Of these, 20 patients (80%) fully completed the therapy after drug change.</p><p>The therapy course, reported in <xref rid=\"tbl3\" ref-type=\"table\">Tables 3</xref> and <xref rid=\"tbl5\" ref-type=\"table\">5</xref>, was as follows:</p><p>Of the 287 patients who initiated endocrine therapy, 191 (66.6%) fully completed the targeted therapy (four patients interrupted therapy for up to 3 months, another for 5 months). Eleven patients (3.8%) discontinued the therapy due to death, whereas 43 patients (15.0%) ceased therapy due to breast cancer recurrence. In nine cases (3.1%), the therapy was discontinued by the physician due to serious medical reasons independent from breast cancer disease and therapy-related adverse effects (advanced age/dementia/need for nursing home care, <italic>n</italic>=5; incurable malignancy other than breast cancer, <italic>n</italic>=3; irreversible coma following severe head trauma, <italic>n</italic>=1).</p><p><italic>Non-adherence</italic>: Thirty-one patients (10.8%) chose independently to end the therapy. The main reasons for this given by the patients are listed in <xref rid=\"tbl5\" ref-type=\"table\">Table 5</xref>. In the seven variables used in our univariate analyses, location of follow-up was the only significant predictor, showing that patients who did not have follow-up in an oncologic unit, but rather with a general practitioner, were more likely to be non-adherent to therapy (<italic>P</italic>=0.0088; <xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>).</p><p><italic>Location of follow-up</italic>: Of the 289 patients who received adjuvant endocrine therapy, 215 (74.4%) were treated in an oncology unit; of these, 194 patients in the internal oncology unit (67.1%) and 21 (7.3%) in an external oncologic unit. Seventy-two patients (24.9) had further follow-up and treatment through a general practitioner. Two patients (0.7%) were lost to follow-up, and the location of the oncologic care was unknown.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2782 prediction: method true label: other\n",
      "<sec><title>Results</title><p>At examination during adolescence (mean age 14.0 years), 199 out of 262 girls (76.0%) had reached menarche (<xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref>). For all birth size variables, except ponderal index, girls who were relatively small at birth were younger at menarche than those with larger birth size (<xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>). Median age at menarche for girls in the shortest birth length category (&lt;49.0 cm) was 12.5 years (95% CI, 12.26–12.74), compared with 13.3 years (95% CI, 12.95–13.72) for the highest category (⩾51.0 cm). Similar results were observed for birth weight, whereas the differences in age at menarche for the other birth size variables were smaller. Adjustment for potentially confounding factors did not substantially alter these results (data not shown).</p><p>We found that birth length, weight and head circumference were positively associated with adolescent height and weight (<xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>). The associations were further strengthened after adjustment for age at examination in adolescence, menarcheal status and geographical region. Comparing the longest and shortest categories at birth (⩾51 <italic>vs</italic> ⩽49 cm), the longest were on average 6.2 cm taller (164.6 <italic>vs</italic> 158.4 cm) and 4.2 kg heavier (55.0 <italic>vs</italic> 50.8 kg) in adolescence. Girls in the heaviest birth category (⩾3700 g) were 4.4 cm taller and 4.7 kg heavier in adolescence than those with birth weight below 3200 g. For ponderal index, there was a negative association with adolescent height; girls who were thin at birth were on average taller in adolescence, but the association was attenuated after adjustment for age, menarcheal status and residential region. Girls in the highest category of birth weight and head circumference had higher BMI in adolescence compared with girls of average birth size (<italic>P</italic>&lt;0.02, results not shown). None of the birth size variables were related to waist/hip ratio in adolescence, and skin-fold thickness at birth showed no association with any of the anthropometric variables in adolescence (data not shown). Adjustment for gestational age had no influence on the observed associations (results not shown).</p><p>A negative association between head circumference at birth and circulating level of prolactin in adolescence (<italic>P</italic>-trend=0.03) was attenuated after adjustment for age at puberty, menarcheal status and residential region (<italic>P</italic>-trend=0.12) (<xref rid=\"tbl3\" ref-type=\"table\">Table 3</xref>). For other birth size factors, we found no relation with prolactin. There was a significant negative association between birth weight, head circumference and subscapular skin-fold thickness and circulating levels of DHEAS in adolescence (i.e. lower DHEAS levels with increasing size), unaffected by adjustment for the above factors. Mean (geometric) DHEAS for adolescent girls in the highest birth weight category was 1.95 <italic>μ</italic>mol l<sup>−1</sup> (95% CI, 1.64–2.32) compared with 2.28 <italic>μ</italic>mol l<sup>−1</sup> (95% CI, 1.88–2.78) for the lowest category of birth weight.</p><p>For girls who were in the follicular phase of the menstrual cycle (<xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>), we found no evidence for any association between birth size and FTI or level of androstenedione in adolescence. For oestradiol, there was a tendency for a weak positive association with all factors of birth size, but significant only for subscapular skin-fold thickness. Adjustment for age at examination, time since menarche and region did not substantially influence these estimates, whereas adjustment for day of menstrual cycle reinforced them.</p><p>Comparing girls from the random sample with those whose mothers had higher risk of delivering an SGA child, we found no difference for any of the variables (all <italic>P</italic>-values &gt;0.1, results not shown). Stratified analyses of these groups showed similar results as the overall analyses.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2789 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"results\"><title>RESULTS</title><p>The geometric mean airborne concentration of cultivable <italic>Staphylococcus</italic> species circulating in the air of poultry houses containing 1- to 2-day-old chickens was 19 (±1.2) × 10<sup>3</sup> CFU m<sup>−3</sup>, with this number representing ∼90% of the total cultivable bacteria present on the nutrient plates (data not shown). Importantly, it was found to be impossible to apply standard culture-dependent methods using impaction of bacteria onto nutrient media after the first stage of the fattening period since all subsequent samples after this stage were so oversaturated with bacteria that colonies could not be counted, even when the sampling duration was very short (5 s corresponding to &lt;5 l of air).</p><p>The estimated Staphylococcal species load, as quantified by Q-PCR during the first stage of sampling [GM ± GSD = 88 (±5.9) × 10<sup>3</sup> m<sup>−3</sup>] (<xref ref-type=\"fig\" rid=\"fig1\">Fig. 1</xref>), is ∼4-fold higher than the value obtained by culture of impacted bacteria. Furthermore, this estimate represents a mean value of only 0.75% (±1%) (<xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>) of the total bacterial load as measured by Q-PCR (<italic>N</italic> = 59) (<xref ref-type=\"fig\" rid=\"fig2\">Fig. 2</xref>). The comparative analysis by Q-PCR of Staphylococcal versus total bacterial load suggests that this ratio changes during the fattening period (<xref ref-type=\"table\" rid=\"tbl1\">Table 1</xref>). Both the number of total bacterial cells and <italic>Staphylococcus</italic> species cells measured by Q-PCR increase massively during the fattening period, as do the inhalable dust and endotoxin levels (<xref ref-type=\"fig\" rid=\"fig1\">Figs 1</xref>– <xref ref-type=\"fig\" rid=\"fig4\">4</xref>). Analysis of variance show that the stage of fattening has a significant influence on the quantity of all bioaerosols, while the bird density in the poultry house has no effect (<xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>). We found significant positive correlations between inhalable dust level and the parameters of endotoxin load, number of total bacterial cells estimated by Q-PCR, number of <italic>Staphylococcus</italic> cells estimated by Q-PCR (respectively, Pearson correlation = 0.573, 0.451 and 0.603; <italic>P</italic> &lt; 0.02, <italic>N</italic> = 42). During the load to the slaughterhouse, the workers who directly handle the birds are exposed to higher levels of bioaerosol than are the forklift drivers (<xref ref-type=\"fig\" rid=\"fig1\">Figs 1</xref>– <xref ref-type=\"fig\" rid=\"fig4\">4</xref>). We found that the geometric mean inhalable dust concentration for bird catchers was of 26.3 ± 1.9 mg m<sup>−3</sup>. Eighty-five per cent (11 of 13) of the catchers were exposed to dust level values &gt;10 mg m<sup>−3</sup>. The mean exposure level of catchers to endotoxin concentration is 6198 ± 2.3 EU m<sup>−3</sup> air. This value is 6-fold more elevated than the Swiss occupational recommended value (1000 EU m<sup>−3</sup>). As measured by Q-PCR, the geometric mean exposure levels of bird catchers to total bacteria and <italic>Staphylococcus</italic> species are, respectively, 53 (±2.6) × 10<sup>7</sup> cells m<sup>−3</sup> and 62 (±1.9) × 10<sup>6</sup> cells m<sup>−3</sup>.</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1.</label><caption><p>Airborne staphylococci as percentage of total bacteria, estimated by Q-PCR, in 12 poultry houses, at different stages of fattening period</p></caption><table frame=\"hsides\" rules=\"groups\"><tbody><tr><td rowspan=\"1\" colspan=\"1\">Stage of fattening</td><td rowspan=\"1\" colspan=\"1\">1 (1–2 days old)</td><td rowspan=\"1\" colspan=\"1\">2 (24–27 days old)</td><td rowspan=\"1\" colspan=\"1\">3 (40–53 days old)</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Staphylococci as percentage of total bacteria</td><td align=\"char\" char=\"plusmn\" rowspan=\"1\" colspan=\"1\">0.75 ± 1.2</td><td align=\"char\" char=\"plusmn\" rowspan=\"1\" colspan=\"1\">9.6 ± 2.4</td><td align=\"char\" char=\"plusmn\" rowspan=\"1\" colspan=\"1\">11 ± 1.5</td></tr></tbody></table></table-wrap><table-wrap id=\"tbl2\" position=\"float\"><label>Table 2.</label><caption><p>Analysis of variance: effects of stages of fattening and density of birds on the values of bioaerosols concentrations, estimated by Q-PCR method, in 12 Swiss poultry houses</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"1\" colspan=\"1\">Source of variation</td><td rowspan=\"1\" colspan=\"1\">d.f.</td><td rowspan=\"1\" colspan=\"1\">Mean sum of square</td><td rowspan=\"1\" colspan=\"1\"><italic>F</italic>-ratio</td><td rowspan=\"1\" colspan=\"1\"><italic>P</italic></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Response: airborne endotoxin, <italic>R</italic><sup>2</sup> = 0.592</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stages</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.882</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20.164</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.018</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.041</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.840</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stage × density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.043</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.097</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.907</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Residuals</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.44</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">Response: airborne Staphylococcus species, <italic>R</italic><sup>2</sup> = 0.676</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stages</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">123.332</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23.106</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18.614</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.487</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.072</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stage × density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.869</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.163</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.851</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Residuals</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.338</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">Response: total airborne bacteria species, <italic>R</italic><sup>2</sup> = 0.830</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stages</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">33.208</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">62.734</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.146</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.276</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.603</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stage × density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.524</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.989</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.384</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Residuals</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.529</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">Response: inhalable dust, <italic>R</italic><sup>2</sup> = 0.792</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stages</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20.416</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">39.972</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">&lt;0.001</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.007</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.013</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.910</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Stage × density</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.133</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.261</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.773</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Residuals</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.511</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr></tbody></table><table-wrap-foot><fn><p><italic>R</italic><sup>2</sup> = percentage of variance explained by the model. Geometric mean values of each stage for each bioaerosol are presented in the <xref ref-type=\"fig\" rid=\"fig1\">Figs 1</xref>– <xref ref-type=\"fig\" rid=\"fig4\">4</xref>. Significance was set at <italic>P</italic> &lt; 0.05.</p></fn></table-wrap-foot></table-wrap><fig id=\"fig1\" position=\"float\"><label>Fig. 1.</label><caption><p>Geometric mean of <italic>Staphylococcus</italic> species (estimated by Q-PCR) per cubic metre of air sampled at different stages of growth (Stage 1, 2 and 3; see details in text) and within the breathing zone of bird's catcher and driver as they caught the birds ready to be transported for slaughter, in 12 poultry houses. Numbers beside the bars indicate the sample size.</p></caption><graphic xlink:href=\"annhygmen021f01_lw\"/></fig><fig id=\"fig2\" position=\"float\"><label>Fig. 2.</label><caption><p>Geometric mean of total bacteria (estimated by Q-PCR) per cubic metre of air sampled at different stages of growth (Stage 1, 2 and 3; see details in text) and within the breathing zone of bird's catcher and driver as they caught the birds ready to be transported for slaughter, in 12 poultry houses. Numbers beside the bars indicate the sample size.</p></caption><graphic xlink:href=\"annhygmen021f02_lw\"/></fig><fig id=\"fig3\" position=\"float\"><label>Fig. 3.</label><caption><p>Geometric mean of endotoxin concentration per cubic metre of air sampled at different stages of growth (Stage 1, 2 and 3; see details in text) and within the breathing zone of bird's catcher and driver as they caught the birds ready to be transported for slaughter, in 12 poultry houses. Numbers beside the bars indicate the sample size.</p></caption><graphic xlink:href=\"annhygmen021f03_lw\"/></fig><fig id=\"fig4\" position=\"float\"><label>Fig. 4.</label><caption><p>Geometric mean of dust concentration per cubic metre of air sampled at different stages of growth (Stage 1, 2 and 3; see details in text) and within the breathing zone of bird's catcher and driver as they caught the birds ready to be transported for slaughter, in 12 poultry houses. Numbers beside the bars indicate the sample size.</p></caption><graphic xlink:href=\"annhygmen021f04_lw\"/></fig></sec> \n",
      "\n",
      "\n",
      "\n",
      "2795 prediction: method true label: other\n",
      "<sec><title>Results</title><p>Tissue was received from 24 patients (mean age 59, range 32–73, 18 women, 6 men). Eighteen patients were taken from the PH1/066 single agent UK CA4P trial (<xref ref-type=\"other\" rid=\"bib26\">Rustin <italic>et al</italic>, 2003</xref>), and 6 from the PH1/092 CA4P+anti-CEA antibody study (Gaya AM <italic>et al</italic>, 2008, unpublished). The most common histological subtypes (<xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref>) were colorectal adenocarcinoma (7), uterine leiomyosarcoma (4), and ovarian adenocarcinoma (3). The age of the tumour blocks varied between 6 months and 21 years. All but one block contained primary tumour from surgical specimens, not metastatic tissue as patients were not rebiopsied before entrance to the phase I trials. We only studied patients who received doses of CA4P between 40 and 114 mg m<sup>−2</sup> that have been associated with changes in DCE-MRI kinetic variables.</p><p>There was a wide range of DCE-MRI variable change following CA4P (<xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>). Percentage change in <italic>K</italic><sup>trans</sup> from mean baseline following CA4P varied between +111 and −75%, and IAUGC<sub>60</sub> between +186 and −90%. In reality, only falls in these parameters (consistent with tumour vascular shutdown) are clinically significant. The 95% confidence interval for <italic>K</italic><sup>trans</sup> change (determined from repeatability analysis in the individual studies and repeated with the combined data) was −62.5 to +166.5%, IAUGC<sub>60</sub>±80%. There was no obvious visual relationship seen between CA4P dose and fall in kinetic parameters (<xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>). Three patients (3, 10, 12) showed significant falls in <italic>K</italic><sup>trans</sup> following CA4P (outside 95% CI), and only 1 patient (12) a significant fall in IAUGC<sub>60</sub>.</p><p>No statistically significant correlations were seen between the continuous or categorical variables and the DCE-MRI kinetic variables (<xref rid=\"tbl5a\" ref-type=\"table\">Tables 5a</xref> and <xref rid=\"tbl5b\" ref-type=\"table\">5b</xref>) other than between ang-2 and <italic>K</italic><sup>trans</sup> (<italic>P</italic>=0.044). Inspecting <xref rid=\"tbl3\" ref-type=\"table\">Tables 3</xref> and <xref rid=\"tbl4\" ref-type=\"table\">4</xref>, there are no obvious relationships between the angiogenic profile and DCE-MRI data.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2810 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\"><title>RESULTS</title><p>NRK-52E normal rat tubular epithelial cells were used to examine TGF-β1-induced EMT. As previously reported for other renal tubular epithelial cells (<xref ref-type=\"bibr\" rid=\"r40\">40</xref>, <xref ref-type=\"bibr\" rid=\"r41\">41</xref>, <xref ref-type=\"bibr\" rid=\"r49\">49</xref>), decreased expression of ZO-1 (epithelial marker) and increased expression of vimentin (mesenchymal marker) were observed in cells treated with TGF-β1 (<xref rid=\"f1\" ref-type=\"fig\">Fig. 1<italic>A</italic></xref>). Similar decreased expression of E-cadherin (epithelial protein; <xref rid=\"f1\" ref-type=\"fig\">Fig. 1<italic>B</italic></xref>, <italic>top</italic>) and increased expression of α-SMA (mesenchymal protein; <xref rid=\"f1\" ref-type=\"fig\">Fig. 1<italic>B</italic></xref>, <italic>middle</italic>) were observed. Thus TGF-β1 induces EMT-like changes in NRK52E cells.</p><p>Many intracellular signaling molecules have been implicated in TGF-β1-induced EMT, including Smad3, ILK, and p38 MAPK. PI3K→PKB/Akt signaling have been suggested to play a role in TGF-β1 signaling in mammary and lens epithelial cells (<xref ref-type=\"bibr\" rid=\"r3\">3</xref>). To assess whether TGF-β1 activated PI3K, we used mouse embryo fibroblasts stably expressing GFP-Akt-PH, the plasma membrane binding of which is indicative of phosphatidylinositol 3,4,5-trisphosphate [PtdIns(3,4,5)P<sub>3</sub>] formation due to PI3K activation. Treatment of mouse embryonic fibroblast (MEF) cells with TGF-β1 induced the translocation of GFP-Akt-PH to the plasma membrane (see Supplemental Fig. 1<italic>A</italic>). (Supplemental data for this article is available online at the <italic>American Journal of Physiology-Renal Physiology</italic> website.) TGF-β1 also stimulated PtdIns(3,4,5)P<sub>3</sub> formation in NRK52E kidney epithelial cells, suggesting that TGF-β1 triggers PI3K activation in these cells (Supplemental Fig. 1<italic>B</italic>). Increased phosphorylation of the downstream target of PI3K PKB/Akt was also observed in NRK52E cells upon TGF-β1 treatment (<xref rid=\"f2\" ref-type=\"fig\">Fig. 2 A</xref>). Phosphorylation of a downstream target of PKB/Akt, GSK-3β, was also elevated upon TGF-β1 stimulation. Pretreatment of cells with an inhibitor of either PI3K (LY-294002) or PKB/Akt (Akt inhibitor II) significantly reduced the phosphorylation of both PKB/Akt and GSK-3β in the absence or presence of TGF-β1. These data were quantified using electrochemiluminescence, revealing similar patterns of TGF-β1-induced activation and inhibition (<xref rid=\"f2\" ref-type=\"fig\">Fig. 2<italic>B</italic></xref>). Together, these data suggest that TGF-β1 activates PKB/Akt in renal epithelial cells in a PI3K-dependent manner.</p><p>Pharmacological inhibitors such as LY-294002 may cross-react with other protein kinases (<xref ref-type=\"bibr\" rid=\"r14\">14</xref>). To further support our observations, plasmids expressing PTEN, a PtdIns(3,4,5)P<sub>3</sub> lipid phosphatase that limits PKB/Akt activity in cells (<xref ref-type=\"bibr\" rid=\"r29\">29</xref>), or a mutant form of PTEN with no lipid phosphatase activity (PTEN C124S) were utilized. As previously observed, TGF-β1 induced the activation of PKB/Akt in cells transfected with control plasmid (pCGN; <xref rid=\"f3\" ref-type=\"fig\">Fig. 3<italic>A</italic></xref>). Transfection of wild-type PTEN abolished PKB/Akt phosphorylation and significantly reduced GSK-3β phosphorylation on Ser<sup>9</sup>. In contrast, expression of the lipid phosphatase mutant C124S PTEN did not affect PKB/Akt or GSK-3β phosphorylation (<xref rid=\"f3\" ref-type=\"fig\">Fig. 3<italic>A</italic></xref>). Again, these data were supported by quantified electrochemiluminescence analysis showing similar results (<xref rid=\"f3\" ref-type=\"fig\">Fig. 3</xref>, <italic>B</italic> and <italic>C</italic>). Together, these data suggest that reductions in the level of PtdIns(3,4,5)P<sub>3</sub> limit the degree of PKB/Akt activation in the absence or presence of TGF-β1 and that TGF-β1-mediated activation of PKB/Akt is a PI3K-dependent event in renal epithelial cells.</p><p>The canonical signaling pathway downstream of TGF-β1 involves the phosphorylation of Smad 2/3, which then dimerize with Smad4 and translocate to the nucleus to regulate gene transcription (<xref ref-type=\"bibr\" rid=\"r28\">28</xref>). In glomerular mesangial cells, TGF-β1-mediated Smad3 phosphorylation and subsequent promoter activation were dependent on PI3K activity (<xref ref-type=\"bibr\" rid=\"r37\">37</xref>). In NRK52E cells, TGF-β1 increased Smad3 phosphorylation within 60 min (<xref rid=\"f4\" ref-type=\"fig\">Fig. 4</xref>). In contrast to results observed in mesangial cells, TGF-β1-stimulated Smad3 phosphorylation was not diminished in the presence of inhibitors of PI3K or PKB/Akt (<xref rid=\"f4\" ref-type=\"fig\">Fig. 4</xref>), suggesting that TGF-β1-induced Smad3 phosphorylation occurs independently of the activity of this kinase pathway in NRK52E cells.</p><p>The role of PI3K→PKB/Akt signaling in TGF-β1-induced EMT in renal epithelial cells was then examined. As previously observed, exposure of NRK52E cells to TGF-β1 decreased ZO-1 levels in the tight junctions between cells and increased cytoskeletal vimentin expression (<xref rid=\"f5\" ref-type=\"fig\">Fig. 5</xref>, <italic>A</italic> and <italic>B</italic>, <italic>top</italic>). Coincubation of cells with LY-294002 or Akt inhibitor II significantly reduced TGF-β1-induced changes in epithelial (ZO-1) and mesenchymal (vimentin) marker expression (<xref rid=\"f5\" ref-type=\"fig\">Fig. 5</xref>, <italic>A</italic> and <italic>B</italic>, <italic>middle</italic> and <italic>bottom</italic>). These observations were supported by Western blot analysis, using changes in additional epithelial (E-cadherin) and mesenchymal (α-SMA) proteins as readouts of TGF-β1-induced EMT (<xref rid=\"f5\" ref-type=\"fig\">Fig. 5</xref>, <italic>C</italic> and <italic>D</italic>). Both TGF-β1-induced decreases in E-cadherin expression and increases in α-SMA expression were inhibited by LY-294002 and Akt inhibitor II. Together, these data suggest that TGF-β1-induced EMT in NRK52E renal tubular cells is a PI3K→PKB/Akt-dependent process.</p><p>EMT is thought to play a significant role in tubulointerstitial fibrosis in many cases of end-stage renal disease (<xref ref-type=\"bibr\" rid=\"r32\">32</xref>, <xref ref-type=\"bibr\" rid=\"r34\">34</xref>, <xref ref-type=\"bibr\" rid=\"r41\">41</xref>, <xref ref-type=\"bibr\" rid=\"r47\">47</xref>). We used the GK non-obese, nonhypertensive rat model of type 2 diabetes (<xref ref-type=\"bibr\" rid=\"r16\">16</xref>) to explore in vivo mechanisms of TGF-β1 signaling and EMT. These rats developed spontaneous type 2 diabetes at 11–12 wk of age with elevated random blood glucose evident (control Wistar, 8.6 ± 2.1 mM; Goto-Kakizaki, 13.1 ± 1.9 mM, <italic>P</italic> &lt; 0.02; Kattla et al., unpublished observations). As previously reported (<xref ref-type=\"bibr\" rid=\"r20\">20</xref>, <xref ref-type=\"bibr\" rid=\"r23\">23</xref>), GK diabetic rats displayed evidence of mild to moderate tubulointerstitial fibrosis at 9–10 mo of age compared with Wistar controls (<xref rid=\"f6\" ref-type=\"fig\">Fig. 6</xref>). Deposition of collagen was evident in the tubulointerstitial space in GK but not control Wistar rats, suggestive of renal fibrosis (<xref rid=\"f6\" ref-type=\"fig\">Fig. 6</xref>, <italic>A</italic> and <italic>B</italic>). Similarly, increased extracellular matrix production suggestive of increased glomerular basement membrane thickening was also evident in diabetic compared with control rat kidneys (<xref rid=\"f6\" ref-type=\"fig\">Fig. 6</xref>, <italic>C</italic> and <italic>D</italic>).</p><p>EMT is thought to be a significant component of renal fibrosis. We detected elevated levels of TGF-β1 mRNA in extracts from 9- to 10-mo-old diabetic GK but not control rat kidneys (data not shown). Protein lysates from kidneys of diabetic GK rats at 9–10 mo displayed elevated levels of phospho-Smad3 compared with controls, with more modest increases observed at 11–12 wk (<xref rid=\"f7\" ref-type=\"fig\">Fig. 7</xref>). Together, these data suggested that amplified TGF-β signaling was present in kidneys of diabetic rats at more advanced stages of disease. We hypothesized that elevated levels of TGF-β1 present in diabetic GK rats at 9–10 mo of age would have consequences for epithelial cell integrity in the renal tubules. Kidney sections from control Wistar and diabetic GK rats were prepared and stained using epithelial (E-cadherin) and mesenchymal (vimentin) cell markers. No changes in either E-cadherin or vimentin staining were detected in either control or hyperglycemic rat kidneys at 11–12 wk; i.e., at the time of onset of diabetes (data not shown). In contrast, 9- to 10-mo-old diabetic GK rats displayed patchy loss of E-cadherin staining in the kidney tubules with concomitant increases in vimentin levels detected compared with nondiabetic Wistar controls (<xref rid=\"f8\" ref-type=\"fig\">Fig. 8</xref>, <italic>A</italic> and <italic>B</italic>). Western blotting of whole kidney lysates also revealed reduced E-cadherin and increased vimentin levels in diabetic compared with control rat kidneys (<xref rid=\"f8\" ref-type=\"fig\">Fig. 8</xref>, <italic>C</italic> and <italic>D</italic>). Similar to the above-described findings, no significant changes in E-cadherin or vimentin staining were observed at the time of onset of diabetes (11–12 wk), suggesting that these effects occurred as a result of long-term hyperglycemia.</p><p>Our data suggested that TGF-β1-induced EMT in renal tubular cells in vitro required PI3K→PKB/Akt signal transduction (<xref rid=\"f5\" ref-type=\"fig\">Fig. 5</xref>). To examine whether elevated PKB/Akt activity was present in the fibrotic kidney in vivo, we performed Western blotting of kidney lysates from control Wistar or diabetic GK rats using antibodies against phospho-PKB/Akt and the phosphorylated (inactivated) form of its downstream target, GSK-3β. Elevated levels of both phosphoproteins were detected in GK compared with control Wistar rats at both age groups measured (<xref rid=\"f9\" ref-type=\"fig\">Fig. 9</xref>). Sections from both control Wistar and GK rats also revealed that increased levels of phospho-PKB/Akt and pGSK-3β were present in GK compared with control kidneys, with staining predominantly located in the kidney tubules (<xref rid=\"f10\" ref-type=\"fig\">Fig. 10</xref>). These data suggest that PKB/Akt activity is involved not only in TGF-β1-induced EMT in kidney epithelial cells in vitro but is elevated in the diabetic kidney disease in GK rats.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2815 prediction: method true label: other\n",
      "<sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" id=\"sec3\"><title>Results</title><sec id=\"sec3.1\"><title>Activation of Recombinant Human Factor V Molecules</title><p>To ascertain the function of the acidic region from the COOH terminus of factor Va heavy chain for cofactor activity, we constructed a molecule that is missing the entire amino acid region 680−709 [factor V<sup>Δ680−709</sup> (Figure <xref rid=\"fig1\" ref-type=\"fig\">1</xref>)]. We have also constructed a factor V molecule with the <sup>695</sup>DYDY<sup>698</sup> → AAAA mutations (factor V<sup>4A</sup>). The recombinant molecules were expressed in mammalian cells and purified to homogeneity as described previously (<xref ref-type=\"bibr\" rid=\"ref44\">44</xref>). Since factor V<sup>Δ680−709</sup> is missing Arg<sup>709</sup> that is the activating cleavage site for α-thrombin (Figure <xref rid=\"fig1\" ref-type=\"fig\">1</xref>), in the functional assays, the recombinant mutant molecule was activated with RVV-V activator. In contrast, factor V<sup>4A</sup> was activated with either RVV-V activator or α-thrombin. SDS−PAGE analyses followed by immunoblotting with specific monoclonal antibodies to the heavy and light chain of the cofactor demonstrate that the mutant recombinant proteins were intact, homogeneous, and migrated according to their expected molecular weights (Figure <xref rid=\"fig2\" ref-type=\"fig\">2</xref>). In addition, control experiments using a polyclonal antibody to factor V demonstrate that all factor V molecules were fully activated prior to use.</p><fig id=\"fig2\" position=\"float\"><label>Figure 2</label><caption><p>Electrophoretic analyses of wild-type factor V and recombinant factor V molecules. (A) Factor V<sup>Wt</sup> and factor V<sup>Δ680−709</sup> were activated with RVV-V activator. (B) Factor V<sup>Wt</sup> and factor V<sup>4A</sup> were activated with α-thrombin as described in <xref rid=\"sec2\"/> and analyzed by SDS−PAGE. After being transferred to a PVDF membrane, immunoreactive fragments were detected with monoclonal antibodies αHFVa<sub>HC</sub>17 (recognizing an epitope on the heavy chain of the cofactor between amino acid residues 307 and 506) and αHFVa<sub>HC</sub>9 (recognizing the light chain). At the right, the positions of the heavy and light chains of factor Va are shown.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0007\" id=\"gr3\" position=\"float\"/></fig></sec><sec id=\"sec3.2\"><title>Cofactor Function of Recombinant Human Factor Va Molecules</title><p>Recent work using recombinant factor Va molecules demonstrated that all cofactors lacking portions or the entire acidic COOH terminus of the heavy chain encompassing amino acid region 680−709 had <italic>K</italic><sub>Dapp</sub> values for membrane-bound factor Xa that are similar to the values for the bimolecular interaction found with the wild type or the intact plasma cofactor (<xref ref-type=\"bibr\" rid=\"ref37\">37</xref>). These values are similar to the <italic>K</italic><sub>Dapp</sub> obtained with truncated plasma-derived factor Va molecules for factor Xa (<xref ref-type=\"bibr\" rid=\"ref20\">20</xref>,<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>). Our studies with factor Va<sup>Δ680−709</sup> also demonstrate that the mutant cofactor has an affinity for plasma-derived factor Xa similar to that of the wild-type molecule (Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>A). Similar results were found for factor Va<sup>4A</sup> (not shown). Thus, elimination of the acidic COOH-terminal region of the factor Va heavy chain (amino acid residues 680−709) is of no consequence with respect to the affinity of the cofactor for factor Xa.</p><fig id=\"fig3\" position=\"float\"><label>Figure 3</label><caption><p>Raw data used for the determination of the parameters of prothrombinase complex assembly and function. (A) Determination of the affinity of recombinant factor Va molecules for plasma-derived factor Xa. Initial rates of α-thrombin generation were determined as described in <xref rid=\"sec2\"/>. Data for prothrombinase assembled with factor Va<sup>Wt</sup> are shown with filled squares (<italic>R</italic><sup>2</sup> = 0.979), while data for prothrombinase assembled with FVa<sup>Δ680−709</sup> are depicted with filled triangles (<italic>R</italic><sup>2</sup> = 0.978). Titrations were carried out to 20 nM factor Va; however, for graphical purposes, the data show the titration up to 10 nM cofactor. The solid lines represent a nonlinear regression fit of the data as detailed in <xref rid=\"sec2\"/> using Prizm and the model for one binding site. The apparent dissociation constant (<italic>K</italic><sub>Dapp</sub>) for each species was obtained from each titration performed at least in triplicate with at least two different preparations of recombinant proteins and is listed in the inset. (B) Determination of kinetic parameters of prothrombinase assembled with various factor Va species. Initial rates of α-thrombin generation were determined using the <italic>K</italic><sub>D</sub> for factor Xa found in panel A to determine the concentration of factor Va as described in <xref rid=\"sec2\"/> in the presence of PCPS vesicles and factor Xa. Data for prothrombinase assembled with two different concentrations of factor Va<sup>Wt</sup><sub>RVV</sub> are shown with empty triangles (10 nM, 95% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.97) and empty inverted triangles (20 nM, 97% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.97). Data for prothrombinase assembled with two different concentrations of factor Va<sup>Δ680−709</sup> are depicted with filled diamonds (10 nM, 96% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.98) and empty circles (20 nM, 98% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.99), while data for prothrombinase assembled with factor Va<sub>IIa</sub><sup>PLASMA</sup> are depicted with filled squares (10 nM, 98% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.97) and with factor Va<sub>RVV</sub><sup>PLASMA</sup> with empty diamonds (10 nM, 98% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.97). Data for prothrombinase assembled with factor Va<sub>RVV</sub><sup>4A</sup> are depicted with filled inverted triangles (10 nM, 96.5% factor Xa saturation; <italic>R</italic><sup>2</sup> = 0.98). The values of <italic>K</italic><sub>m</sub> and <italic>V</italic><sub>max</sub>/<italic>E</italic><sub>T</sub> (= <italic>k</italic><sub>cat</sub>) extracted directly from these graphs are listed in Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0008\" id=\"gr4\" position=\"float\"/></fig><p>Several studies have shown that plasma-derived or recombinant meizothrombin has increased activity (<italic>k</italic><sub>cat</sub>) toward chromogenic or fluorescent substrates used to assess α-thrombin activity (<xref ref-type=\"bibr\" rid=\"ref8\">8</xref>,<xref ref-type=\"bibr\" rid=\"ref25\">25</xref>,<xref ref-type=\"bibr\" rid=\"ref31\">31</xref>,<xref ref-type=\"bibr\" rid=\"ref32\">32</xref>,<xref ref-type=\"bibr\" rid=\"ref38\">38</xref>). This fact had been established previously for DAPA (<xref ref-type=\"bibr\" rid=\"ref8\">8</xref>,<xref ref-type=\"bibr\" rid=\"ref32\">32</xref>) and TAME (<xref ref-type=\"bibr\" rid=\"ref30\">30</xref>). One early study comparing the amidolytic activities between α-thrombin and meizothrombin using S-2238 demonstrated an ∼25% increase in the <italic>k</italic><sub>cat</sub> and an ∼18% increase in the second-order rate constant of meizothrombin for S-2238 as compared to the <italic>k</italic><sub>cat</sub> and the second-order rate constant of α-thrombin for the same substrate (<xref ref-type=\"bibr\" rid=\"ref31\">31</xref>), and the general consensus is that there are subtle but significant differences between the active sites of α-thrombin and meizothrombin (<xref ref-type=\"bibr\" rid=\"ref32\">32</xref>,<xref ref-type=\"bibr\" rid=\"ref33\">33</xref>). We evaluated the ability of all recombinant factor Va molecules to function as a cofactor for prothrombinase. A clotting assay using factor V-deficient plasma and an α-thrombin generation assay using Spectrozyme-TH and purified reagents were employed. In addition, we have followed the activation of prothrombin by gel electrophoresis. Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>B illustrates the fitted hyperbolic plots used to extract the kinetic constants obtained following plasma-derived prothrombin activation by prothrombinase assembled with all control or modified cofactor molecules, and summarized in Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>. Because we wanted to prevent the possibility that differences between prothrombinase assembled with factor Va<sup>Wt</sup> and prothrombinase assembled with factor Va<sup>Δ680−709</sup> or factor Va<sup>4A</sup> may be attributed to subtle differences in the <italic>K</italic><sub>Dapp</sub> of factor Va for factor Xa, which would result in a smaller amount of prothrombinase formed, all experiments described below were conducted under conditions in which more than 95% of factor Xa was saturated with factor Va. The amount of factor Xa saturation by each cofactor was calculated using the quadratic equation provided in the literature (<xref ref-type=\"bibr\" rid=\"ref50\">50</xref>) and the <italic>K</italic><sub>Dapp</sub> obtained from the data shown in Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>A. The percent saturation of factor Xa by each cofactor species and the correlation coefficient (<italic>R</italic><sup>2</sup>) of the fitted hyperbolic plots used to extract the values of <italic>V</italic><sub>max</sub> and <italic>K</italic><sub>m</sub> are provided in the legend of Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>B together with the actual reagent concentrations used in each experiment.</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1</label><caption><title>Characteristics of Various Factor Va Molecules Assembled into Prothrombinase<xref rid=\"tbl1-fn1\" ref-type=\"table-fn\">a</xref></title></caption><table frame=\"hsides\" rules=\"groups\" border=\"1\"><colgroup><col align=\"left\"/><col align=\"char\" char=\"±\"/><col align=\"left\"/><col align=\"char\" char=\"±\"/><col align=\"char\" char=\"±\"/><col align=\"char\" char=\"±\"/><col align=\"char\" char=\"±\"/><col align=\"left\"/><col align=\"char\" char=\"±\"/><col align=\"char\" char=\"±\"/></colgroup><thead><tr><th style=\"border:none;\" align=\"center\"> </th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>IIa</sub><sup>PLASMA</sup></th><th style=\"border:none;\" align=\"center\">FVa<sub>RVV</sub><sup>PLASMA</sup></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>IIa</sub><sup>Wt</sup></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>RVV</sub><sup>Wt</sup></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>RVV</sub><sup>Δ680−709</sup></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>IIa/CG</sub><xref rid=\"tbl1-fn4\" ref-type=\"table-fn\">d</xref></th><th style=\"border:none;\" align=\"center\">FVa<sub>IIa/NN</sub></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>IIa</sub><sup>4A</sup></th><th style=\"border:none;\" align=\"center\" char=\"±\">FVa<sub>RVV</sub><sup>4A</sup></th></tr></thead><tbody><tr><td style=\"border:none;\" align=\"left\">II Consumption<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref>[mol s<sup>−1</sup> (mol of fXa)<sup>−1</sup>]</td><td style=\"border:none;\" align=\"char\" char=\"±\">16.2 ± 1.8</td><td style=\"border:none;\" align=\"left\">ND<xref rid=\"tbl1-fn6\" ref-type=\"table-fn\">f</xref></td><td style=\"border:none;\" align=\"char\" char=\"±\">17.3 ± 1.2</td><td style=\"border:none;\" align=\"char\" char=\"±\">15.8 ± 1.1</td><td style=\"border:none;\" align=\"char\" char=\"±\">5.1 ± 0.54</td><td style=\"border:none;\" align=\"char\" char=\"±\">5.9 ± 0.6</td><td style=\"border:none;\" align=\"left\">ND<xref rid=\"tbl1-fn6\" ref-type=\"table-fn\">f</xref></td><td style=\"border:none;\" align=\"char\" char=\"±\">4.4 ± 0.42</td><td style=\"border:none;\" align=\"char\" char=\"±\">4.7 ± 0.4</td></tr><tr><td style=\"border:none;\" align=\"left\"><italic>K</italic><sub>m</sub> (μM)</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.1 ± 0.011</td><td style=\"border:none;\" align=\"left\">0.24 ± 0.04</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.1 ± 0.013</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.22 ± 0.04</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.24 ± 0.03</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.26 ± 0.03</td><td style=\"border:none;\" align=\"left\">0.15 ± 0.07</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.31 ± 0.1</td><td style=\"border:none;\" align=\"char\" char=\"±\">0.25 ± 0.03</td></tr><tr><td style=\"border:none;\" align=\"left\"><italic>k</italic><sub>cat</sub> (min<sup>−1</sup>)</td><td style=\"border:none;\" align=\"char\" char=\"±\">1715 ± 45</td><td style=\"border:none;\" align=\"left\">1754 ± 84</td><td style=\"border:none;\" align=\"char\" char=\"±\">1697 ± 48</td><td style=\"border:none;\" align=\"char\" char=\"±\">1721 ± 90</td><td style=\"border:none;\" align=\"char\" char=\"±\">2396 ± 71</td><td style=\"border:none;\" align=\"char\" char=\"±\">2122 ± 62</td><td style=\"border:none;\" align=\"left\">2066 ± 86</td><td style=\"border:none;\" align=\"char\" char=\"±\">2031 ± 81</td><td style=\"border:none;\" align=\"char\" char=\"±\">2018 ± 75</td></tr><tr><td style=\"border:none;\" align=\"left\">specific activity (units/mg)<xref rid=\"tbl1-fn3\" ref-type=\"table-fn\">c</xref></td><td style=\"border:none;\" align=\"char\" char=\"±\">3337 ± 480</td><td style=\"border:none;\" align=\"left\">2753 ± 267</td><td style=\"border:none;\" align=\"char\" char=\"±\">2926 ± 340</td><td style=\"border:none;\" align=\"char\" char=\"±\">3200 ± 550</td><td style=\"border:none;\" align=\"char\" char=\"±\">869 ± 192</td><td style=\"border:none;\" align=\"char\" char=\"±\">1280 ± 114</td><td style=\"border:none;\" align=\"left\">ND<xref rid=\"tbl1-fn5\" ref-type=\"table-fn\">e</xref><sup>,</sup><xref rid=\"tbl1-fn6\" ref-type=\"table-fn\">f</xref></td><td style=\"border:none;\" align=\"char\" char=\"±\">663 ± 210</td><td style=\"border:none;\" align=\"char\" char=\"±\">640 ± 120</td></tr></tbody></table><table-wrap-foot><fn id=\"tbl1-fn1\"><label>a</label><p>The rate of thrombin formation following activation of prothrombin by prothrombinase assembled with the various factor Va species was calculated as described in <xref rid=\"sec2\"/> with the knowledge of the dissociation constant of each factor Va species for factor Xa. In each case, more than 95% of factor Xa was saturated with factor Va. Some of the data in the table were extracted directly from the graphs shown in Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>B.</p></fn><fn id=\"tbl1-fn2\"><label>b</label><p>The rate of prothrombin consumption was determined following quantitative scanning densitometry of several gels stained with Coomassie Blue as described in <xref rid=\"sec2\"/>. Some of the gels used are shown in Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>.</p></fn><fn id=\"tbl1-fn3\"><label>c</label><p>All clotting activities were determined in a two-stage clotting assay following activation of factor V species by RVV-V activator or α-thrombin as described previously (<xref ref-type=\"bibr\" rid=\"ref39\">39</xref>,<xref ref-type=\"bibr\" rid=\"ref40\">40</xref>).</p></fn><fn id=\"tbl1-fn4\"><label>d</label><p>Plasma factor V was activated with α-thrombin and treated with cathepsin G (CG) as described previously (<xref ref-type=\"bibr\" rid=\"ref20\">20</xref>).</p></fn><fn id=\"tbl1-fn5\"><label>e</label><p>The clotting activity of the truncated cofactor is ∼50% of that of plasma-derived factor Va (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>).</p></fn><fn id=\"tbl1-fn6\"><label>f</label><p>Not determined in this study.</p></fn></table-wrap-foot></table-wrap><p>Factor Va<sub>RVV</sub><sup>Δ680−709</sup> had a 39.2% greater <italic>k</italic><sub>cat</sub> [Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>B (○)] and 73% lower clotting activity than factor Va<sub>RVV</sub><sup>Wt</sup> [Figure <xref rid=\"fig3\" ref-type=\"fig\">3</xref>B (▽)], while factor Va<sub>IIa/CG</sub> had a 23.7% greater <italic>k</italic><sub>cat</sub> and 62% lower clotting activity than factor Va<sub>IIa</sub><sup>PLASMA</sup> (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). In separate experiments, we have found that under similar experimental conditions, factor Va<sub>IIa/NN</sub> that has a 50% lower clotting activity than the plasma cofactor (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>) had a 20.5% increased <italic>k</italic><sub>cat</sub> compared to that of factor Va<sub>IIa</sub><sup>PLASMA</sup> (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). All these data are in complete agreement with earlier and recent findings (<xref ref-type=\"bibr\" rid=\"ref18\">18</xref>–<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>,<xref ref-type=\"bibr\" rid=\"ref37\">37</xref>) and demonstrate that elimination of the COOH-terminal region of the factor Va heavy chain produces a molecule with poor clotting activity, yet when the truncated cofactor is incorporated into prothrombinase, it produces an enzyme with increased catalytic efficiency for prothrombin activation as assessed using purified components and under conditions in which the majority of membrane-bound cofactor is bound to factor Xa. Overall, the data listed in Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref> also demonstrate that the plasma and wild-type molecule behave similarly following activation by either RVV-V activator or α-thrombin with respect to prothrombin activation. These data are in complete accord with earlier findings (<xref ref-type=\"bibr\" rid=\"ref20\">20</xref>,<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>) and demonstrate that plasma and wild-type factor Va are interchangeable. As a consequence, throughout this work control experiments were conducted, and their results are shown with only one of the two molecules as indicated.</p><fig id=\"fig4\" position=\"float\"><label>Figure 4</label><caption><p>Analysis of the activation of plasma-derived prothrombin by prothrombinase. Plasma-derived prothrombin (1.4 μM) was incubated in different mixtures with PCPS vesicles (20 μM), and prothrombinase assembled with either wild-type factor Va [(A) 10 nM] or factor Va<sup>Δ680−709</sup> [(B) 10 nM] as described in <xref rid=\"sec2\"/>, and the reaction was started by the addition of factor Xa. (C) Prothrombinase assembled with factor Va<sup>Δ680−709</sup> in the presence of 20 μM DYDYQ (same conditions as in panel B). (D) Prothrombinase assembled with plasma-derived factor Va (10 nM). (E) Prothrombinase assembled with factor Va<sup>4A</sup> (10 nM). At selected time intervals, aliquots of the reaction mixtures were withdrawn and treated as described in <xref rid=\"sec2\"/>. Lane M contained the molecular weight markers (from top to bottom): 98000, 64000, 50000, 36000, and 22000. Lanes 1−19 represent samples from the reaction mixture before (0 min) the addition of factor Xa and 20 s, 40 s, 60 s, 80 s, 100 s, 120 s, 140 s, 160 s, 180 s, 200 s, 220 s, 240 s, 5 min, 6 min, 10 min, 20 min, 30 min, and 60 min, respectively, following the addition of factor Xa. The prothrombin-derived fragments are shown as follows: II, prothrombin (amino acid residues 1−579); prethrombin-1 (amino acid residues 156−579); F1·2-A, fragment 1·2-A chain (amino acid residues 1−320); F1·2, fragment 1·2 (amino acid residues 1−271); P2, prethrombin-2 (amino acid residues 272−579); P2′, prethrombin-2 cleaved at Arg<sup>284</sup>; B, B chain of α-thrombin (amino acid residues 321−579).</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0001\" id=\"gr5\" position=\"float\"/></fig><p>We next evaluated the ability of prothrombinase assembled with the truncated molecules to activate prothrombin by gel electrophoresis. Under the conditions that were employed, prothrombinase assembled with factor Va<sub>RVV</sub><sup>Wt</sup> activates prothrombin quickly, with meizothrombin as a short-lived intermediate as demonstrated by the brief half-life of fragment 1·2-A (Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>A). Scanning densitometry demonstrated a peak of meizothrombin early in the reaction at 80 s, and no meizothrombin was detected following 180 s (Figure <xref rid=\"fig5\" ref-type=\"fig\">5</xref>A). In contrast, in the presence of prothrombinase assembled with factor Va<sup>Δ680−709</sup>, prothrombin is consumed with a rate that is approximately 3-fold slower than the rate of consumption of prothrombin by prothrombinase assembled with the wild-type molecule (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>), with persistence of meizothrombin as indicated by the lingering of fragment 1·2-A even at the late time points of the reaction (Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>B). Scanning densitometry demonstrated a peak of meizothrombin late in the reaction at 250 s. Meizothrombin was more concentrated and persisted for up to 6 min (Figure <xref rid=\"fig5\" ref-type=\"fig\">5</xref>B). In addition, the rate of appearance of the B chain is also delayed (by approximately 3-fold) when prothrombin is activated by prothrombinase assembled with factor Va<sup>Δ680−709</sup>, compared with the rate of appearance of the B chain obtained following incubation of prothrombin with prothrombinase assembled with factor Va<sub>RVV</sub><sup>Wt</sup>. Similar results were obtained with factor Va<sub>IIa/CG</sub> (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). In contrast, in the presence of DYDYQ, no meizothrombin is observed following prothrombin activation by prothrombinase assembled with factor Va<sup>Δ680−709</sup>, and α-thrombin is formed through the alternate pathway characterized by initial cleavage at Arg<sup>271</sup> and formation of prethrombin 2 as the intermediate (Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>C). Overall, the data demonstrate that elimination of amino acid region 680−709 from factor Va results in a cofactor molecule that when incorporated into prothrombinase produces an enzyme responsible for both delayed prothrombin activation and persistence of meizothrombin during activation of prothrombin.</p><fig id=\"fig5\" position=\"float\"><label>Figure 5</label><caption><p>Reaction profiles for the activation of prothrombin by prothrombinase. Progress curves for products and reactants for the activation of prothrombin by prothrombinase assembled with factor Va<sup>Wt</sup> (A) or factor Va<sup>Δ680−709</sup> (B) were obtained by quantitative scanning densitometry of gels shown in panels A and B of Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>, as described in <xref rid=\"sec2\"/>. The graphs illustrate the disappearance of prothrombin (◼), the transient formation of meizothrombin (○), and the accumulation of the B chain of α-thrombin (●). There is a 3-fold difference in the <italic>x</italic>-axis between the two panels, because prothrombinase assembled with factor Va<sup>Δ680−709</sup> consumes prothrombin with an ∼3-fold slower rate than prothrombinase assembled with factor Va<sup>Wt</sup>. The lines for the disappearance of prothrombin were drawn according to the equation of a one-phase exponential decay (for factor Va<sup>Wt</sup>, <italic>R</italic><sup>2</sup> = 0.985; and for factor Va<sup>Δ680−709</sup>, <italic>R</italic><sup>2</sup> = 0.968). The lines depicting the formation of meizothrombin and the accumulation of the B chain of α-thrombin were arbitrarily drawn. Additional data points extending to incubation for 1 h (shown in Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>) have been omitted for clarity.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0002\" id=\"gr6\" position=\"float\"/></fig><p>In preliminary experiments using several preparations of recombinant proteins, we observed that while the truncated factor Va molecules are impaired in their clotting activity, factor Va<sup>4A</sup> is also deficient in its clotting activity. To understand the properties of this molecule and the effect of the mutations on cofactor activity, we used the same preparation of recombinant protein to perform three different experiments. Factor V<sup>4A</sup> was first activated with α-thrombin, and the solution was split into three separate samples. One sample was used for assessment of clotting activity, one sample was used to measure the kinetic parameters of prothrombinase assembled with a saturating concentration of factor Va<sub>IIa</sub><sup>4A</sup>, and the third sample was used for analysis of prothrombin activation by gel electrophoresis. The results reveal that while factor Va<sub>IIa</sub><sup>4A</sup> is severely impaired in its clotting activity (∼22% of that of factor Va<sub>IIa</sub><sup>Wt</sup>), prothrombinase assembled with the mutant molecule shows a 19.7% increased <italic>k</italic><sub>cat</sub> (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). Gel electrophoresis followed by scanning densitometry analysis demonstrated that prothrombinase assembled with factor Va<sub>IIa</sub><sup>4A</sup> cleaves prothrombin with a rate that is approximately 3.8-fold slower than the rate of cleavage of prothrombin assembled with factor Va<sub>IIa</sub><sup>Wt</sup> or factor Va<sub>IIa</sub><sup>PLASMA</sup> (Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>D,E and Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). Scanning densitometry of several gels studying prothrombin activation by prothrombinase assembled with factor Va<sub>IIa</sub><sup>4A</sup> revealed a peak of meizothrombin at approximately 240 s with significant amounts of meizothrombin remaining for up to 10 min (not shown). The data also showed more meizothrombin formed and delayed B chain formation by prothrombinase assembled with factor Va<sup>4A</sup> as compared to the levels of meizothrombin produced by prothrombinase assembled with factor Va<sup>Wt</sup>. Similar data were obtained with factor Va<sub>RVV</sub><sup>4A</sup> (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). These findings strongly support our previous conclusion suggesting that amino acid sequence 695−699 regulates meizothrombin formation by prothrombinase (<xref ref-type=\"bibr\" rid=\"ref35\">35</xref>,<xref ref-type=\"bibr\" rid=\"ref36\">36</xref>). These data also demonstrate that, in the presence of prothrombinase assembled with factor Va<sup>4A</sup>, the excess meizothrombin formed as assessed functionally by clotting assays can compensate for the absence of α-thrombin in the assay using purified reagents because the increased amidolytic activity of meizothrombin toward the chromogenic substrate is read as α-thrombin activity.</p></sec><sec id=\"sec3.3\"><title>Role of the Acidic COOH-Terminal Portion of the Factor Va Heavy Chain: A Potential Mechanism</title><p>The data obtained thus far studying plasma-derived prothrombin activation by prothrombinase assembled with the mutant cofactor molecules suggest that meizothrombin formation is not greatly delayed but rather that meizothrombin accumulates to higher and more persistent levels and lingers throughout the time course (Figure <xref rid=\"fig5\" ref-type=\"fig\">5</xref>B). The appearance of the B chain of α-thrombin is also delayed when prothrombin is activated by prothrombinase assembled with the mutant cofactor (Figures <xref rid=\"fig4\" ref-type=\"fig\">4</xref> and <xref rid=\"fig5\" ref-type=\"fig\">5</xref>). These data indicate that one or both of the two prothrombin activation cleavages are impaired (affected) when prothrombinase is assembled with a cofactor molecule missing the acidic COOH terminus of the heavy chain or with factor Va<sup>4A</sup>. To verify if under the conditions employed, cleavage at Arg<sup>320</sup> in prothrombin is delayed when prothrombinase is assembled with factor Va<sub>IIa</sub><sup>4A</sup> as compared to cleavage of prothrombin at the same site by prothrombinase assembled with factor Va<sub>IIa</sub><sup>Wt</sup>, we studied prothrombin cleavage during the first 90 s of the reaction. Quantitative scanning densitometry of the gels revealed no differences in the rate of plasma-derived prothrombin consumption by prothrombinase assembled with either factor Va<sub>IIa</sub><sup>4A</sup> or factor Va<sup>Wt</sup> during this period of time (not shown). These data combined with the data shown in Figures <xref rid=\"fig4\" ref-type=\"fig\">4</xref> and <xref rid=\"fig5\" ref-type=\"fig\">5</xref> illustrating delayed prothrombin consumption which is accompanied by an increase in the concentration of meizothrombin (up to ∼20%) in the case of the activation of prothrombin by prothrombinase assembled with factor Va<sup>Δ680−709</sup> as compared to the concentration of meizothrombin produced following activation of prothrombin by prothrombinase assembled with factor Va<sup>Wt</sup> suggest that delayed initial cleavage at Arg<sup>320</sup> in plasma-derived prothrombin is not a direct consequence of the mutation in factor Va, but rather the result of product inhibition by the accumulating meizothrombin. This inhibition is observed only late in the reaction when sufficient meizothrombin accumulates, as clearly demonstrated by our combined results.</p><p>It is important to note that during the course of the assay using plasma-derived prothrombin we are measuring the sum of meizothrombin and α-thrombin. A more rigorous test to verify which cleavage in prothrombin is affected by the deletion at the COOH terminus of factor Va heavy chain involves the study of the activation of a prothrombin molecule that cannot be cleaved at Arg<sup>271</sup> such as rMZ-II (<xref ref-type=\"bibr\" rid=\"ref41\">41</xref>). In addition, while cleavage of rMZ-II can be followed by gel electrophoresis, generation of rMZ-IIa can be also assessed with the chromogenic substrate used to assess α-thrombin formation. We thus followed activation of rMZ-II by prothrombinase assembled with either factor Va<sub>RVV</sub><sup>Δ680−709</sup> or factor Va<sub>RVV</sub><sup>Wt</sup> by gel electrophoresis (Figure <xref rid=\"fig6\" ref-type=\"fig\">6</xref>A) and activity assays using Spectrozyme-TH (Figure <xref rid=\"fig6\" ref-type=\"fig\">6</xref>B). Both sets of data demonstrate that there are no significant differences between the rates of activation of rMZ-II by either enzyme. The rates of rMZ-II consumption were similar as determined by scanning densitometry of the gels depicted in Figure <xref rid=\"fig6\" ref-type=\"fig\">6</xref>A, and the profiles of both titrations are coincident [similar <italic>V</italic><sub>max</sub> values and similar <italic>K</italic><sub>m</sub> values of 91 ± 9 and 121 ± 18 μM (Figure <xref rid=\"fig6\" ref-type=\"fig\">6</xref>B)]. Comparable results were found in three independent measurements using two different preparations of recombinant factor Va molecules and two separate preparations of rMZ-II. The data confirm our overall findings with plasma-derived prothrombin and demonstrate that prothrombinase-mediated cleavage at Arg<sup>320</sup> in prothrombin is not affected by the COOH-terminal portion of the factor Va heavy chain.</p><fig id=\"fig6\" position=\"float\"><label>Figure 6</label><caption><p>Analysis of the activation of rMZ-II. (A) Gel electrophoretic analyses. rMZ-II (1.4 μM) was incubated in different mixtures with PCPS vesicles (20 μM), DAPA (3 μM), and factor Va<sup>Wt</sup> (left panel, 20 nM) or factor Va<sup>Δ680−709</sup> (right panel, 20 nM). The reaction was started by the addition of factor Xa, and the samples were treated as detailed in <xref rid=\"sec2\"/>. Lanes 1−9 contained samples of the reaction mixture following incubation of prothrombinase with rMZ-II before (lane 1) or following incubation for 0.5, 1, 2.5, 4, 6, 10, 20, and 30 min with factor Xa, respectively. Positions of prothrombin-derived fragments are indicated at the right as detailed in the legend of Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>. The factor Va species used for the reconstitution of prothrombinase are shown under each panel. (B) Kinetic analyses. Initial rates of α-thrombin generation were determined as described in <xref rid=\"sec2\"/>, and the data were plotted according to the Michaelis−Menten equation. Data for activation of rMZ-II by prothrombinase assembled with factor Va<sup>Wt</sup> are shown with filled squares (<italic>R</italic><sup>2</sup> = 0.98), while data for activation of rMZ-II by prothrombinase assembled with factor Va<sup>Δ680−709</sup> are depicted with filled triangles (<italic>R</italic><sup>2</sup> = 0.96). Kinetic constants reported in the text were extracted directly from the graphs.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0003\" id=\"gr7\" position=\"float\"/></fig><fig id=\"fig7\" position=\"float\"><label>Figure 7</label><caption><p>Gel electrophoretic analyses for cleavage of FPR-meizothrombin. FPR-meizothrombin (1.4 μM) was incubated in different mixtures with PCPS vesicles (20 μM) and factor Va as described in the legend of Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>. The reaction and the samples were further treated, scanned, and quantified as detailed in <xref rid=\"sec2\"/>: (A) control, no factor Va [13.4 mol of FPR-meizo consumed s<sup>−1</sup> (mol of factor Xa)<sup>−1</sup>], (B) factor Va<sub>RVV</sub><sup>PLASMA</sup> [42.6 mol of FPR-meizo consumed s<sup>−1</sup> (mol of factor Xa)<sup>−1</sup>], (C) factor Va<sub>RVV</sub><sup>Δ680−709</sup> [8.4 mol of FPR-meizo consumed s<sup>−1</sup> (mol of factor Xa)<sup>−1</sup>], and (D) factor Va<sub>RVV</sub><sup>4A</sup> [14.6 mol of FPR-meizo consumed s<sup>−1</sup> (mol of factor Xa)<sup>−1</sup>]. Lane M contained the molecular weight markers (from top to bottom): 50000, 36000, and 22000. Lanes 1−19 contained samples from the reaction mixture before (0 min) the addition of factor Xa and 20 s, 40 s, 60 s, 80 s, 100 s, 120 s, 140 s, 160 s, 180 s, 200 s, 220 s, 240 s, 5 min, 6 min, 10 min, 20 min, 30 min, and 60 min, respectively, following the addition of factor Xa. The prothrombin-derived fragments are shown as detailed in the legend of Figure <xref rid=\"fig4\" ref-type=\"fig\">4</xref>. The factor Va species used for the reconstitution of prothrombinase are also shown under each panel.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bi-2008-00593k_0004\" id=\"gr8\" position=\"float\"/></fig><p>Analysis of the results obtained thus far suggests that cleavage of meizothrombin at Arg<sup>271</sup> by prothrombinase assembled with factor Va<sub>RVV</sub><sup>Δ680−709</sup> may be slower than cleavage of meizothrombin at the same site by prothrombinase assembled with factor Va<sub>RVV</sub><sup>Wt</sup>, resulting in slower conversion of meizothrombin to α-thrombin. We next compared the rate of cleavage of FPR-meizothrombin by prothrombinase assembled with various factor Va molecules (Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref>). The data demonstrate a delay for cleavage of FPR-meizothrombin at Arg<sup>271</sup> by prothrombinase assembled with factor Va<sub>RVV</sub><sup>Δ680−709</sup> (panel C) or factor Va<sub>RVV</sub><sup>4A</sup> (panel D) as compared to the same reaction catalyzed by prothrombinase assembled with factor Va<sub>RVV</sub><sup>PLASMA</sup> (panel B) or factor Va<sub>RVV</sub><sup>Wt</sup> (not shown). A direct comparison between the rates of cleavage of FPR-meizothrombin by membrane-bound factor Xa alone (Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref>A) or by prothrombinase assembled with either factor Va<sub>RVV</sub><sup>Δ680−709</sup> (Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref>C) or factor Va<sub>RVV</sub><sup>4A</sup> (Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref>D) does not show any significant differences. These data imply that following incorporation of factor Va into prothrombinase it is the acidic COOH-terminal hirudin-like portion of the factor Va heavy chain and in particular amino acid sequence <sup>695</sup>DYDY<sup>698</sup> that is responsible for the increase in the rate of cleavage of meizothrombin at Arg<sup>271</sup>. Quantitative scanning densitometry of fragment 1·2-A present on the gels shown in Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref> demonstrated an ∼3−5-fold delay in cleavage of FPR-meizothrombin at Arg<sup>271</sup> by either membrane-bound factor Xa alone or prothrombinase assembled with the recombinant mutant factor Va molecules, compared to cleavage at Arg<sup>271</sup> by prothrombinase assembled with the plasma-derived cofactor (Figure <xref rid=\"fig7\" ref-type=\"fig\">7</xref>B) or factor Va<sup>Wt</sup>. These data are in complete agreement with all the findings presented herein and demonstrate that the lingering of meizothrombin during plasma-derived prothrombin activation by prothrombinase assembled with factor Va<sub>RVV</sub><sup>Δ680−709</sup> or factor Va<sup>4A</sup> is the result of impaired cleavage of the molecule at Arg<sup>271</sup>. Overall, the findings suggest that amino acid sequence <sup>695</sup>DYDY<sup>698</sup> from the COOH-terminal portion of the factor Va heavy chain regulates meizothrombin concentration (processing) during activation of prothrombin by prothrombinase.</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2827 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" id=\"Sec6\" sec-type=\"results\"><title>Results</title><p>The described technique was used for 15 patients (all female) with a mean age of 57 years (range = 43–75 years). Ultrasonic energy was applied for an average of 2 min (range = 45 s–6.5 min). The mean aspiration volume was 125 ml (range = 25–200 ml). No hematoma, seroma, infection, skin necrosis, contour irregularity, or other complications were encountered in this series. The preoperative status (degree of obliquity of the cervicomental angle) and the postoperative results at 6 months, as graded by the panel, are given in Table <xref rid=\"Tab1\" ref-type=\"table\">1</xref>. After treatment significant improvement of the cervicomental angle was observed (Figs. <xref rid=\"Fig2\" ref-type=\"fig\">2</xref>–<xref rid=\"Fig5\" ref-type=\"fig\">5</xref>).<fig id=\"Fig2\"><label>Fig. 2</label><caption><p>Preoperative (<italic>left</italic>) grade III and postoperative (<italic>right</italic>) grade I–II views of patient with cervicomental angle deformity</p></caption><graphic position=\"anchor\" xlink:href=\"266_2008_9215_Fig2_HTML\" id=\"MO2\"/></fig><fig id=\"Fig3\"><label>Fig. 3</label><caption><p>Preoperative (<italic>left</italic>) grade III and postoperative (<italic>right</italic>) grade I–II views of patient with cervicomental angle deformity</p></caption><graphic position=\"anchor\" xlink:href=\"266_2008_9215_Fig3_HTML\" id=\"MO3\"/></fig><fig id=\"Fig4\"><label>Fig. 4</label><caption><p>Preoperative (<italic>left</italic>) grade II–III and postoperative (<italic>right</italic>) grade I views of patient with cervicomental angle deformity</p></caption><graphic position=\"anchor\" xlink:href=\"266_2008_9215_Fig4_HTML\" id=\"MO4\"/></fig><fig id=\"Fig5\"><label>Fig. 5</label><caption><p>Skin contraction in a 73-year-old patient. Preoperative (<italic>left</italic>) grade IV and postoperative (<italic>right</italic>) grade III views of patient with cervicomental angle deformity</p></caption><graphic position=\"anchor\" xlink:href=\"266_2008_9215_Fig5_HTML\" id=\"MO5\"/></fig></p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2842 prediction: method true label: other\n",
      "<sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" id=\"sec3\"><title>RESULT AND DISCUSSION</title><sec id=\"sec3.1\"><title>Chemistry</title><p>Our synthesis of [<sup>18</sup>F]FBABM builds on the approach reported by Toyokuni et al. (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>) (Scheme <xref rid=\"sch1\" ref-type=\"scheme\">1</xref>). Briefly, triflate <bold>1</bold> was heated with dried [<sup>18</sup>F]KF in anhydrous acetonitrile in the presence of Kryptofix[2,2,2] to generate [<sup>18</sup>F]FBA <bold>2</bold>, followed by the addition of maleimide <bold>3</bold> in methanol and purification by HPLC. Their two-step/two-pot synthesis of [<sup>18</sup>F]FBABM was used to label the thiol-containing tripeptide glutathione (GSH) and a 5′-thiol-functionalized oligodeoxynucleotide (5′-S-ODN). This synthesis started with the generation of <bold>2</bold> followed by a C18 SepPak purification. After coupling to the heterobifunctional linker <italic>N</italic>-[4-(aminoxy)butyl]maleimide <bold>3</bold> using the oxime O-ether formation reaction and HPLC purification, [<sup>18</sup>F]FBABM was formulated in acetonitrile and successfully labeled GSH or 5′-S-ODN. This method, however, has disadvantages for routine protein labeling. First, the synthesis of [<sup>18</sup>F]FBABM was labor-intensive, with a two-pot setup and a C18 SepPak purification step in between. Second, although the labeling of GSH gave good yield (70% decay corrected), the yield in labeling larger biomolecules such as 5′-S-ODN (5% decay corrected) was low. The concentrations of the GSH or the 5′-S-ODN in these reactions were 6 mM and 0.1 mM, respectively. The radiochemical yield of radiolabeled peptides/proteins or other biomolecules via prosthetic groups typically decreases as the concentration of the biomolecule decreases. For an even larger molecule such as annexin V-128, which is usually obtained in ∼30 μM concentration, the radiochemical yield might be even lower than 5%. Indeed, our initial attempts to label annexin V-128 with [<sup>18</sup>F]FBABM synthesized by closely following the literature procedure (<xref ref-type=\"bibr\" rid=\"ref12\">12</xref>) gave no yield. In addition to the slow rate of reaction associated with dilute conditions, the presence of nonradioactive maleimide-containing impurities, which would compete with [<sup>18</sup>F]FBABM for the available thiol groups, could be an important factor that results in the dramatic decrease in yield. In the Toyokuni report, there was little information on the chemical purity of the [<sup>18</sup>F]FBABM produced. We focused our investigation on simplification of the radiosynthesis and purification of the [<sup>18</sup>F]FBABM.</p><fig id=\"sch1\" position=\"float\"><label>Scheme 1</label><caption><title>Synthesis of <bold>[</bold><sup><bold>18</bold></sup><bold>F]FBABM</bold></title></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bc-2008-00164d_0002\" id=\"gs1\" position=\"float\"/></fig><p>We found that the C18 SepPak purification was not required for [<sup>18</sup>F]FBABM synthesis; the two-step reactions proceed smoothly in one pot. Starting with ∼850 mCi [<sup>18</sup>F]fluoride, 110 ± 14 mCi (<italic>n</italic> = 4) of purified [<sup>18</sup>F]FBABM was obtained in ∼92 min (23 ± 4% decay corrected radiochemical yield) with &gt;99% radiochemical purity and a specific activity of ∼6000 Ci/mmol. This yield is comparable to that from the two-pot procedure (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>), but with a process that can be automated. Although theoretically, [<sup>18</sup>F]FBABM could be formed as both <italic>E</italic>- and <italic>Z</italic>-isomers, we observed only one isomer (tentatively assigned to possess the <italic>E</italic>-configuration) (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>) under our reaction condition. This was confirmed by coeluting with the cold standard of FBABM, whose NMR spectrum revealed only one isomer, on HPLC. During the preparation of this manuscript, Berndt et al. reported a one-pot method to prepare a similar thiol-reactive labeling agent <italic>N</italic>-[6-(4-[<sup>18</sup>F]fluorobenzylidene)aminooxyhexyl]maleimide ([<sup>18</sup>F]FBAM) with similar yield to ours (<xref ref-type=\"bibr\" rid=\"ref23\">23</xref>). Their method, however, includes an extra prepurification step using solid-phase extraction on LiChrolut RP18 cartridge before semipreparative HPLC purification.</p><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bc-2008-00164d_0003\" id=\"gr100\" position=\"float\"/><p>The synthesis of [<sup>18</sup>F]FBA <bold>2</bold> from the triflate precursor <bold>1</bold> was traditionally carried out by heating the dried [<sup>18</sup>F]KF/Kryptofix[2,2,2] in a high-boiling polar aprotic solvent such as DMSO (<xref ref-type=\"bibr\" rid=\"ref24\">24</xref>,<xref ref-type=\"bibr\" rid=\"ref25\">25</xref>). We chose the lower boiling acetonitrile because it could be easily removed by evaporation. Compared with DMSO, the use of acetonitrile did not compromise the yield of [<sup>18</sup>F]FBA (Table <xref rid=\"tbl1\" ref-type=\"table\">1</xref>). Similar yields were obtained while the same amounts of precursors were used. A smaller amount of precursors resulted in decreased yields of <bold>2</bold>. Using 4 mg triflate precursor struck a balance between reasonably high yield and acceptably lower quantity of material introduced into the system. Compared to the Toyokuni report (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>), the yield of the coupling reaction dropped from ∼70% to ∼40% (calculated from the yield of the first step and the overall yield), but the advantage of easier production of this radiotracer outweighed this reduction of yield. It is important to note that, because FBA is a volatile compound at elevated temperature, it is crucial to cool the reaction mixture to r.t. before opening the reaction vessel to add the coupling precursor.</p><table-wrap id=\"tbl1\" position=\"float\"><label>Table 1</label><caption><title>Yield of FBA (<bold>2</bold>) under Different Reaction Conditions<xref rid=\"tbl1-fn1\" ref-type=\"table-fn\">a</xref></title></caption><table frame=\"hsides\" rules=\"groups\" border=\"1\"><colgroup><col align=\"left\"/><col align=\"char\" char=\".\"/><col align=\"char\" char=\".\"/><col align=\"char\" char=\".\"/></colgroup><thead valign=\"middle\"><tr><th style=\"border:none;\" align=\"center\">solvent</th><th style=\"border:none;\" align=\"center\" char=\".\">amt. of precursor <bold>1</bold> (mg)</th><th style=\"border:none;\" align=\"center\" char=\".\">temperature (°C)</th><th style=\"border:none;\" align=\"center\" char=\".\">yield (%)<xref rid=\"tbl1-fn3\" ref-type=\"table-fn\">c</xref></th></tr></thead><tbody><tr><td style=\"border:none;\" align=\"left\">DMSO</td><td style=\"border:none;\" align=\"char\" char=\".\">8</td><td style=\"border:none;\" align=\"char\" char=\".\">100</td><td style=\"border:none;\" align=\"char\" char=\".\">90</td></tr><tr><td style=\"border:none;\" align=\"left\">THF</td><td style=\"border:none;\" align=\"char\" char=\".\">4</td><td style=\"border:none;\" align=\"char\" char=\".\">80<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref></td><td style=\"border:none;\" align=\"char\" char=\".\">0</td></tr><tr><td style=\"border:none;\" align=\"left\">CH<sub>3</sub>CN</td><td style=\"border:none;\" align=\"char\" char=\".\">2</td><td style=\"border:none;\" align=\"char\" char=\".\">100<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref></td><td style=\"border:none;\" align=\"char\" char=\".\">10</td></tr><tr><td style=\"border:none;\" align=\"left\">CH<sub>3</sub>CN</td><td style=\"border:none;\" align=\"char\" char=\".\">3</td><td style=\"border:none;\" align=\"char\" char=\".\">100<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref></td><td style=\"border:none;\" align=\"char\" char=\".\">20</td></tr><tr><td style=\"border:none;\" align=\"left\">CH<sub>3</sub>CN</td><td style=\"border:none;\" align=\"char\" char=\".\">4</td><td style=\"border:none;\" align=\"char\" char=\".\">100<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref></td><td style=\"border:none;\" align=\"char\" char=\".\">75</td></tr><tr><td style=\"border:none;\" align=\"left\">CH<sub>3</sub>CN</td><td style=\"border:none;\" align=\"char\" char=\".\">8</td><td style=\"border:none;\" align=\"char\" char=\".\">100<xref rid=\"tbl1-fn2\" ref-type=\"table-fn\">b</xref></td><td style=\"border:none;\" align=\"char\" char=\".\">90</td></tr></tbody></table><table-wrap-foot><fn id=\"tbl1-fn1\"><label>a</label><p>All reactions were carried out by heating the precursor in 1 mL solvent for 15 min in a closed reaction vessel in a GE Tracerlab FX<sub>FN</sub> automated synthesis module.</p></fn><fn id=\"tbl1-fn2\"><label>b</label><p>Heated under helium pressure (200 mbar).</p></fn><fn id=\"tbl1-fn3\"><label>c</label><p>Determined from radio-HPLC chromatograms of the crude product. Data represent an average of 3 runs.</p></fn></table-wrap-foot></table-wrap><p>The critical factor for optimal protein labeling with [<sup>18</sup>F]FBABM is the chemical purity of the product. We suspected that a maleimide side product was present that would compete with [<sup>18</sup>F]FBABM for reaction with thiol groups. HPLC-MS was used to identify this major contaminant as <italic>N</italic>-[4-[(4-dimethylaminobenzylidene)aminooxy]butyl]maleimide (<bold>4</bold>) (estimated &gt;100 μg from a typical run described above) derived from the 4-dimethylaminobenzaldehyde formed inevitably during the initial nucleophilic fluorination step (Scheme <xref rid=\"sch2\" ref-type=\"scheme\">2</xref>). In a typical run, about 300 μg 4-dimethylaminobenzaldehyde was detected after this first step. The identity and quantity was determined by HPLC-MS analysis, in comparison to an authentic sample. Our initial attempt to label annexin V-128 with [<sup>18</sup>F]FBABM containing <bold>4</bold> gave no yield, presumably due to a competition for the single thiol group by the vast molar excess of the alternative maleimide contaminant <bold>4</bold>. In a typical labeling reaction, the molar ratio of FBABM to annexin V-128 (generally 0.6 mg was used) was approximately 1:1, whereas that of <bold>4</bold> was greater than 20:1. Attempts to remove the undesired 4-dimethylaminobenzaldehyde after the first step with a C18 SepPak were unsuccessful. The removal of <bold>4</bold> from the final product [<sup>18</sup>F]FBABM proved to be challenging. The calculated log <italic>P</italic> values for FBABM and <bold>4</bold> are 1.91 and 1.85, respectively (Molinspiration program: <uri xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"http://www.molinspiration.com/cgi-bin/properties\">http://www.molinspiration.com/cgi-bin/properties</uri>). The two compounds coeluted using the HPLC method from the literature (<xref ref-type=\"bibr\" rid=\"ref21\">21</xref>). We developed a useful separation by using Phenomenex Prodigy ODS(3) 5 μm 250 × 10 mm column. At 50 °C and 45% ethanol/water as the eluent at 3 mL/min, <bold>4</bold> was completely removed from [<sup>18</sup>F]FBABM (impurity <italic>t</italic><sub>R</sub>(<bold>4</bold>) ∼ 26−28 min, <italic>t</italic><sub>R</sub>(FBABM) ∼ 34−38 min). A typical semipreparative HPLC chromatogram is shown in Figure <xref rid=\"fig1\" ref-type=\"fig\">1</xref>. No detectable amount of <bold>4</bold> was present in the purified [<sup>18</sup>F]FBABM product. We realize that, in this isocratic HPLC method, although a complete separation of impurity <bold>4</bold> was achieved, the long retention time of the [<sup>18</sup>F]FBABM is disadvantageous for routine production. We are currently investigating alternative purification method with gradient HPLC.</p><fig id=\"sch2\" position=\"float\"><label>Scheme 2</label><caption><title>Formation of <bold>4</bold></title></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bc-2008-00164d_0004\" id=\"gs2\" position=\"float\"/></fig><fig id=\"fig1\" position=\"float\"><label>Figure 1</label><caption><p>A typical semipreparative HPLC chromatogram for [<sup>18</sup>F]FBABM purification. (Phenomenex Prodigy ODS-3, 250 × 10 mm, 5 μm, eluted with 45% ethanol/water v:v at 3 mL/min and 50 °C).</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bc-2008-00164d_0005\" id=\"gr3\" position=\"float\"/></fig><p>Annexin V-128 (<xref ref-type=\"bibr\" rid=\"ref26\">26</xref>) contains one cysteine in its N-terminus. The single naturally occurring cysteine residue in its interior domain has been mutated to a serine in order to eliminate the possibility of modification by thiol-reactive prosthetic conjugating agents. Annexin V-128 is freshly reduced immediately prior to each labeling study with DTT to ensure that the free thiol is fully accessible. The protein is purified of excess DTT by eluting through a Sephadex column.</p><p>The purified [<sup>18</sup>F]FBABM was concentrated onto a C18 SepPak, washed off with diethyl ether, and taken to dryness. After evaporation, the [<sup>18</sup>F]FBABM was incubated with annexin V-128 in HEPES buffer (pH = 7.4, 0.5 mL). It is noted that the number of liquid transfers and the contact surface area with the reaction vessel should be minimized to reduce unnecessary loss of annexin V-128 due to surface adsorption of the protein. Even using a careful transfer technique, approximately 1/3 of the total radioactivity is typically lost in transfers for injection onto the size-exclusion column (SEC). After HPLC purification, radiochemically pure (&gt;98%) [<sup>18</sup>F]FAN-128 was obtained with 37 ± 9% yield (<italic>n</italic> = 4, decay-corrected) from [<sup>18</sup>F]FBABM. The total recovery of radioactivity from the SEC was &gt;90% with the radioactive peak at 7−9 min being [<sup>18</sup>F]FAN-128. The unreacted [<sup>18</sup>F]FBABM was retained for &gt;1 h on the SEC column.</p><p>As a negative control, we repeated the reaction under the same condition using wild-type annexin V instead of mutated annexin V-128. Wild-type annexin V has one internal cysteine but it does not have a free thiol group on its N-terminus. In this reaction, the yield of <sup>18</sup>F-labeled protein was negligible (&lt;0.2%). This result shows that, under our reaction conditions, [<sup>18</sup>F]FBABM does not react with other nucleophiles such as the free NH<sub>2</sub> groups in annexin V, and the single cysteine residue that is present in the interior domain of wild-type annexin V is not readily accessible for maleimide binding. Therefore, we are able to conclude that the [<sup>18</sup>F]FBABM labeling of annexin V-128 is site-specific. The mutation of Cys-316 to Ser in annexin V-128 may not be necessary for future annexin V based tracer design.</p></sec><sec id=\"sec3.2\"><title>[<sup>18</sup>F]FAN-128 PS Binding Assay</title><p>The <italic>in vitro</italic> membrane-binding affinity of [<sup>18</sup>F]FAN-128 was determined by erythrocyte binding using the method reported by Tait et al.(<xref ref-type=\"bibr\" rid=\"ref16\">16</xref>) The commercially available RBCs had exposed PS on their outer surfaces of the cell membranes because of the preservatives used (<xref ref-type=\"bibr\" rid=\"ref27\">27</xref>). Annexin binding to PS is calcium dependent. By titrating calcium instead of protein, this method allows an accurate measurement of annexin binding at &lt;1% occupancy, a value similar to that for <italic>in vivo</italic> imaging where the membrane is far from saturated with protein. A typical binding curve for [<sup>18</sup>F]FAN-128 is presented in Figure <xref rid=\"fig2\" ref-type=\"fig\">2</xref>. The binding affinity was measured by determining EC<sub>50</sub> and slope from this experimental titration curve (<xref ref-type=\"bibr\" rid=\"ref22\">22</xref>). The negative log of the binding constant, <italic>p</italic>K, is equal to −(<italic>n</italic> log<sub>10</sub>EC<sub>50</sub> + log<sub>10</sub>[membrane]), where <italic>n</italic> is the number of Ca<sup>2+</sup> ions interacting with each annexin V molecule, the EC<sub>50</sub> is the molarity of Ca<sup>2+</sup> at which cell binding is half of the maximal value, and [membrane] is the concentration of annexin V-binding sites in the assay (<xref ref-type=\"bibr\" rid=\"ref16\">16</xref>). Seven batches of [<sup>18</sup>F]FAN-128 gave an average p<italic>K</italic> of 28.5 ± 1.8, an EC<sub>50</sub> of approximately 1 mM, and a slope (Hill coefficient) between 7 and 8. This compares favorably with reported values for wild-type annexin V of a p<italic>K</italic> around 30, EC<sub>50</sub> close to 1, and a slope of 8 (<xref ref-type=\"bibr\" rid=\"ref26\">26</xref>,<xref ref-type=\"bibr\" rid=\"ref16\">16</xref>). These data indicate that the <italic>in vitro</italic> binding activity of this site-specific, monosubstituted protein was not significantly different from that of the wild-type and an <italic>in vivo</italic> investigation of [<sup>18</sup>F]FAN-128 as an apoptosis imaging agent is warranted.</p><fig id=\"fig2\" position=\"float\"><label>Figure 2</label><caption><p>Typical binding curve of [<sup>18</sup>F]FAN-128 to PS-enriched membranes. Typical binding curve of [<sup>18</sup>F]FAN-128 as a factor of calcium concentration. [<sup>18</sup>F]FAN-128 is held constant well below its <italic>K</italic><sub>d</sub> (at 1 nM), while calcium is titrated between 0 and 3 mM (maximal binding is observed at 2−2.5 mM Ca<sup>2+</sup>). The membrane-bound radioactivity is plotted versus the concentration of Ca<sup>2+</sup> present. p<italic>K</italic> value is calculated from slope (N) and midpoint of titration curve (EC<sub>50</sub>) (<xref ref-type=\"bibr\" rid=\"ref16\">16</xref>). Two measurements are done for each data point.</p></caption><graphic xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"bc-2008-00164d_0001\" id=\"gr4\" position=\"float\"/></fig></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2878 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"results\"><title>RESULTS</title><p>All measurements were carried out between November 2002 and December 2003. Most of the workers (77%) were monitored on more than one occasion. Results are shown in <xref ref-type=\"table\" rid=\"tbl2\">Tables 2</xref><xref ref-type=\"table\" rid=\"tbl3\"/><xref ref-type=\"table\" rid=\"tbl4\"/><xref ref-type=\"table\" rid=\"tbl5\"/><xref ref-type=\"table\" rid=\"tbl6\"/><xref ref-type=\"table\" rid=\"tbl7\"/><xref ref-type=\"table\" rid=\"tbl8\"/>–<xref ref-type=\"table\" rid=\"tbl9\">9</xref> and <xref ref-type=\"fig\" rid=\"fig2\">Figs 2</xref> and <xref ref-type=\"fig\" rid=\"fig3\">3</xref>, and the GM referred to in the following text is the GM adjusted with mixed effect models.</p><table-wrap id=\"tbl2\" position=\"float\"><label>Table 2.</label><caption><p>Summary of fibre levels by plant and job group, exposure intensity in samples collected with 0.5–3.5 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Fibres (fibres per cm<sup>3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn1\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn1\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn1\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn2\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn3\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn4\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn5\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn2\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn3\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn4\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn5\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn2\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn3\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn4\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn5\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.097</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.096</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.072</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.046</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.047</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.43</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.82</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.39</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.88</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.086</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.086</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.037</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.025</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.025</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.098</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.58</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.082</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.081</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.056</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.036</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.037</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.082</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.039</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.037</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.87</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.46</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.46</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.062</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.061</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.61</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.080</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.078</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.48</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.66</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.50</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.43</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.43</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.97</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.029</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.022</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.022</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.073</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.058</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.037</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.040</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.032</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.021</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.019</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.097</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.050</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.030</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.031</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.019</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.015</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.015</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.044</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.019</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.013</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.013</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.048</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.015</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.011</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.011</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.044</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">60</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.010</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.007</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.007</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.034</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.014</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.011</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.011</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.035</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.057</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.034</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.035</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.050</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.017</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.017</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.092</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.049</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.051</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.086</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.085</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.39</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.096</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.041</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.043</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.025</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.017</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.018</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.059</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for SiC fibre is 0.1 fibre per cm<sup>3</sup> (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn id=\"tblfn1\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn2\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn3\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn4\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn5\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl3\" position=\"float\"><label>Table 3.</label><caption><p>Summary of respirable quartz levels by plant and job group, exposure intensity in samples collected with 6–8 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Respirable quartz (μg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn6\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Undjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn6\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn6\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn7\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn8\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn9\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn10\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn7\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn8\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn9\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn10\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn7\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn8\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn9\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn10\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">30</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">58</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">33</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">34</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.51</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.90</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.88</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.93</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">56</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.44</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.67</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.75</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">123</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for respirable quartz is 100 μg m<sup>−3</sup> (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn><p>NA, not applicable, all measurements below limit of detection (5.2 μg m<sup>−3</sup> for 8-h sampling time).</p></fn><fn id=\"tblfn6\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn7\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn8\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn9\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn10\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl4\" position=\"float\"><label>Table 4.</label><caption><p>Summary of respirable cristobalite levels by plant and job group, exposure intensity in samples collected with 6–8 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Respirable cristobalite (μg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn11\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn11\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn11\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn12\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn13\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn14\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn15\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn12\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn13\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn14\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn15\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn12\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn13\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn14\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn15\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">148</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">102</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">46</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">38</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">98</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.99</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">116</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.55</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.55</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">135</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">33</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">115</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">39</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">36</td><td rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">24</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">56</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.50</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.52</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.87</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.74</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.8</td><td rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.33</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.80</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.79</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.1</td><td rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for respirable cristobalite is 50 μg m<sup>−3</sup> (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn><p>NA, not applicable, all measurements below limit of detection (10.4 μg m<sup>−3</sup> for 8-h sampling time).</p></fn><fn id=\"tblfn11\"><label>a</label><p>Adjusted with mixed effect models</p></fn><fn id=\"tblfn12\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn13\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn14\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn15\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl5\" position=\"float\"><label>Table 5.</label><caption><p>Summary of respirable non-fibrous SiC levels by plant and job group, exposure intensity in samples collected with 6–8 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Respirable non-fibrous SiC (mg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn17\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn17\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn17\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn18\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn19\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn20\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn21\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn18\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn19\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn20\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn21\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn18\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn19\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn20\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn21\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.014</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.009</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.010</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.066</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.036</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.034</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.075</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.067</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.068</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.53</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.047</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.033</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.071</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.046</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.044</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.044</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.052</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.046</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.032</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.032</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.014</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.008</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.008</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.097</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.019</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.015</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.015</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.052</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.016</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.004</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.004</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.044</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.038</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.035</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.097</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.034</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.022</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.022</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.011</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.008</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.008</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.029</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.54</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.77</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.086</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.083</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.35</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.68</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.89</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.65</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.65</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.54</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.39</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.69</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.46</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.087</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.087</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.38</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.083</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.081</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">56</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.98</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.84</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.098</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.054</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.053</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.40</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.076</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.079</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.52</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.075</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.039</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.042</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.068</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.052</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.052</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.068</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.067</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.55</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.086</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.038</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.038</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.33</td></tr></tbody></table><table-wrap-foot><fn><p>There is no Norwegian OEL for respirable SiC; however, American Conference of Industrial Hygienists have suggested a threshold limit value of 3 mg m<sup>−3</sup> (ACGIH, 2007).</p></fn><fn id=\"tblfn17\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn18\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn19\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn20\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn21\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl6\" position=\"float\"><label>Table 6.</label><caption><p>Summary of respirable dust levels by plant and job group, exposure intensity in samples collected with 6–8 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Respirable dust (mg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn22\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn22\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn22\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn23\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn24\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn25\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn26\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn23\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn24\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn25\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn26\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn23\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn24\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn25\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn26\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.99</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.2</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.80</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.70</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.70</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.99</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.93</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.54</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.40</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.38</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.33</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.50</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.26</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.55</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.76</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.40</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.79</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.75</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.74</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.86</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.85</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.00</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.82</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.63</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.68</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.80</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.55</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.56</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.43</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.54</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.40</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.81</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.65</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">56</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.34</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.77</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.59</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.40</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.85</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.82</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.54</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for respirable dust is 0.5 mg m<sup>−3</sup> in the furnace department and 5.0 mg m<sup>−3</sup> in other areas (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn id=\"tblfn22\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn23\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn24\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn25\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn26\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl7\" position=\"float\"><label>Table 7.</label><caption><p>Summary of total dust full-shift levels by plant and job group</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Total dust (mg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn27\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn27\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn27\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn28\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn29\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn30\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn31\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn28\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn29\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn30\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn31\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn28\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn29\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn30\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn31\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">24</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.85</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.29</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.6</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">25</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.82</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.82</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">30</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.3</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">57</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.4</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">48</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.95</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.97</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.88</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.88</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.3</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for total dust is 10 mg m<sup>−3</sup> (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn id=\"tblfn27\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn28\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn29\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn30\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn31\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl8\" position=\"float\"><label>Table 8.</label><caption><p>Summary of total dust short-term levels by plant and job group, exposure intensity in samples collected with 0.5–3.5 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"21\" rowspan=\"1\">Total dust short-term (mg m<sup>−3</sup>)</td></tr><tr><td colspan=\"7\" rowspan=\"1\">Plant A</td><td colspan=\"7\" rowspan=\"1\">Plant B</td><td colspan=\"7\" rowspan=\"1\">Plant C</td></tr><tr><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn32\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn32\">a</xref></td><td colspan=\"5\" rowspan=\"1\">Unadjusted</td><td colspan=\"2\" rowspan=\"1\">Adjusted<xref ref-type=\"table-fn\" rid=\"tblfn32\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn33\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn34\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn35\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn36\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn33\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn34\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn35\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn36\">e</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn33\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn34\">c</xref></td><td rowspan=\"1\" colspan=\"1\">AM</td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn35\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn36\">e</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.6</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.57</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.84</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.52</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.61</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.52</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.38</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.81</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.85</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.9</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.99</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td rowspan=\"1\" colspan=\"1\">4</td><td rowspan=\"1\" colspan=\"1\">4.3</td><td rowspan=\"1\" colspan=\"1\">2.7</td><td rowspan=\"1\" colspan=\"1\">2.6</td><td rowspan=\"1\" colspan=\"1\">2.5</td><td rowspan=\"1\" colspan=\"1\">13</td><td rowspan=\"1\" colspan=\"1\">21</td><td rowspan=\"1\" colspan=\"1\">9</td><td rowspan=\"1\" colspan=\"1\">7.5</td><td rowspan=\"1\" colspan=\"1\">5.4</td><td rowspan=\"1\" colspan=\"1\">2.1</td><td rowspan=\"1\" colspan=\"1\">5.2</td><td rowspan=\"1\" colspan=\"1\">23</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Processing</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Refinery</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">        Crusher</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">14</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.1</td></tr><tr><td rowspan=\"1\" colspan=\"1\">        Other</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.98</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.87</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.94</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.95</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">32</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Fines</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">31</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">60</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">49</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td rowspan=\"1\" colspan=\"1\">Maintenance</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mechanics</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.75</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.76</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">47</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">16</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.6</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Electrician</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">21</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.85</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.65</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.63</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.41</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.43</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.74</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.34</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.1</td></tr></tbody></table><table-wrap-foot><fn id=\"tblfn32\"><label>a</label><p>Adjusted with mixed effect models.</p></fn><fn id=\"tblfn33\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn34\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn35\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn36\"><label>e</label><p>95th percentile.</p></fn></table-wrap-foot></table-wrap><table-wrap id=\"tbl9\" position=\"float\"><label>Table 9.</label><caption><p>Summary of sulphur dioxide exposure levels by plant and job group, actual concentrations after 6–8 h sampling time</p></caption><table frame=\"hsides\" rules=\"groups\"><thead><tr><td rowspan=\"4\" colspan=\"1\">Job group</td><td colspan=\"24\" rowspan=\"1\">Sulphur dioxide (p.p.m.)</td></tr><tr><td colspan=\"8\" rowspan=\"1\">Plant A</td><td colspan=\"8\" rowspan=\"1\">Plant B</td><td colspan=\"8\" rowspan=\"1\">Plant C</td></tr><tr><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td colspan=\"3\" rowspan=\"1\">Maximum peak value<xref ref-type=\"table-fn\" rid=\"tblfn37\">a</xref></td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td colspan=\"3\" rowspan=\"1\">Maximum peak value<xref ref-type=\"table-fn\" rid=\"tblfn37\">a</xref></td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td colspan=\"3\" rowspan=\"1\">Maximum peak value<xref ref-type=\"table-fn\" rid=\"tblfn37\">a</xref></td></tr><tr><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn38\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn39\">c</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn40\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GM<xref ref-type=\"table-fn\" rid=\"tblfn41\">e</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn42\">f</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn38\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn39\">c</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn40\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM<xref ref-type=\"table-fn\" rid=\"tblfn41\">e</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GM<xref ref-type=\"table-fn\" rid=\"tblfn41\">e</xref></td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn42\">f</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>N</italic><xref ref-type=\"table-fn\" rid=\"tblfn38\">b</xref></td><td rowspan=\"1\" colspan=\"1\"><italic>K</italic><xref ref-type=\"table-fn\" rid=\"tblfn39\">c</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GSD<xref ref-type=\"table-fn\" rid=\"tblfn40\">d</xref></td><td rowspan=\"1\" colspan=\"1\">GM<xref ref-type=\"table-fn\" rid=\"tblfn41\">e</xref></td><td rowspan=\"1\" colspan=\"1\">GM</td><td rowspan=\"1\" colspan=\"1\">GM<xref ref-type=\"table-fn\" rid=\"tblfn41\">e</xref></td><td rowspan=\"1\" colspan=\"1\">P95<xref ref-type=\"table-fn\" rid=\"tblfn42\">f</xref></td></tr></thead><tbody><tr><td rowspan=\"1\" colspan=\"1\">Furnace</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Mix</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.001</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.001</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.45</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.008</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.007</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.63</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.62</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.0</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">11</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Charger/mix</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">10</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.37</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Payloader</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.002</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">15</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.002</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.72</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.006</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">25</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.006</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.42</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.90</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">18</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.065</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.065</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.74</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.73</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.2</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Crane</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">23</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.016</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.016</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">22</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.28</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.013</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.013</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.7</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Control room</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.036</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.036</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3.2</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">58</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">5</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.27</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">13</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">36</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">8</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.061</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.6</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.061</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">17</td></tr><tr><td rowspan=\"1\" colspan=\"1\">    Cleaning</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">1</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/><td rowspan=\"1\" colspan=\"1\"/></tr><tr><td rowspan=\"1\" colspan=\"1\">    Sorting</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">3</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td rowspan=\"1\" colspan=\"1\">NA</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.19</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.7</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">24</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">20</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">9</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">4.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">0.12</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.0</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">2.1</td><td align=\"char\" char=\".\" rowspan=\"1\" colspan=\"1\">6.0</td></tr></tbody></table><table-wrap-foot><fn><p>The Norwegian OEL for sulphur dioxide is 2 p.p.m. (The Norwegian Labour Inspection Authority, 2007).</p></fn><fn><p>NA, not applicable, all measurements below limit of detection, a measurement is below the limit of detection when all its 10-s reading periods are &lt;0.2 p.p.m.</p></fn><fn id=\"tblfn37\"><label>a</label><p>Maximum observed peak value for a 10-s averaging period within shift measurements.</p></fn><fn id=\"tblfn38\"><label>b</label><p>Number of measurements.</p></fn><fn id=\"tblfn39\"><label>c</label><p>Number of persons.</p></fn><fn id=\"tblfn40\"><label>d</label><p>Geometric standard deviation.</p></fn><fn id=\"tblfn41\"><label>e</label><p>GM adjusted with linear mixed effect models.</p></fn><fn id=\"tblfn42\"><label>f</label><p>95th percentile adjusted with linear mixed effect models.</p></fn></table-wrap-foot></table-wrap><fig id=\"fig2\" position=\"float\"><label>Fig. 2.</label><caption><p>Box plot of fibre exposure in fibre per cm<sup>3</sup> for job groups in Plants A, B and C. The exposure for the cleaning operator is not included due to too few exposure measurements. The box bounds the 25th and 75th percentiles, encompasses 50% of the data and includes the median (solid line within the box). Dispersion of the data above and below this range is marked by whiskers that extend to the 10th and 90th percentiles. Points above or below the whiskers represent the 95th and 5th percentiles.</p></caption><graphic xlink:href=\"annhygmen029f02_lw\"/></fig><fig id=\"fig3\" position=\"float\"><label>Fig. 3.</label><caption><p>Box plot of respirable dust exposure in mg m<sup>−3</sup> for job groups in Plants A, B and C. The exposure for the cleaning operator is not included due to too few exposure measurements. The description of the box plots is the same as in <xref ref-type=\"fig\" rid=\"fig2\">Fig. 2</xref>.</p></caption><graphic xlink:href=\"annhygmen029f03_lw\"/></fig><sec><title>Fibres</title><p>Most of the fibre sampling (90%) was initiated during the first half of the shift due to practical considerations. A total of 40% of the samples were below the detection limit. However, the fibre count was zero in only 9% of the samples. The fibre exposure levels are shown in <xref ref-type=\"fig\" rid=\"fig2\">Fig. 2</xref> and <xref ref-type=\"table\" rid=\"tbl2\">Table 2</xref>. Highest GM of fibre exposures was found in the furnace and sorting areas in all plants, and the cleaning operators in Plant A had the highest exposure to fibres (2.7 fibres cm<sup>−3</sup>). The control room, cleaning and sorting operators in Plant A, sorting operators in Plant B and charger, charger/mix, payloader and sorting operators in Plant C had all GM exposures of 0.1 fibres cm<sup>−3</sup> or more. The sorting operators had significantly higher exposure in Plant C compared to the other plants (<italic>P</italic> &lt; 0.05). Control room operators had significantly higher exposure to fibres in Plant A compared to the two other plants (<italic>P</italic> &lt; 0.05). The Norwegian occupational exposure limit (OEL) for SiC fibres of 0.1 fibre cm<sup>−3</sup> was exceeded by 53% of the samples from the furnace department and 17% of the samples from the maintenance department (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). Only 0.2% of the samples from the processing department exceeded the OEL.</p></sec><sec><title>Crystalline silica</title><p>The cleaning operators in Plant A had the highest GM exposure to respirable quartz (20 μg m<sup>−3</sup>). The GM exposure of the mix operators in Plants A and C and charger/mix and charger operators in Plant C varied between 13 and 8.0 μg m<sup>−3</sup>, while all other job groups had a GM exposure of &lt;5 μg m<sup>−3</sup> (<xref ref-type=\"table\" rid=\"tbl3\">Table 3</xref>). The sorting operators in Plant C had a significantly higher exposure to quartz than the sorting operators in the two other plants (<italic>P</italic> &lt; 0.05). The quartz exposures were generally low and &lt;1% of the samples exceeded the OEL of 100 μg m<sup>−3</sup> (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). The samples exceeding the OEL were all from the maintenance department in Plant B.</p><p>The job group exposed to the highest levels of respirable cristobalite was the charger/mix operators in Plant C (GM = 35 μg m<sup>−3</sup>) (<xref ref-type=\"table\" rid=\"tbl4\">Table 4</xref>). GM exposures &gt;10 μg m<sup>−3</sup> were found among the cleaning operators in Plant A, sorting operators in Plants B and C and the mix, charger and payloader operators in Plant C. The mix operators, crane and sorting operators had significantly lower exposure in Plant A compared to the two other plants (<italic>P</italic> &lt; 0.01). The OEL of 50 μg m<sup>−3</sup> was exceeded in 2.1% of the samples (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>).</p><p>The crystalline silica exposures for workers in the processing department and in the maintenance department were generally low. More than 90% of the cristobalite samples from each of these departments were below the detection limit and the corresponding results for quartz exposure levels in these departments were 65 and 58%, respectively.</p></sec><sec><title>Respirable non-fibrous SiC</title><p>The highest GM exposure to respirable non-fibrous SiC was found among the crusher operators (GM = 0.39–0.65 mg m<sup>−3</sup>) and the cleaning operators in Plant A (GM = 0.49 mg m<sup>−3</sup>) (<xref ref-type=\"table\" rid=\"tbl5\">Table 5</xref>). The mix, sorting and fines operators had a significantly lower exposure in Plant A compared to the two other plants (<italic>P</italic> &lt; 0.05). The control room operators had a significantly lower exposure in Plant C compared to the two other plants (<italic>P</italic> &lt; 0.001). Norway does not have a specific OEL for respirable non-fibrous SiC. The American Conference of Industrial Hygienists has recommended a threshold limit value of 3 mg m<sup>−3</sup> (<xref ref-type=\"bibr\" rid=\"bib1\">ACGIH, 2007</xref>) and only 0.4% of the measurements exceeded this limit.</p></sec><sec><title>Quantified crystalline components</title><p>The respirable dust in the furnace department contained on average 18% SiC, 1.1% quartz and 2.1% cristobalite. In the respirable dust from the processing department, we found 57% SiC, 0.2% quartz and 0.1% cristobalite, while the respirable dust in the maintenance department contained 21% SiC, 0.5% quartz and 0.2% cristobalite.</p></sec><sec><title>Respirable dust</title><p>The respirable dust exposure levels are shown in <xref ref-type=\"fig\" rid=\"fig3\">Fig. 3</xref> and <xref ref-type=\"table\" rid=\"tbl6\">Table 6</xref>. The highest GM exposures to respirable dust (&gt;0.5 mg m<sup>−3</sup>) were found among cleaning operators in Plant A, mix, charger, charger/mix and sorting operators in Plant C and crusher operators in all plants. When comparing plants, sorting operators in Plant A had significantly lower exposure levels than the sorting operators in the other plants (<italic>P</italic> &lt; 0.05) and the mechanics in Plant B had a significantly higher exposure than the other two plants (<italic>P</italic> &lt; 0.05). The Norwegian OEL for mixed respirable dust is 0.5 mg m<sup>−3</sup> in the furnace department and furnace-related areas of the SiC industry (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). A total of 26% of the samples from the furnace department and 15% of the samples from maintenance workers performing maintenance in the furnace department exceeded this limit. Since there is no specific OEL for respirable dust in the processing department, the OEL for nuisance respirable dust of 5 mg m<sup>−3</sup> was applied here (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). Only 0.4% of the samples in the processing department and none of the samples from maintenance workers performing maintenance work in the processing department exceeded this limit.</p></sec><sec><title>Total dust</title><p>Cleaning and crusher operators in Plant A and charger, charger/mix and sorting operators in Plant C had GM exposures to total dust of 4 mg m<sup>−3</sup> or higher (<xref ref-type=\"table\" rid=\"tbl7\">Table 7</xref>). The cleaning operators had four times higher GM exposure than any of the other job groups, but this observation was based on only two measurements. The sorting operators had a significantly lower exposure in Plant A compared to Plants B and C (<italic>P</italic> &lt; 0.001) and the crane operators in Plant B had a significantly higher exposure compared to crane operators in the two other plants (<italic>P</italic> &lt; 0.001). As there is no specific OEL for total dust in the SiC industry, the Norwegian OEL for nuisance total dust of 10 mg m<sup>−3</sup> was applied here (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). This OEL does not take into account that the dust might contain harmful components. Four per cent of the samples were above the OEL and two-thirds of these were from the fines or refinery crusher operators.</p><p><xref ref-type=\"table\" rid=\"tbl8\">Table 8</xref> summarizes the total dust short-term exposure levels. The job group GM ratios of short-term to full-shift total dust samples varied from 0.19 to 1.4 in Plant A, 2.3 to 0.55 in Plant B and 0.4 to 1.2 in Plant C. However, the GM ratios for the separate plants were close to 1; 0.9 for Plant A and C and 1.0 for Plant B.</p></sec><sec><title>Sulphur dioxide</title><p>Sulphur dioxide exposure was assessed for job groups in the furnace hall. The mean exposure to sulphur dioxide over a full shift was generally low. The charger and charger/mix operators had the highest measured GM (0.37 p.p.m.), which is one-fifth of the OEL of 2 p.p.m. (<xref ref-type=\"table\" rid=\"tbl9\">Table 9</xref>) (<xref ref-type=\"bibr\" rid=\"bib21\">The Norwegian Labour Inspection Authority, 2007</xref>). The highest GM for maximum peak value was found among the control room operators in Plant B (13 p.p.m.) (<xref ref-type=\"table\" rid=\"tbl9\">Table 9</xref>). The crane and control room operators in Plant B had a significantly higher maximum peak value compared to the same job groups in the other plants (<italic>P</italic> &lt; 0.05). The sorting operators in Plant A had a significantly lower exposure than the sorting operators in the two other plants (<italic>P</italic> &lt; 0.001).</p></sec><sec><title>Use of respirators</title><p>Respirators were available for all workers. The use of respirators was mandatory for workers in the furnace hall and for some operations in the refinery department (e.g. packing and cleaning). Different types of respirators were used [i.e. disposable half-masks with P2 or P3 particulate filters, half-masks with particulate filter and gas filter for acid gases (SO<sub>2</sub>), powered air-purifying respirators, compressed air-fed respirators and self-contained breathing apparatus (when concentrations of CO were high)]. Most of the workers (74%) reported using respirators some or all of the time during the sampling. The use of respirators varied between plants, and 79% of the workers in Plant C used respirators all or some of the time compared to ∼50% of the workers in Plants A and B. The use also varied within plants with 78% of the workers in the furnace department using respirators some or all of the time compared to 46% in refinery and maintenance departments. When measurements exceeded the OEL, between 79 and 100% of the workers had used respirators some or all of the time depending on component. The GM exposure for total dust was 79% higher among workers using respirators all the time, and 65% higher among workers using respirators some of the time compared to workers not using respirators. Similar trends were seen for all other agents.</p></sec><sec><title>Work conditions</title><p>Ninety-three per cent of the workers reported their perception of the work conditions of the shift. Of these, 84% reported that the conditions were normal, 6% reported it to be worse than normal and 10% reported better than normal.</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2883 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\"><title>INTRODUCTION</title><p>F<sub>1</sub>-ATPase is an ATP-driven rotary molecular motor in which the central <italic>γ</italic>-subunit rotates against a surrounding cylinder made of <italic>α</italic><sub>3</sub><italic>β</italic><sub>3</sub> subunits arranged alternately (<xref ref-type=\"bibr\" rid=\"bib1\">1</xref>–<xref ref-type=\"bibr\" rid=\"bib5\">5</xref>). ATP hydrolysis occurs in three catalytic sites, each hosted primarily by a <italic>β</italic>-subunit (<xref ref-type=\"bibr\" rid=\"bib1\">1</xref>). As expected, rotation proceeds basically in steps of 120°, each coupled to consumption of one ATP molecule (<xref ref-type=\"bibr\" rid=\"bib6\">6</xref>,<xref ref-type=\"bibr\" rid=\"bib7\">7</xref>). Direction of rotation is counterclockwise when viewed from the side of <italic>γ</italic>-protrusion (<xref ref-type=\"bibr\" rid=\"bib3\">3</xref>), and the three <italic>β</italic>-subunits have been shown to bind ATP sequentially in this direction (<xref ref-type=\"bibr\" rid=\"bib8\">8</xref>). The F<sub>1</sub> is a reversible molecular machine in that reverse rotation of <italic>γ</italic> by an external force results in ATP synthesis (<xref ref-type=\"bibr\" rid=\"bib9\">9</xref>,<xref ref-type=\"bibr\" rid=\"bib10\">10</xref>). The reversal of chemical reaction (ATP hydrolysis) by manipulation of the <italic>γ</italic>-angle alone implies a <italic>γ</italic>-dictator mechanism, in which the orientation of the <italic>γ</italic>-rotor relative to the <italic>α</italic><sub>3</sub><italic>β</italic><sub>3</sub> stator determines which of the chemical reactions (binding/release of ADP, phosphate, and ATP, and synthesis/hydrolysis of ATP) is to occur in each catalytic site (<xref ref-type=\"bibr\" rid=\"bib5\">5</xref>). The basic coupling scheme for hydrolysis-driven rotation has been worked out for an F<sub>1</sub>-ATPase of thermophilic origin (<xref ref-type=\"bibr\" rid=\"bib11\">11</xref>). The 120° rotational step is split into 80–90° and 40–30° substeps (<xref ref-type=\"bibr\" rid=\"bib12\">12</xref>), which we refer to here as 80° and 40° substeps. The first 80° substep is initiated and driven by ATP binding (<xref ref-type=\"bibr\" rid=\"bib12\">12</xref>). ADP release occurs immediately after ATP binding (at least when rotation is slowed down by an external force), and the release assists the 80° substep (<xref ref-type=\"bibr\" rid=\"bib11\">11</xref>). An ATP is hydrolyzed in the interim between 80° and 40° substeps (<xref ref-type=\"bibr\" rid=\"bib13\">13</xref>), after the ATP has been bound at 0° and 200° rotation has taken place (<xref ref-type=\"bibr\" rid=\"bib8\">8</xref>). Another event at the 80° interim is phosphate release, which initiates and drives the last 40° rotation (<xref ref-type=\"bibr\" rid=\"bib11\">11</xref>). The ATP hydrolysis and phosphate release at 80° both take ∼1 ms (<xref ref-type=\"bibr\" rid=\"bib11\">11</xref>,<xref ref-type=\"bibr\" rid=\"bib12\">12</xref>), whereas ATP binding at 0° is fast with a bimolecular rate of ∼2 × 10<sup>7</sup> M<sup>−1</sup> s<sup>−1</sup> (<xref ref-type=\"bibr\" rid=\"bib12\">12</xref>). Above the apparent Michaelis-Menten constant, <italic>K</italic><sub>m</sub>, of ∼15 <italic>μ</italic>M, therefore, the rate-limiting processes are the two reactions at 80°.</p><p>The scheme above was obtained at room temperature (23°C), whereas the optimum temperature for the thermophilic enzyme is ∼75°C (<xref ref-type=\"bibr\" rid=\"bib14\">14</xref>). A different scheme(s) may apply at different temperatures, and the temperature dependence of the rate-limiting process(es) may reveal the nature of that process. Here, we have investigated the temperature dependence of rotation and hydrolysis of the F<sub>1</sub>-ATPase derived from a thermophile <italic>Bacillus</italic> PS3. We used its minimal subcomplex <italic>α</italic><sub>3</sub><italic>β</italic><sub>3</sub><italic>γ</italic>, which is active in rotation and hydrolysis, and refer to it simply as F<sub>1</sub>. Unexpected findings are that a new rate-limiting step emerges at low temperatures and that F<sub>1</sub> quickly falls into an inhibited state at high temperatures.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2905 prediction: method true label: other\n",
      "<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" id=\"Sec1\" sec-type=\"introduction\"><title>Introduction</title><p>Chronic prostatitis is one of the most common urological diseases in men [<xref ref-type=\"bibr\" rid=\"CR1\">1</xref>]. According to the latest classification by the National Institutes of Health (NIH), this type belongs to group III of the so-called chronic prostatitis/chronic pelvic pain syndrome (CP/CPPS) [<xref ref-type=\"bibr\" rid=\"CR15\">15</xref>]. <italic>Chlamydia trachomatis</italic> (<italic>C. trachomatis</italic>) is a sexually transmitted intracellular microorganism which can survive in the human organism for several years in a state of latency. It is an etiologic factor of prostatitis [<xref ref-type=\"bibr\" rid=\"CR3\">3</xref>, <xref ref-type=\"bibr\" rid=\"CR19\">19</xref>]. Chlamydial prostatitis was formerly included in group III of the NIH classification [<xref ref-type=\"bibr\" rid=\"CR12\">12</xref>]. Currently it is frequently placed in group II (so-called chronic bacterial prostatitis — CBP) [<xref ref-type=\"bibr\" rid=\"CR20\">20</xref>].</p><p>Polymorphonuclear (PMN) leukocyte count is thought to be the main marker of inflammation in the genitourinary tract. It is usually determined in urethral swabs, expressed prostatic secretions (EPS), semen, or urinary sediment [<xref ref-type=\"bibr\" rid=\"CR18\">18</xref>, <xref ref-type=\"bibr\" rid=\"CR21\">21</xref>]. PMN elastase, secreted by PMN leukocytes, is also a sensitive indicator of inflammation [<xref ref-type=\"bibr\" rid=\"CR28\">28</xref>]. Its importance in detecting latent genitourinary inflammation in men has been described in recent literature [<xref ref-type=\"bibr\" rid=\"CR29\">29</xref>].</p><p>The concentration of citric acid is a significant parameter of prostate gland function [<xref ref-type=\"bibr\" rid=\"CR11\">11</xref>]. Substantial amounts of citric acid are produced and stored in the gland [<xref ref-type=\"bibr\" rid=\"CR7\">7</xref>]. A decrease in its concentration is observed in cases of inflammation or cancer of the prostate gland [<xref ref-type=\"bibr\" rid=\"CR2\">2</xref>, <xref ref-type=\"bibr\" rid=\"CR10\">10</xref>].</p><p>The aim of this study was to assess the occurrence of inflammation markers and citric acid levels in patients with chronic prostatitis (CP/CPPS or/and CBP) irrespective of etiology. The study was also conducted to evaluate the relationship of <italic>C. trachomatis</italic> infection with the parameters of prostate inflammation and function.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2927 prediction: method true label: other\n",
      "<sec><title>Results</title><sec><title>Derivation of a GPS</title><p>To identify a set of genes whose expression was associated with tumour cell proliferative activity, two proliferation-based systems were analysed and integrated as described below and illustrated in <xref ref-type=\"fig\" rid=\"fig1\">Figure 1</xref>.</p><p><italic>A. Gene expression analysis of a CRC in vitro model</italic></p><p>The <italic>in vitro</italic> system involved identification of genes that were to reflect the proliferative activity of CRC cell lines. Genes DE between exponentially growing (nonconfluent) and growth-inhibited (confluent) cell lines were identified (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1A–C</xref>). Firstly, DE genes between Cy5-labelled nonconfluent and confluent samples were identified by statistical analysis of microarray (two-class paired, FDR&lt;1%; <xref ref-type=\"other\" rid=\"bib30\">Tusher <italic>et al</italic>, 2001</xref>). Secondly, DE genes between Cy3-labelled nonconfluent and confluent samples (biological replicates) were identified using the same approach. To minimise gene-specific dye bias and other sources of variation, only genes that were present in both SAM-generated gene sets were selected. The gene set was further reduced to genes whose expression was consistently altered in the same direction in at least 8 out of 10 cell lines, yielding a total 881 DE genes with known annotation (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1D</xref>).</p><p>Gene ontology (GO) analysis showed that categories related to cell cycle and DNA metabolism were the most over-represented biological themes among the DE genes, mainly because of genes that were overexpressed in exponentially growing cells (<italic>P</italic>&lt;10<sup>−5</sup>).</p><p><italic>B. Gene expression profile of the proliferative compartment of colon crypts</italic></p><p>The second gene set used for the design of a GPS was based on the physiological expression profile of human colon crypts. <xref ref-type=\"other\" rid=\"bib16\">Kosinski <italic>et al</italic> (2007)</xref> compared the proliferative bottom part of crypts with the differentiated crypt top, and identified 299 DE genes highly expressed in the proliferative bottom (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1E</xref>). The GO terms that were over-represented within this gene list were related to cell proliferation and renewal, consistent with the physiological function of the bottom crypt compartment.</p><p><italic>C. Definition of the GPS</italic></p><p>To define a final GPS enriched in key-proliferative genes, the <italic>in vitro</italic> and <italic>in vivo</italic>-derived DE gene lists were screened for common genes (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1F</xref>). Thirty-six genes were found to be overexpressed in both exponentially growing CRC cell lines and the proliferative crypt compartment. These genes were defined as GPS that included 15 cell cycle-related genes (<xref ref-type=\"other\" rid=\"sup1\">Supplementary Table 2</xref>).</p><p>The expression of 7 genes randomly selected from the 15 cell cycle-related genes was validated by qRT–PCR on the cell line cDNAs. A close correlation between qRT–PCR and array data was observed (<xref ref-type=\"other\" rid=\"sup1\">Supplementary Figure 1</xref>).</p></sec><sec><title>Classification of colon tumours according to the GPS expression level</title><p>To investigate the association between the GPS and patient parameters, expression values of the GPS genes were first obtained from the array-generated expression profiles. Expression data were available for all 36 genes in cohort A, and 35 genes (except CDCA5) in cohort B tumours. For each cohort, tumours were split into two consensus clusters on the basis of their GPS expression using K-means clustering. Tumours with relatively high or low GPS expression were defined as having high or low proliferative activity, respectively (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1G</xref>).</p></sec><sec><title>Reduced expression of the GPS is associated with unfavourable clinico-pathologic variables</title><p>Intriguingly, we observed an association between <italic>reduced</italic> GPS expression and an increased risk of recurrence in both cohorts (<xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref>). Groups with reduced GPS expression were significantly enriched for recurrent tumours (<italic>P</italic>=0.021 and 0.005 for cohort A and B, respectively). In cohort A, reduced GPS expression was also associated with higher disease stage (<italic>P</italic>=0.015). Further, reduced GPS expression was significantly more frequent in cohort A tumours with lymphatic invasion compared with those without lymphatic invasion (<italic>P</italic>=0.018). Gene set comparison analysis confirmed that GPS contained a higher-than-random proportion of DE genes among subgroups with clinico-pathologic parameters for which a significant association with the GPS was found in nonparametric tests (<xref ref-type=\"other\" rid=\"sup1\">Supplementary Table 3</xref>).</p></sec><sec><title><italic>Reduced</italic> GPS expression is associated with <italic>shorter</italic> DFS in colon cancer</title><p>To examine whether a difference in cell proliferation determined by the GPS may be associated with time to recurrence, DFS was plotted for low GPS and high GPS tumours (<xref ref-type=\"fig\" rid=\"fig2\">Figure 2</xref>). DFS was significantly shorter in patients with reduced GPS expression (<italic>P</italic>=0.033 and 0.011 for cohort A and B patients, respectively; <xref ref-type=\"fig\" rid=\"fig2\">Figure 2A and C</xref>). This association remained significant in cohort A, when patients with adjuvant therapy were excluded (<italic>P</italic>=0.029; <xref ref-type=\"fig\" rid=\"fig2\">Figure 2B</xref>).</p><p>If prolifetative activity is a crucial factor influencing DFS, then a GPS consisting of only genes directly implicated in cell cycle process should be sufficient to separate patients into groups with different survival rates. Therefore, the GPS was reduced to the 15 cell cycle-regulated genes and survival analysis was repeated. The modified GPS stratified patients into groups that differed in their DFS more stringently (<italic>P</italic>=0.022 and 0.003 for cohort A and B patients, respectively; <xref ref-type=\"fig\" rid=\"fig2\">Figure 2D–F</xref>) than the original GPS including all 36 genes. Therefore, colon cancer patients with tumours of lower proliferative activity are more likely to recur.</p><p>When the parameters predicting patient outcome in univariate analysis were investigated in a multivariate model, disease stage was the only independant predictor of 5-year DFS in cohort A (<xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>).</p></sec><sec><title><italic>Increased</italic> GPS expression is associated with <italic>shorter</italic> DFS in breast cancer</title><p>As the association between reduced GPS expression and poor colon cancer prognosis was an unexpected finding, we tested the validity of our GPS on public array data from two independant breast cancer cohorts. Using these data, an association between increased proliferation and bad outcome has been established earlier (<xref ref-type=\"other\" rid=\"bib31\">van de Vijver <italic>et al</italic>, 2002</xref>; <xref ref-type=\"other\" rid=\"bib23\">Pawitan <italic>et al</italic>, 2005</xref>).</p><p>For each breast cancer data set, patients were stratified into two groups with either low or high GPS expression and their DFS was plotted. In both data sets, patients with increased GPS expression had significantly shorter DFS compared with patients with reduced GPS expression (<italic>P</italic>&lt;0.0001; <xref ref-type=\"fig\" rid=\"fig3\">Figure 3</xref>), confirming the previously established association. Therefore, the association between low proliferation and bad colon cancer outcome is not the result of biased methodology.</p></sec><sec><title>Ki-67 PI is not associated with outcome</title><p>Ki-67 PI ranged from 25 to 96%, with a mean value of 76.3±17.5 and a median value of 81.8%. When these 73 patients were stratified into two groups differing in their GPS expression, a significant difference in DFS was apparent (<italic>P</italic>=0.01; <xref ref-type=\"other\" rid=\"sup1\">Supplementary Figure 2C</xref>). However, when patients were stratified into two groups according to the mean Ki-67 PI, the DFS of the group with a low PI was similar to that with a high PI (<italic>P</italic>=0.55; <xref ref-type=\"other\" rid=\"sup1\">Supplementary Figure 2D</xref>). Furthermore, when analysis was limited to patients with the highest and the lowest Ki-67 values, no statistical difference in DFS was observed (data not shown). No correlation was found between the Ki-67 PI and mean GPS expression of tumours (Spearman <italic>R</italic>=0.06; <italic>P</italic>=0.63).</p></sec></sec> \n",
      "\n",
      "\n",
      "\n",
      "2930 prediction: method true label: other\n",
      "<sec><title>Results</title><p>Data were collected on 936 patients from 81 individual and combined hospital sources (e.g,, multihospital NHS trusts). In terms of the geographical distribution of the hospital sources that submitted waiting times data, 76 (94%) were in the United Kingdom or Wales, 4 (5%) were in Scotland and 1 (1%) in Northern Ireland (see <xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref>). The 76 UK and Welsh hospitals were drawn from 67 out of 184 acute NHS trusts in the two countries. However, not all acute trusts offer chemotherapy. Of these, 13 (19%) trusts represented were teaching or university trusts (For the purpose of this audit, a teaching/university trust was defined by ‘teaching' or ‘university' in the title of the organisation.) and the proportion is similar (14% among those who did not submit data). <xref ref-type=\"fig\" rid=\"fig2\">Figure 2</xref> shows that there does not appear to be a bias towards any particular parts of the United Kingdom and Wales with regard to participation.</p><p><xref ref-type=\"fig\" rid=\"fig3\">Figure 3</xref> shows that the most frequent diagnosis was breast cancer, followed by colorectal and lung cancer. <xref ref-type=\"fig\" rid=\"fig4\">Figure 4</xref> shows that intravenous chemotherapy comprises the most frequent category of systemic therapy at 85%. Treatment intent was curative (including adjuvant) in 38%, palliative in 49% and don't know in 8%. Chemo/radiotherapy was administered to 6%, making a total of 44% treated radically.</p><p>In terms of waiting list status, 1% of cases (8) were emergencies, 27% (252) were urgent, 48% (445) were routine and 25% (231) were unknown. Forty-one per cent (388) of treatments were FDT. Fifteen per cent (141) of all treatments were subject to elective delay. The reasons given for elective delay (140) (One of the data was missing.) were recovering from surgery (18%; 25), patient request (19%; 26), intercurrent illness (9%; 13), don't know (1%; 2) and other (53%; 74). Elective delays were excluded from further calculations.</p><p><xref ref-type=\"fig\" rid=\"fig5\">Figure 5</xref> shows waiting times for treatment categorised by treatment intent and by waiting list status. Overall, there is only a small improvement in the rapidity of treatment for those categorised as urgent rather than routine. The median waiting time from first oncology consultation to the start of urgent chemotherapy was 9 days (IQR 16 days) and from first oncology consultation to the start of routine chemotherapy was 12 days (IQR 11 days) (<xref ref-type=\"fig\" rid=\"fig5\">Figure 5A</xref>). Patients categorised as requiring urgent palliative therapy were treated fastest (<xref ref-type=\"fig\" rid=\"fig5\">Figure 5B</xref>). Waiting times by treatment intent and by waiting list status are shown in <xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>. Patients in the urgent category were treated between 3 and 5 days sooner. These differences in median waiting times were significant for overall and for radical cases, but not for palliative cases using the Wilcoxon two-sample test (<xref ref-type=\"fig\" rid=\"fig5\">Figure 5</xref>).</p><p><xref ref-type=\"fig\" rid=\"fig6\">Figure 6</xref> shows the results in terms of the Department of Health's 31-day and 62-day targets (<xref ref-type=\"other\" rid=\"bib5\">Department of Health, 2000</xref>). The median time from DDT to FDT was 9 days (IQR 9 days) and from urgent GP referral to FDT 38 days (IQR 37 days). <xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref> shows compliance with the treatment targets as defined by <xref ref-type=\"other\" rid=\"bib12\">Joint Collegiate Council for Oncology (1993)</xref> and by the <xref ref-type=\"other\" rid=\"bib5\">Department of Health (2000)</xref>. Achievement of the Department of Health 31-day target was 98.1%. Performance on the 62-day target is less satisfactory at 76.2%. The JCCO target that radical chemotherapy should start within 21 days was met in 83.7% of cases, but the recommendation that urgent treatment should start within 2 days was only met in 23.4% of cases (<xref rid=\"tbl3\" ref-type=\"table\">Table 3</xref>).</p><p><xref ref-type=\"fig\" rid=\"fig7\">Figure 7</xref> shows the results for individual hospital units ranked in order and identified by their individual code number. The varying numbers in each chart reflect the number of patients in the different categories. For the 62-day target, there were 320 patients whose systemic therapy was their FDT and who were not subject to elective delay. However, the date of urgent referral by the GP was only available on 84 patients (26%). Lack of the appropriate data probably influenced the apparently low achievement of the 62-day target, with only 61% (20 of 33) of hospitals achieving it for all patients, compared with 93% (66/71) for the 31-day target. There was poor compliance with the JCCO target that urgent treatment should be given within 48 h, with only 10 hospitals (19%) achieving it for all their patients.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2932 prediction: method true label: other\n",
      "<sec><title>Results</title><p>The Child Health System contained data on 444 168 live births delivered between January 1971 and December 1986 in Northern Ireland. The Registrar General's Office, which registers all births within Northern Ireland, registered 444 111 live births for the same period. Only live births were included in this analysis. Newborns with congenital malformations (8883), missing birth weight or gestation age (4923), who were part of a twin or multiple gestations (<italic>n</italic>=9235), or with Down's syndrome (<italic>n</italic>=440) were excluded. The Northern Ireland Cancer Registry contains 251 patients diagnosed with brain tumours in Northern Ireland who were born between 1971 and 1986; of whom, 208 (82.9%) were identified within the Child Health System; 155 cases were malignant. The characteristics of the identified cases (<italic>n</italic>=155) and non-cases (<italic>n</italic>=420 436) are shown in <xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref>. Mean age at diagnosis was 13.7 years (s.d. 7.61).</p><p><xref rid=\"tbl1\" ref-type=\"table\">Table 1</xref> shows the association between exposures of interest and risk of brain tumour. Boys had an increased risk of brain tumour as compared with girls, and children born between 1977 and 1978 were at an increased risk, although this may reflect differences in the completeness of data for earlier periods. Women who had a history of miscarriage were more likely to have a child with a brain tumour (OR=1.68; 95% CI: 1.16–2.42). Children who were born into households with ⩾3 children had a significantly increased risk of developing a brain tumour. There was no association between other perinatal factors and brain tumour risk.</p><p><xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref> shows the mutually adjusted odds ratios for predictor variables and brain tumour risk. Newborn boys and newborns to mothers with a miscarriage history continued to be at an increased risk in the multivariate models. In addition, having either ⩾3 children or ⩾3 adults in the household at the time of the index birth increased the risk of brain tumour. Other perinatal factors were not associated with brain tumour risk in the multivariate model.</p></sec> \n",
      "\n",
      "\n",
      "\n",
      "2937 prediction: method true label: other\n",
      "<sec><title>Results</title><sec><title>Patient characteristics</title><p>Overall, 45 patients were treated, 29 of whom received the MTD. Patient demographics and clinical characteristics are shown in <xref rid=\"tbl2\" ref-type=\"table\">Table 2</xref>. In total, seven patients did not receive two cycles of SCOUT and were not evaluable for response assessment; four of these patients were in the MTD group. The evaluable population therefore comprised 38 patients in total and 25 patients treated at the MTD. Overall, five patients were not assessable for time to progression (TTP), two of whom were treated at the MTD (three patients died before completing an assessment scan after two cycles and two withdrew for psycho-social reasons). All patients were assessable for toxicity and OS. A majority of patients had multiple lesions; only six patients had disease confined to the liver.</p></sec><sec><title>Response to treatment</title><p>Response to treatment is shown in <xref rid=\"tbl3\" ref-type=\"table\">Table 3</xref>.</p><p>An ORR of 68% (95% CI: 46–85%) and a disease-control rate of 100% (95% CI: 86–100%) were observed in the 25 evaluable patients who received the MTD. Median OS was 19.6 months in 29 assessable patients (95% CI: 15.5–27.2) (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1A</xref>) and median TTP was 8.5 months in 27 assessable patients (95% CI: 7.6–11.1) (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1B</xref>). One-year survival was 72.4% (95% CI: 52.3–85.1%) and 2-year survival was 37.2% (95% CI: 13.2–61.8%).</p><p>The ORR in the 38 evaluable patients in the phase I/II study was 63% (95% CI: 46–78%), with a disease-control rate of 89% (95% CI: 75–97%). After a median follow-up of 14.9 months, median OS was 16.8 months (95% CI: 9.6–25.3) in 45 evaluable patients (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1C</xref>) and median TTP was 8.7 months (95% CI: 7.9–10.4) in 40 evaluable patients (<xref ref-type=\"fig\" rid=\"fig1\">Figure 1D</xref>). One-year survival was 62.2% (95% CI: 46.4–74.6%) and 2-year survival was 38.7% (95% CI: 21.6–55.5%).</p><p>Liver resections were performed on three patients. One patient with a partial response after four cycles underwent an R0 resection, one patient who had stable disease after three cycles had an R1 resection and one patient who had 5 cycles and achieved a partial response had an R1 resection. Another patient, who underwent a total of 11 treatment cycles and achieved a partial response on a re-challenge with SCOUT is being assessed for liver resection.</p></sec><sec><title>Tolerability</title><p>Tolerability data for the phase I cohort are detailed in our earlier publication. In the MTD group, 138 UFT courses were prescribed to the 29 patients, 16 of which were reduced by 1 capsule daily and 8 of which were reduced by 2 capsules daily; 21 patients (72%) received UFT without a dose reduction, 13 of whom (45%) received all 6 UFT cycles at full dose. A total of 139 doses of irinotecan therapy were administered, 17 doses were reduced by 20% and 3 by 50%; 132 doses of oxaliplatin were administered, 9 were reduced by 20% and 5 by 50%. The median dose intensity was 89% for both irinotecan and oxaliplatin after two cycles; 92% and 91% for irinotecan and oxaliplatin, respectively, after four cycles; and 95% and 92% for irinotecan and oxaliplatin, respectively, after six cycles.</p><p>SCOUT was well tolerated at the MTD, as shown in <xref rid=\"tbl4\" ref-type=\"table\">Table 4</xref>. One patient developed grade 3 neutropaenia and another had grade 3 leucopaenia. One patient with a history of hypertension, atrial fibrillation and type II diabetes had a grade 4 cardiac event and died following a myocardial infarction before cycle four. Alopecia and neurotoxicity were minimal: three patients (10%) had grade 2 alopecia and three (10%) had grade 2/3 neurotoxicity. Hand–foot syndrome grade &gt;1 was not observed.</p></sec><sec><title>Second-line therapy</title><p>After failure of first-line therapy, 24 out of the 29 patients (83%) treated at the MTD were considered for second-line treatment; 3 patients had not progressed at the time of the analysis and a further 2 patients were excluded as they were not assessable for progression, as described above. Six patients (25%) were deemed unfit to receive second-line therapy. Eighteen patients received second-line therapy (<xref rid=\"tbl5\" ref-type=\"table\">Table 5</xref>). Nine patients (38%) resumed SCOUT, eight of whom were assessable for response: one patient had a complete response (13%); three had partial responses (38%); one had stable disease (13%) and three had progressive disease (38%). Therefore, the ORR to re-treatment with SCOUT in the MTD group was 50% and the disease control rate was 63%. Twelve patients received other second-line regimens, including capecitabine-, irinotecan- and oxaliplatin-based regimens, as detailed in <xref rid=\"tbl5\" ref-type=\"table\">Table 5</xref>.</p><p>If patients treated at all doses are considered, 36 out of the 45 patients (80%) were considered for second-line therapy, 12 (33%) of whom were deemed unfit for further chemotherapy. Twelve (50%) of the 24 patients who received second-line chemotherapy resumed SCOUT. Two of the additional three patients from the phase I study who resumed SCOUT responded to the re-challenge with stable disease, and one had progressive disease. Therefore, the ORR to re-treatment with SCOUT chemotherapy in all patients was 36% and the disease control rate was 64%.</p><p>Nine out of the 12 patients (75%) who received second-line SCOUT went on to receive a third-line chemotherapy regimen, as shown in <xref rid=\"tbl5\" ref-type=\"table\">Table 5</xref>.</p><p>Out of the remaining 12 patients who received second-line therapies other than SCOUT, 3 (25%) went on to receive a third-line agent (mitomycin C/capecitabine: <italic>n</italic>=2; XELOX: <italic>n</italic>=1).</p></sec></sec> \n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "cnt = 0\n",
    "for index in misclassified[0][500:]:\n",
    "    cnt += 1\n",
    "    print(index, \"prediction:\", y_predictions[index], \"true label:\", np_y_test[index])\n",
    "    print(X_test.iloc[index],\"\\n\\n\\n\")\n",
    "    if cnt == 20:\n",
    "        break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "prediction: other true label: method\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'<sec xmlns:xlink=\"http://www.w3.org/1999/xlink\" xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" sec-type=\"\" id=\"sec2\"><title>Experimental</title><sec id=\"sec2.1\"><title/><sec id=\"sec2.1.1\"><title>Crystal data</title><p>\\n                  <list list-type=\"simple\"><list-item><p>C<sub>22</sub>H<sub>22</sub>N<sub>2</sub>\\n                        </p></list-item><list-item><p>\\n                           <italic>M</italic>\\n                           <italic><sub>r</sub></italic> = 314.42</p></list-item><list-item><p>Trigonal, <inline-formula><inline-graphic xlink:href=\"e-64-00o78-efi4.jpg\" mimetype=\"image\" mime-subtype=\"gif\"/></inline-formula>\\n                        </p></list-item><list-item><p>\\n                           <italic>a</italic> = 21.173 (8) Å</p></list-item><list-item><p>\\n                           <italic>c</italic> = 10.476 (2) Å</p></list-item><list-item><p>\\n                           <italic>V</italic> = 4067 (2) Å<sup>3</sup>\\n                        </p></list-item><list-item><p>\\n                           <italic>Z</italic> = 9</p></list-item><list-item><p>Mo <italic>K</italic>α radiation</p></list-item><list-item><p>μ = 0.07 mm<sup>−1</sup>\\n                        </p></list-item><list-item><p>\\n                           <italic>T</italic> = 293 (2) K</p></list-item><list-item><p>0.21 × 0.18 × 0.15 mm</p></list-item></list>\\n               </p></sec><sec id=\"sec2.1.2\"><title>Data collection</title><p>\\n                  <list list-type=\"simple\"><list-item><p>Enraf–Nonius CAD-4 diffractometer</p></list-item><list-item><p>Absorption correction: none</p></list-item><list-item><p>6148 measured reflections</p></list-item><list-item><p>1956 independent reflections</p></list-item><list-item><p>793 reflections with <italic>I</italic> &gt; 2σ(<italic>I</italic>)</p></list-item><list-item><p>\\n                           <italic>R</italic>\\n                           <sub>int</sub> = 0.075</p></list-item><list-item><p>3 standard reflections every 100 reflections intensity decay: none</p></list-item></list>\\n               </p></sec><sec id=\"sec2.1.3\"><title>Refinement</title><p>\\n                  <list list-type=\"simple\"><list-item><p>\\n                           <italic>R</italic>[<italic>F</italic>\\n                           <sup>2</sup> &gt; 2σ(<italic>F</italic>\\n                           <sup>2</sup>)] = 0.072</p></list-item><list-item><p>\\n                           <italic>wR</italic>(<italic>F</italic>\\n                           <sup>2</sup>) = 0.233</p></list-item><list-item><p>\\n                           <italic>S</italic> = 1.02</p></list-item><list-item><p>1956 reflections</p></list-item><list-item><p>110 parameters</p></list-item><list-item><p>H-atom parameters constrained</p></list-item><list-item><p>Δρ<sub>max</sub> = 0.19 e Å<sup>−3</sup>\\n                        </p></list-item><list-item><p>Δρ<sub>min</sub> = −0.14 e Å<sup>−3</sup>\\n                        </p></list-item></list>\\n               </p></sec></sec><sec id=\"d5e337\"><title/><p>Data collection: <italic>CAD-4 Software</italic> (Enraf–Nonius, 1989<xref ref-type=\"bibr\" rid=\"bb2\"> ▶</xref>); cell refinement: <italic>CAD-4 Software</italic>; data reduction: <italic>NRCVAX</italic> (Gabe <italic>et al.</italic>, 1989<xref ref-type=\"bibr\" rid=\"bb4\"> ▶</xref>); program(s) used to solve structure: <italic>SHELXS97</italic> (Sheldrick, 1997<xref ref-type=\"bibr\" rid=\"bb8\"> ▶</xref>); program(s) used to refine structure: <italic>SHELXL97</italic> (Sheldrick, 1997<xref ref-type=\"bibr\" rid=\"bb8\"> ▶</xref>); molecular graphics: <italic>SHELXTL/PC</italic> (Sheldrick, 1990<xref ref-type=\"bibr\" rid=\"bb7\"> ▶</xref>); software used to prepare material for publication: <italic>WinGX</italic> (Farrugia, 1999<xref ref-type=\"bibr\" rid=\"bb3\"> ▶</xref>).</p></sec></sec>'"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "index = 7\n",
    "print(\"prediction:\", y_predictions[index], \"true label:\", np_y_test[index])\n",
    "X_test.iloc[index] # works even though X_test index is between 32000-40000"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "or use cross validation "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.model_selection import cross_validate"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [],
   "source": [
    "sk_model = LogisticRegression(random_state = 0)\n",
    "scoring = ['f1_macro', 'precision_macro', 'recall_macro', 'accuracy']\n",
    "scores = cross_validate(sk_model, embeddings_train, labels_train, \n",
    "                        scoring=scoring,cv=5, return_train_score=True) # cv =rskf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'fit_time': array([0.93449569, 1.12037468, 1.04741478, 1.00343919, 1.06040764]),\n",
       " 'score_time': array([0.11991429, 0.1769011 , 0.16690731, 0.19689083, 0.10893917]),\n",
       " 'test_f1_macro': array([0.87622222, 0.87481792, 0.87889957, 0.87110543, 0.87286268]),\n",
       " 'train_f1_macro': array([0.87727107, 0.87789996, 0.87684717, 0.87924096, 0.87814096]),\n",
       " 'test_precision_macro': array([0.88884046, 0.89740086, 0.89900688, 0.88791116, 0.88719244]),\n",
       " 'train_precision_macro': array([0.89524   , 0.89488418, 0.89458883, 0.89642344, 0.89599976]),\n",
       " 'test_recall_macro': array([0.86546091, 0.85742225, 0.86299287, 0.85743794, 0.86088778]),\n",
       " 'train_recall_macro': array([0.86277175, 0.86405312, 0.86250291, 0.86524919, 0.8637068 ]),\n",
       " 'test_accuracy': array([0.90974843, 0.91052052, 0.91302296, 0.90688896, 0.90767537]),\n",
       " 'train_accuracy': array([0.91149294, 0.91177164, 0.91114606, 0.91275801, 0.91208964])}"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "scores"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "There is also a sklearn wrapper that I don't know how to use and isn't super well documented.\n",
    "I'd have to change up how I am using my dfs. See here:\n",
    "https://github.com/RaRe-Technologies/gensim/blob/master/docs/notebooks/sklearn_api.ipynb\n",
    "and docs:\n",
    "https://radimrehurek.com/gensim/sklearn_api/d2vmodel.html"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 60,
   "metadata": {},
   "outputs": [],
   "source": [
    "from gensim.sklearn_api import D2VTransformer"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_D2V = D2VTransformer(min_count=1)\n",
    "#embedding_train = model_D2V.fit(X_train)\n",
    "# do some more stuff\n",
    "#embedding_test = model_D2V.transform(X_test)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "and then use those doc2vec embeddings with sklearn"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rskf = RepeatedStratifiedKFold(n_splits = 5, n_repeats = 5, random_state = 1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "model_svc = LinearSVC()\n",
    "model_svc.fit(embedding_train, y_train)\n",
    "y_predictions = model_svc.predict(embedding_test)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "how does the model do?"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "print(\"f1\", f1_score(y_test, y_predictions, average = 'macro'))\n",
    "print(\"recall\", recall_score(y_test, y_predictions, average = 'macro'))\n",
    "print(\"accuracy\", accuracy_score(y_test, y_predictions))\n",
    "print(\"precision\", precision_score(y_test, y_predictions, average = 'macro'))\n",
    "print(\"confusion_matrix\", confusion_matrix(y_test, y_predictions))"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
